<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  
  <meta name="renderer" content="webkit">
  <meta http-equiv="X-UA-Compatible" content="IE=edge" >
  <link rel="dns-prefetch" href="http://example.com">
  <title>Hongtu Liu</title>
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
  <meta property="og:type" content="website">
<meta property="og:title" content="Hongtu Liu">
<meta property="og:url" content="http://example.com/page/8/index.html">
<meta property="og:site_name" content="Hongtu Liu">
<meta property="og:locale" content="en_US">
<meta property="article:author" content="Hongtu Liu">
<meta name="twitter:card" content="summary">
  
    <link rel="alternative" href="/atom.xml" title="Hongtu Liu" type="application/atom+xml">
  
  
    <link rel="icon" href="/favicon.png">
  
  <link rel="stylesheet" type="text/css" href="/./main.0cf68a.css">
  <style type="text/css">
  
    #container.show {
      background: linear-gradient(200deg,#a0cfe4,#e8c37e);
    }
  </style>
  

  

<meta name="generator" content="Hexo 5.3.0"></head>

<body>
  <div id="container" q-class="show:isCtnShow">
    <canvas id="anm-canvas" class="anm-canvas"></canvas>
    <div class="left-col" q-class="show:isShow">
      
<div class="overlay" style="background: #4d4d4d"></div>
<div class="intrude-less">
	<header id="header" class="inner">
		<a href="/" class="profilepic">
			<img src="" class="js-avatar">
		</a>
		<hgroup>
		  <h1 class="header-author"><a href="/"></a></h1>
		</hgroup>
		

		<nav class="header-menu">
			<ul>
			
				<li><a href="/">Home</a></li>
	        
				<li><a href="/categories">Categories</a></li>
	        
				<li><a href="/tags/%E9%9A%8F%E7%AC%94/">随笔</a></li>
	        
			</ul>
		</nav>
		<nav class="header-smart-menu">
    		
    			
    			<a q-on="click: openSlider(e, 'innerArchive')" href="javascript:void(0)">All Items</a>
    			
            
    			
    			<a q-on="click: openSlider(e, 'friends')" href="javascript:void(0)">Links</a>
    			
            
    			
    			<a q-on="click: openSlider(e, 'aboutme')" href="javascript:void(0)">About Me</a>
    			
            
		</nav>
		<nav class="header-nav">
			<div class="social">
				
					<a class="github" target="_blank" href="#" title="github"><i class="icon-github"></i></a>
		        
					<a class="weibo" target="_blank" href="#" title="weibo"><i class="icon-weibo"></i></a>
		        
					<a class="rss" target="_blank" href="#" title="rss"><i class="icon-rss"></i></a>
		        
					<a class="zhihu" target="_blank" href="#" title="zhihu"><i class="icon-zhihu"></i></a>
		        
			</div>
		</nav>
	</header>		
</div>

    </div>
    <div class="mid-col" q-class="show:isShow,hide:isShow|isFalse">
      
<nav id="mobile-nav">
  	<div class="overlay js-overlay" style="background: #4d4d4d"></div>
	<div class="btnctn js-mobile-btnctn">
  		<div class="slider-trigger list" q-on="click: openSlider(e)"><i class="icon icon-sort"></i></div>
	</div>
	<div class="intrude-less">
		<header id="header" class="inner">
			<div class="profilepic">
				<img src="" class="js-avatar">
			</div>
			<hgroup>
			  <h1 class="header-author js-header-author"></h1>
			</hgroup>
			
			
			
				
			
				
			
				
			
			
			
			<nav class="header-nav">
				<div class="social">
					
						<a class="github" target="_blank" href="#" title="github"><i class="icon-github"></i></a>
			        
						<a class="weibo" target="_blank" href="#" title="weibo"><i class="icon-weibo"></i></a>
			        
						<a class="rss" target="_blank" href="#" title="rss"><i class="icon-rss"></i></a>
			        
						<a class="zhihu" target="_blank" href="#" title="zhihu"><i class="icon-zhihu"></i></a>
			        
				</div>
			</nav>

			<nav class="header-menu js-header-menu">
				<ul style="width: 70%">
				
				
					<li style="width: 33.333333333333336%"><a href="/">Home</a></li>
		        
					<li style="width: 33.333333333333336%"><a href="/categories">Categories</a></li>
		        
					<li style="width: 33.333333333333336%"><a href="/tags/%E9%9A%8F%E7%AC%94/">随笔</a></li>
		        
				</ul>
			</nav>
		</header>				
	</div>
	<div class="mobile-mask" style="display:none" q-show="isShow"></div>
</nav>

      <div id="wrapper" class="body-wrap">
        <div class="menu-l">
          <div class="canvas-wrap">
            <canvas data-colors="#eaeaea" data-sectionHeight="100" data-contentId="js-content" id="myCanvas1" class="anm-canvas"></canvas>
          </div>
          <div id="js-content" class="content-ll">
            
  
    <article id="post-20210424" class="article article-type-post  article-index" itemscope itemprop="blogPost">
  <div class="article-inner">
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2021/04/24/20210424/">20210424</a>
    </h1>
  

        
        <a href="/2021/04/24/20210424/" class="archive-article-date">
  	<time datetime="2021-04-23T16:00:00.000Z" itemprop="datePublished"><i class="icon-calendar icon"></i>20210424</time>
</a>
        
      </header>
    
    <div class="article-entry" itemprop="articleBody">
    

      
<div id="toc" class="toc-article">
  <strong class="toc-title">Contents</strong>
  <ol class="toc"><li class="toc-item toc-level-1"><a class="toc-link" href="#Working-directories"><span class="toc-number">1.</span> <span class="toc-text">Working directories</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#Google-search"><span class="toc-number">1.1.</span> <span class="toc-text">Google search</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#%E8%AE%BE%E5%A4%87%E6%9C%AA%E8%8E%B7%E5%BE%97play%E4%BF%9D%E6%8A%A4%E6%9C%BA%E5%88%B6%E8%AE%A4%E8%AF%81"><span class="toc-number">1.2.</span> <span class="toc-text">设备未获得play保护机制认证</span></a></li></ol></li><li class="toc-item toc-level-1"><a class="toc-link" href="#Age-Adjusted-Rates"><span class="toc-number">2.</span> <span class="toc-text">Age Adjusted Rates</span></a></li><li class="toc-item toc-level-1"><a class="toc-link" href="#iPlay30-Pro"><span class="toc-number">3.</span> <span class="toc-text">iPlay30 Pro</span></a></li><li class="toc-item toc-level-1"><a class="toc-link" href="#PDB"><span class="toc-number">4.</span> <span class="toc-text">PDB</span></a></li></ol>
</div>

        <h1 id="Working-directories"><a href="#Working-directories" class="headerlink" title="Working directories"></a>Working directories</h1><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;苏州CDC&#x2F;苏州项目-提纲_草.docx&#39;</span><br><span class="line">setsid wps $&#123;i&#125;</span><br><span class="line">#</span><br></pre></td></tr></table></figure>
<h2 id="Google-search"><a href="#Google-search" class="headerlink" title="Google search"></a>Google search</h2><ul>
<li>设备未获得play保护机制认证</li>
</ul>
<h2 id="设备未获得play保护机制认证"><a href="#设备未获得play保护机制认证" class="headerlink" title="设备未获得play保护机制认证"></a>设备未获得play保护机制认证</h2><ul>
<li><a target="_blank" rel="noopener" href="http://www.xiaxuelo.com/2020/05/13/%E8%AE%BE%E5%A4%87%E6%9C%AA%E8%8E%B7%E5%BE%97play%E4%BF%9D%E6%8A%A4%E6%9C%BA%E5%88%B6%E8%AE%A4%E8%AF%81%E7%9A%84%E8%A7%A3%E5%86%B3%E6%96%B9%E6%B3%95/">设备未获得play保护机制认证的解决方法 – 让心灵躲躲雨</a></li>
<li><a target="_blank" rel="noopener" href="https://www.bilibili.com/video/BV11i4y1T7Wu/?spm_id_from=333.788.videocard.5">mate40/P40等华为手机安装GooglePlay荣耀30系列和V30和nova8系列和mateX2谷歌框架的方法。华为安装谷歌教程。真机安装方法!_哔哩哔哩 (゜-゜)つロ 干杯~-bilibili</a></li>
<li><a target="_blank" rel="noopener" href="https://club.huawei.com/thread-22834424-1-1.html">（已解决）谷歌密码都输完了，设备未获得play保护机制认证？ - 华为nova6系列功能建议 花粉俱乐部</a></li>
<li><a target="_blank" rel="noopener" href="https://www.3cep.cn/index.php/2020/08/07/%E5%AE%89%E8%A3%85google-play%E5%95%86%E5%BA%97%E5%87%BA%E7%8E%B0%E6%AD%A4%E8%AE%BE%E5%A4%87%E6%9C%AA%E7%BB%8Fplay%E4%BF%9D%E6%8A%A4%E6%9C%BA%E5%88%B6%E8%AE%A4%E8%AF%81%E9%97%AE/">安装Google Play商店出现“此设备未经Play保护机制认证”问题-维修狮</a></li>
<li><a target="_blank" rel="noopener" href="https://zhangyiming748.github.io/2020/03/06/GoogleSecurityDevices/">设备未获得play保护机制认证的解决方法 - 技术博客</a></li>
</ul>
<h1 id="Age-Adjusted-Rates"><a href="#Age-Adjusted-Rates" class="headerlink" title="Age Adjusted Rates"></a>Age Adjusted Rates</h1><ul>
<li><a target="_blank" rel="noopener" href="https://www.google.com/search?q=R+adjusted+incidence+rate&oq=R+adjusted+incidence+rate&aqs=chrome..69i57j0l7.7539j0j7&client=ubuntu&sourceid=chrome&ie=UTF-8">R adjusted incidence rate - Google Search</a></li>
<li><a target="_blank" rel="noopener" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0235562">Age and sex-adjusted incidence and yearly prevalence of multiple sclerosis (MS) in Mazandaran province, Iran: An 11-years study</a></li>
<li><a target="_blank" rel="noopener" href="https://cran.r-project.org/web/packages/epiR/epiR.pdf">epiR.pdf</a></li>
<li><a target="_blank" rel="noopener" href="https://rdrr.io/cran/epiR/man/epi.directadj.html">epi.directadj: Directly adjusted incidence rate estimates in epiR: Tools for the Analysis of Epidemiological Data</a></li>
<li><a target="_blank" rel="noopener" href="https://www.rdocumentation.org/packages/epiR/versions/2.0.19/topics/epi.2by2">epi.2by2 function - RDocumentation</a></li>
<li><a target="_blank" rel="noopener" href="https://cran.r-project.org/web/packages/epiR/vignettes/epiR_descriptive.html">Descriptive Epidemiology using epiR</a></li>
<li><a target="_blank" rel="noopener" href="https://rpubs.com/bpoulin-CUNY/321735">RPubs - Age Adjusted Rates</a></li>
<li><a target="_blank" rel="noopener" href="https://seer.cancer.gov/seerstat/tutorials/aarates/step3.html">Step 3: Calculating Age-adjusted Rates - SEER*Stat Tutorials</a></li>
</ul>
<h1 id="iPlay30-Pro"><a href="#iPlay30-Pro" class="headerlink" title="iPlay30 Pro"></a>iPlay30 Pro</h1><ul>
<li><a target="_blank" rel="noopener" href="https://pan.baidu.com/s/1FzTDP6CIU524P1x2gW1ttg?_at_=1619283975692">iPlay30 Pro(T1012C)-Android10-210129-固件&amp;教程.rar_免费高速下载|百度网盘-分享无限制</a>, jn9m</li>
<li><a target="_blank" rel="noopener" href="https://alldocube.lanzous.com/b00ocv5oh">iPlay30教育soft</a>, h27j</li>
<li><a target="_blank" rel="noopener" href="https://pan.baidu.com/s/1URcwiBUJc-qrklXO_9DlWQ">iPlay30(T1012)-Android10-201020-固件&amp;教程.rar_免费高速下载|百度网盘-分享无限制</a>, grc6</li>
</ul>
<ul>
<li><a target="_blank" rel="noopener" href="https://www.google.com/search?q=iplay30+pro+%E9%AD%94%E6%96%B9&newwindow=1&client=ubuntu&hs=q2O&sxsrf=ALeKk02U2Z1Ackr5IyO6pEndQTyPSaIAFA:1619282725329&ei=JUuEYIG3E8f3qwH4mKiADA&oq=iplay30+pro+%E9%AD%94%E6%96%B9&gs_lcp=Cgdnd3Mtd2l6EAMyBwgjELECECcyBwgjELECECcyBwgjELECECcyBwgAEMkDEAoyBAgAEAoyBAgAEAoyBAgAEAoyBAgAEAoyBAgAEAoyBAgAEAo6BwgAEEcQsAM6BwgAELADEENQ1IsBWIW4AWCcuQFoAXACeACAAeATiAGTF5IBBzQtMS45LTGYAQCgAQGqAQdnd3Mtd2l6yAEJwAEB&sclient=gws-wiz&ved=0ahUKEwjB3YidqpfwAhXH-yoKHXgMCsAQ4dUDCA4&uact=5">iplay30 pro 魔方 - Google Search</a></li>
<li><a target="_blank" rel="noopener" href="https://www.51cube.com/firmware/?search=iplay30+pro">全部固件 – 酷比魔方官网</a></li>
<li><a target="_blank" rel="noopener" href="https://www.51cube.com/firmware/iplay30-t1012-fimware-download/">iPlay30(T1012)-Android10-201020-固件及教程 – 酷比魔方官网</a></li>
<li><a target="_blank" rel="noopener" href="https://www.51cube.com/firmware/iplay30-education-app-download/">iPlay30/iPlay30 Pro 适配教育软件下载 – 酷比魔方官网</a></li>
<li><a target="_blank" rel="noopener" href="https://www.alldocube.com/en/support/">Support – Alldocube Official Site</a></li>
<li><a target="_blank" rel="noopener" href="https://www.51cube.com/firmware/iplay30pro-t1012c-firmware-download/">iPlay30 Pro(T1012C)-Android10-210129-固件及教程 – 酷比魔方官网</a></li>
</ul>
<h1 id="PDB"><a href="#PDB" class="headerlink" title="PDB"></a>PDB</h1><ul>
<li><a target="_blank" rel="noopener" href="https://pubs.acs.org/doi/10.1021/acs.jctc.9b00474#">Protein Docking Using a Single Representation for Protein Surface, Electrostatics, and Local Dynamics | Journal of Chemical Theory and Computation</a></li>
<li><a target="_blank" rel="noopener" href="https://www.sciencedirect.com/topics/medicine-and-dentistry/docking-protein">Docking Protein - an overview | ScienceDirect Topics</a></li>
<li><a target="_blank" rel="noopener" href="https://www.google.com/search?q=pdb+%E6%A0%BC%E5%BC%8F&oq=pdb+%E6%A0%BC%E5%BC%8F&aqs=chrome..69i57j0j0i67j0l5.7119j0j1&client=ubuntu&sourceid=chrome&ie=UTF-8">pdb 格式 - Google Search</a></li>
<li><a target="_blank" rel="noopener" href="https://blog.chembiosim.com/PDB-file-format/">PDB 文件格式说明 | Hope</a></li>
<li><a target="_blank" rel="noopener" href="http://homepage.fudan.edu.cn/wenningwang/academic/technology/pdb/">PDB文件的获得和准备 | 欢迎来到王文宁课题组</a></li>
<li><a target="_blank" rel="noopener" href="http://www.dwenzhao.cn/lesson/datalibuse/pdbformat.html">www.dwenzhao.cn/lesson/datalibuse/pdbformat.html</a></li>
<li><a target="_blank" rel="noopener" href="https://jerkwin.github.io/2015/06/05/PDB%E6%96%87%E4%BB%B6%E6%A0%BC%E5%BC%8F%E8%AF%B4%E6%98%8E/#:~:text=PDB(Protein%20Data%20Bank)%E6%98%AF,%E4%BD%BF%E7%94%A8%E8%BF%99%E7%A7%8D%E6%A0%87%E5%87%86%E6%A0%BC%E5%BC%8F.">PDB文件格式说明|Jerkwin</a></li>
</ul>

      

      
    </div>
    <div class="article-info article-info-index">
      
      
      
	<div class="article-category tagcloud">
		<i class="icon-book icon"></i>
		<ul class="article-tag-list">
			 
        		<li class="article-tag-list-item">
        			<a href="/categories/Log//" class="article-tag-list-link color4">Log</a>
        		</li>
      		
		</ul>
	</div>


      
        <p class="article-more-link">
          <a class="article-more-a" href="/2021/04/24/20210424/">Expand all items >></a>
        </p>
      

      
      <div class="clearfix"></div>
    </div>
  </div>
</article>

<aside class="wrap-side-operation">
    <div class="mod-side-operation">
        
        <div class="jump-container" id="js-jump-container" style="display:none;">
            <a href="javascript:void(0)" class="mod-side-operation__jump-to-top">
                <i class="icon-font icon-back"></i>
            </a>
            <div id="js-jump-plan-container" class="jump-plan-container" style="top: -11px;">
                <i class="icon-font icon-plane jump-plane"></i>
            </div>
        </div>
        
        
    </div>
</aside>




  
    <article id="post-20210421" class="article article-type-post  article-index" itemscope itemprop="blogPost">
  <div class="article-inner">
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2021/04/21/20210421/">20210421</a>
    </h1>
  

        
        <a href="/2021/04/21/20210421/" class="archive-article-date">
  	<time datetime="2021-04-20T16:00:00.000Z" itemprop="datePublished"><i class="icon-calendar icon"></i>20210421</time>
</a>
        
      </header>
    
    <div class="article-entry" itemprop="articleBody">
    

      
<div id="toc" class="toc-article">
  <strong class="toc-title">Contents</strong>
  <ol class="toc"><li class="toc-item toc-level-1"><a class="toc-link" href="#Working-directories"><span class="toc-number">1.</span> <span class="toc-text">Working directories</span></a></li><li class="toc-item toc-level-1"><a class="toc-link" href="#Bacteriophage"><span class="toc-number">2.</span> <span class="toc-text">Bacteriophage</span></a></li><li class="toc-item toc-level-1"><a class="toc-link" href="#Hexo"><span class="toc-number">3.</span> <span class="toc-text">Hexo</span></a></li></ol>
</div>

        <h1 id="Working-directories"><a href="#Working-directories" class="headerlink" title="Working directories"></a>Working directories</h1><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br></pre></td><td class="code"><pre><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;HIV&#39;; cd $&#123;i&#125;</span><br><span class="line">#</span><br><span class="line">wget https:&#x2F;&#x2F;nodejs.org&#x2F;dist&#x2F;latest&#x2F;node-v16.0.0-linux-x64.tar.gz</span><br><span class="line">i&#x3D;&#39;node-v16.0.0-linux-x64.tar.gz&#39;</span><br><span class="line">tar -zxvf $&#123;i&#125;</span><br></pre></td></tr></table></figure>
<h1 id="Bacteriophage"><a href="#Bacteriophage" class="headerlink" title="Bacteriophage"></a>Bacteriophage</h1><ul>
<li><a href="/media/ht/ht_5T/Work/Projects/Phage/Ref/A-rapid-and-simple-protocol-for-the-isolation-of-bacteriophages-f_2019_Metho.pdf">A-rapid-and-simple-protocol-for-the-isolation-of-bacteriophages-f_2019_Metho.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/Phage/Ref/Bacteriophage_remediation_of_bacterial_pathogens_in_aquaculture-a_review_of_the_technology.pdf">Bacteriophage_remediation_of_bacterial_pathogens_in_aquaculture-a_review_of_the_technology.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/Phage/Ref/Bacteriophages_for_prophylaxis_n_therapy_in_cattle,_poultry_n_pigs.pdf">Bacteriophages_for_prophylaxis_n_therapy_in_cattle,_poultry_n_pigs.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/Phage/Ref/Biotechnological_applications_of_bacteriophages-State_of_the_art.pdf">Biotechnological_applications_of_bacteriophages-State_of_the_art.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/Phage/Ref/Disparate_effects_of_bacteriophages_on_antibiotic-resistant_bacteria.pdf">Disparate_effects_of_bacteriophages_on_antibiotic-resistant_bacteria.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/Phage/Ref/Hospital-acquired_infection.pdf">Hospital-acquired_infection.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/Phage/Ref/In_vivo_efficiency_evaluation_of_a_phage_cocktail_in_controlling_severe_colibacillosis_in_confined_conditions_n_experimental_poultry_houses.pdf">In_vivo_efficiency_evaluation_of_a_phage_cocktail_in_controlling_severe_colibacillosis_in_confined_conditions_n_experimental_poultry_houses.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/Phage/Ref/Master-mechanisms-of-Staphylococcus-aureus--consider-its-ex_2018_Microbiolog.pdf">Master-mechanisms-of-Staphylococcus-aureus–consider-its-ex_2018_Microbiolog.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/Phage/Ref/Pahge_20210406.pdf">Pahge_20210406.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/Phage/Ref/Phage_20210421.pdf">Phage_20210421.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/Phage/Ref/Phage-based-nanomaterials-for-biomedical-applications_2014_Acta-Biomateriali.pdf">Phage-based-nanomaterials-for-biomedical-applications_2014_Acta-Biomateriali.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/Phage/Ref/Phage_Genomics.pdf">Phage_Genomics.pdf</a></li>
</ul>
<h1 id="Hexo"><a href="#Hexo" class="headerlink" title="Hexo"></a>Hexo</h1><ul>
<li><a target="_blank" rel="noopener" href="https://nodejs.org/dist/latest/">Index of /dist/latest/</a></li>
</ul>
<hr>
<ul>
<li><a target="_blank" rel="noopener" href="https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1000373">Synthetic Double-Stranded RNAs Are Adjuvants for the Induction of T Helper 1 and Humoral Immune Responses to Human Papillomavirus in Rhesus Macaques</a></li>
<li><a target="_blank" rel="noopener" href="https://www.myfreax.com/how-to-install-node-js-on-ubuntu-18-04/">如何在Ubuntu 18.04上安装Node.js和npm</a></li>
<li><a target="_blank" rel="noopener" href="https://www.jianshu.com/p/9b2f2de68838">ubuntu下搭建Hexo+GitHub博客 - 简书</a></li>
<li><a target="_blank" rel="noopener" href="https://zhuanlan.zhihu.com/p/64283853">在ubuntu上搭建HEXO博客 - 知乎</a></li>
<li><a target="_blank" rel="noopener" href="https://ywnz.com/linuxyffq/2835.html">在Ubuntu系统中Hexo安装的方法（非常简单的方法）_Linux云服务器_云网牛站</a></li>
<li><a target="_blank" rel="noopener" href="https://developer.aliyun.com/article/760687">如何在 Ubuntu 20.04 上安装 Node.js 和 npm-阿里云开发者社区</a></li>
<li><a target="_blank" rel="noopener" href="https://www.jianshu.com/p/18f1fb45a70b">ubuntu 安装npm - 简书</a></li>
<li><a target="_blank" rel="noopener" href="https://www.howtoforge.com/how-to-install-and-create-a-blog-with-hexo-on-ubuntu-20-04/">How to Install and Create a Blog with Hexo on Ubuntu 20.04</a></li>
<li><a target="_blank" rel="noopener" href="https://blog.kavoori.com/2014-10-13/installing-hexo-on-ubuntu.html">Rabi Kavoori: Installing Hexo on Ubuntu</a></li>
<li><a target="_blank" rel="noopener" href="https://www.reuters.com/business/healthcare-pharmaceuticals/chinas-sinovac-covid-19-vaccine-67-effective-preventing-symptomatic-infection-2021-04-16/">China’s Sinovac COVID-19 vaccine 67% effective in preventing symptomatic infection - Chile govt report | Reuters</a></li>
<li><a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/creationix/nvm/master/install.sh">https://raw.githubusercontent.com/creationix/nvm/master/install.sh</a></li>
</ul>

      

      
    </div>
    <div class="article-info article-info-index">
      
      
	<div class="article-tag tagcloud">
		<i class="icon-price-tags icon"></i>
		<ul class="article-tag-list">
			 
        		<li class="article-tag-list-item">
        			<a href="javascript:void(0)" class="js-tag article-tag-list-link color4">Bacteriophage</a>
        		</li>
      		 
        		<li class="article-tag-list-item">
        			<a href="javascript:void(0)" class="js-tag article-tag-list-link color5">Hexo</a>
        		</li>
      		
		</ul>
	</div>

      
	<div class="article-category tagcloud">
		<i class="icon-book icon"></i>
		<ul class="article-tag-list">
			 
        		<li class="article-tag-list-item">
        			<a href="/categories/Log//" class="article-tag-list-link color4">Log</a>
        		</li>
      		
		</ul>
	</div>


      
        <p class="article-more-link">
          <a class="article-more-a" href="/2021/04/21/20210421/">Expand all items >></a>
        </p>
      

      
      <div class="clearfix"></div>
    </div>
  </div>
</article>

<aside class="wrap-side-operation">
    <div class="mod-side-operation">
        
        <div class="jump-container" id="js-jump-container" style="display:none;">
            <a href="javascript:void(0)" class="mod-side-operation__jump-to-top">
                <i class="icon-font icon-back"></i>
            </a>
            <div id="js-jump-plan-container" class="jump-plan-container" style="top: -11px;">
                <i class="icon-font icon-plane jump-plane"></i>
            </div>
        </div>
        
        
    </div>
</aside>




  
    <article id="post-20210420" class="article article-type-post  article-index" itemscope itemprop="blogPost">
  <div class="article-inner">
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2021/04/20/20210420/">20210420</a>
    </h1>
  

        
        <a href="/2021/04/20/20210420/" class="archive-article-date">
  	<time datetime="2021-04-19T16:00:00.000Z" itemprop="datePublished"><i class="icon-calendar icon"></i>20210420</time>
</a>
        
      </header>
    
    <div class="article-entry" itemprop="articleBody">
    

      
<div id="toc" class="toc-article">
  <strong class="toc-title">Contents</strong>
  <ol class="toc"><li class="toc-item toc-level-1"><a class="toc-link" href="#Working-directories"><span class="toc-number">1.</span> <span class="toc-text">Working directories</span></a></li><li class="toc-item toc-level-1"><a class="toc-link" href="#Bacteriophage"><span class="toc-number">2.</span> <span class="toc-text">Bacteriophage</span></a></li></ol>
</div>

        <h1 id="Working-directories"><a href="#Working-directories" class="headerlink" title="Working directories"></a>Working directories</h1><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">cd &#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;HIV # send HIV proposal to Sun Liwen</span><br><span class="line">#</span><br></pre></td></tr></table></figure>
<h1 id="Bacteriophage"><a href="#Bacteriophage" class="headerlink" title="Bacteriophage"></a>Bacteriophage</h1><ul>
<li><a href="/media/ht/ht_5T/Work/Projects/Phage/Ref/Bacteriophage_remediation_of_bacterial_pathogens_in_aquaculture-a_review_of_the_technology.pdf">Bacteriophage_remediation_of_bacterial_pathogens_in_aquaculture-a_review_of_the_technology.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/Phage/Ref/Bacteriophages_for_prophylaxis_n_therapy_in_cattle,_poultry_n_pigs.pdf">Bacteriophages_for_prophylaxis_n_therapy_in_cattle,_poultry_n_pigs.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/Phage/Ref/In_vivo_efficiency_evaluation_of_a_phage_cocktail_in_controlling_severe_colibacillosis_in_confined_conditions_n_experimental_poultry_houses.pdf">In_vivo_efficiency_evaluation_of_a_phage_cocktail_in_controlling_severe_colibacillosis_in_confined_conditions_n_experimental_poultry_houses.pdf</a></li>
</ul>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br></pre></td><td class="code"><pre><span class="line">setsid okular \</span><br><span class="line">&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;Phage&#x2F;Ref&#x2F;Bacteriophage_remediation_of_bacterial_pathogens_in_aquaculture-a_review_of_the_technology.pdf \</span><br><span class="line">&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;Phage&#x2F;Ref&#x2F;Bacteriophages_for_prophylaxis_n_therapy_in_cattle,_poultry_n_pigs.pdf \</span><br><span class="line">&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;Phage&#x2F;Ref&#x2F;In_vivo_efficiency_evaluation_of_a_phage_cocktail_in_controlling_severe_colibacillosis_in_confined_conditions_n_experimental_poultry_houses.pdf</span><br><span class="line">#</span><br><span class="line">pdftk &#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;Phage&#x2F;Ref&#x2F;Bacteriophage_remediation_of_bacterial_pathogens_in_aquaculture-a_review_of_the_technology.pdf \</span><br><span class="line">&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;Phage&#x2F;Ref&#x2F;Bacteriophages_for_prophylaxis_n_therapy_in_cattle,_poultry_n_pigs.pdf \</span><br><span class="line">&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;Phage&#x2F;Ref&#x2F;In_vivo_efficiency_evaluation_of_a_phage_cocktail_in_controlling_severe_colibacillosis_in_confined_conditions_n_experimental_poultry_houses.pdf cat output &#96;Date&#96;.pdf</span><br></pre></td></tr></table></figure>
      

      
    </div>
    <div class="article-info article-info-index">
      
      
	<div class="article-tag tagcloud">
		<i class="icon-price-tags icon"></i>
		<ul class="article-tag-list">
			 
        		<li class="article-tag-list-item">
        			<a href="javascript:void(0)" class="js-tag article-tag-list-link color4">Bacteriophage</a>
        		</li>
      		 
        		<li class="article-tag-list-item">
        			<a href="javascript:void(0)" class="js-tag article-tag-list-link color3">凉山</a>
        		</li>
      		
		</ul>
	</div>

      
	<div class="article-category tagcloud">
		<i class="icon-book icon"></i>
		<ul class="article-tag-list">
			 
        		<li class="article-tag-list-item">
        			<a href="/categories/Log//" class="article-tag-list-link color4">Log</a>
        		</li>
      		
		</ul>
	</div>


      
        <p class="article-more-link">
          <a class="article-more-a" href="/2021/04/20/20210420/">Expand all items >></a>
        </p>
      

      
      <div class="clearfix"></div>
    </div>
  </div>
</article>

<aside class="wrap-side-operation">
    <div class="mod-side-operation">
        
        <div class="jump-container" id="js-jump-container" style="display:none;">
            <a href="javascript:void(0)" class="mod-side-operation__jump-to-top">
                <i class="icon-font icon-back"></i>
            </a>
            <div id="js-jump-plan-container" class="jump-plan-container" style="top: -11px;">
                <i class="icon-font icon-plane jump-plane"></i>
            </div>
        </div>
        
        
    </div>
</aside>




  
    <article id="post-20210418" class="article article-type-post  article-index" itemscope itemprop="blogPost">
  <div class="article-inner">
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2021/04/18/20210418/">20210418</a>
    </h1>
  

        
        <a href="/2021/04/18/20210418/" class="archive-article-date">
  	<time datetime="2021-04-17T16:00:00.000Z" itemprop="datePublished"><i class="icon-calendar icon"></i>20210418</time>
</a>
        
      </header>
    
    <div class="article-entry" itemprop="articleBody">
    

      
<div id="toc" class="toc-article">
  <strong class="toc-title">Contents</strong>
  <ol class="toc"><li class="toc-item toc-level-1"><a class="toc-link" href="#Working-directories"><span class="toc-number">1.</span> <span class="toc-text">Working directories</span></a></li><li class="toc-item toc-level-1"><a class="toc-link" href="#Hexo"><span class="toc-number">2.</span> <span class="toc-text">Hexo</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#TOC"><span class="toc-number">2.1.</span> <span class="toc-text">TOC</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#%E5%85%8D%E5%AF%86%E4%B8%8A%E4%BC%A0"><span class="toc-number">2.2.</span> <span class="toc-text">免密上传</span></a></li></ol></li><li class="toc-item toc-level-1"><a class="toc-link" href="#Proteomics"><span class="toc-number">3.</span> <span class="toc-text">Proteomics</span></a></li><li class="toc-item toc-level-1"><a class="toc-link" href="#B95-8-Nrf2-Project"><span class="toc-number">4.</span> <span class="toc-text">B95-8_Nrf2 Project</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#Viral-Load"><span class="toc-number">4.1.</span> <span class="toc-text">Viral Load</span></a></li></ol></li></ol>
</div>

        <h1 id="Working-directories"><a href="#Working-directories" class="headerlink" title="Working directories"></a>Working directories</h1><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br><span class="line">13</span><br><span class="line">14</span><br><span class="line">15</span><br><span class="line">16</span><br><span class="line">17</span><br><span class="line">18</span><br><span class="line">19</span><br><span class="line">20</span><br><span class="line">21</span><br><span class="line">22</span><br><span class="line">23</span><br><span class="line">24</span><br><span class="line">25</span><br><span class="line">26</span><br><span class="line">27</span><br><span class="line">28</span><br><span class="line">29</span><br><span class="line">30</span><br><span class="line">31</span><br><span class="line">32</span><br><span class="line">33</span><br><span class="line">34</span><br><span class="line">35</span><br><span class="line">36</span><br><span class="line">37</span><br><span class="line">38</span><br><span class="line">39</span><br><span class="line">40</span><br><span class="line">41</span><br><span class="line">42</span><br><span class="line">43</span><br><span class="line">44</span><br><span class="line">45</span><br><span class="line">46</span><br><span class="line">47</span><br><span class="line">48</span><br><span class="line">49</span><br><span class="line">50</span><br><span class="line">51</span><br><span class="line">52</span><br><span class="line">53</span><br><span class="line">54</span><br><span class="line">55</span><br><span class="line">56</span><br><span class="line">57</span><br><span class="line">58</span><br><span class="line">59</span><br><span class="line">60</span><br><span class="line">61</span><br><span class="line">62</span><br><span class="line">63</span><br><span class="line">64</span><br></pre></td><td class="code"><pre><span class="line"># ganttproject installation</span><br><span class="line"># &#x2F;usr&#x2F;bin&#x2F;ganttproject %U</span><br><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Stored_Softs&#x2F;Cross_Platform_Softs&#x2F;MS_Project_Alternative&#x2F;GanttProject&#39;; cd $&#123;i&#125;</span><br><span class="line">sudo gdebi &#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Stored_Softs&#x2F;Cross_Platform_Softs&#x2F;MS_Project_Alternative&#x2F;GanttProject&#x2F;ganttproject_3.0.3000-1_all.deb</span><br><span class="line">#</span><br><span class="line"># GanttProject, Command line</span><br><span class="line">&#x2F;usr&#x2F;bin&#x2F;ganttproject %U</span><br><span class="line">ganttproject %U</span><br><span class="line">ganttproject %U &#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Management&#x2F;ht_Gantt.gan</span><br><span class="line">#</span><br><span class="line">&#x2F;usr&#x2F;bin&#x2F;ganttproject %U &#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Management&#x2F;ht_Gantt.gan</span><br><span class="line">#</span><br><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Management&#x2F;ht_Gantt.gan&#39;; &#x2F;usr&#x2F;bin&#x2F;ganttproject %U $&#123;i&#125;</span><br><span class="line">#</span><br><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;ht.gan&#39;; ganttproject $&#123;i&#125;</span><br><span class="line">#</span><br><span class="line">sudo apt install img2pdf</span><br><span class="line">#</span><br><span class="line">cd &#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;NFE&#x2F;B95-8&#x2F;Ref&#x2F;20210414&#x2F;EBV&#x2F;EBV_Ref&#x2F;075_Differentiation-Dependent_KLF4_Expression_Promotes_Lytic_EBV_Infection_in_Epithelial_Cells</span><br><span class="line">#</span><br><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;NFE&#x2F;B95-8&#x2F;Ref&#x2F;20210414&#x2F;EBV&#x2F;EBV_Ref&#x2F;075_Differentiation-Dependent_KLF4_Expression_Promotes_Lytic_EBV_Infection_in_Epithelial_Cells&#39;; cd $&#123;i&#125;</span><br><span class="line">#</span><br><span class="line">wget https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC4592227&#x2F;bin&#x2F;ppat.1005195.s008.tif https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC4592227&#x2F;bin&#x2F;ppat.1005195.s007.tif https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC4592227&#x2F;bin&#x2F;ppat.1005195.s006.tif https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC4592227&#x2F;bin&#x2F;ppat.1005195.s005.tif https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC4592227&#x2F;bin&#x2F;ppat.1005195.s004.tif https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC4592227&#x2F;bin&#x2F;ppat.1005195.s003.tif https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC4592227&#x2F;bin&#x2F;ppat.1005195.s002.tif https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC4592227&#x2F;bin&#x2F;ppat.1005195.s001.tif</span><br><span class="line">#</span><br><span class="line">wget https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC4592227&#x2F;bin&#x2F;ppat.1005195.s008.tif https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC4592227&#x2F;bin&#x2F;ppat.1005195.s007.tif https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC4592227&#x2F;bin&#x2F;ppat.1005195.s006.tif https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC4592227&#x2F;bin&#x2F;ppat.1005195.s005.tif https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC4592227&#x2F;bin&#x2F;ppat.1005195.s004.tif  https:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;pmc&#x2F;articles&#x2F;PMC4592227&#x2F;bin&#x2F;ppat.1005195.s001.tif</span><br><span class="line">#</span><br><span class="line">cd &#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;NFE&#x2F;B95-8&#x2F;Ref&#x2F;20210414&#x2F;EBV&#x2F;EBV_Ref&#x2F;006_LMP1_and_2A_Induce_the_Expression_of_Nrf2_Through_Akt_Signaling_Pathway_in_EBV-Transformed_B_Cells</span><br><span class="line">#</span><br><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;NFE&#x2F;B95-8&#x2F;Ref&#x2F;20210414&#x2F;EBV&#x2F;EBV_Ref&#x2F;006_LMP1_and_2A_Induce_the_Expression_of_Nrf2_Through_Akt_Signaling_Pathway_in_EBV-Transformed_B_Cells&#39;; cd $&#123;i&#125;</span><br><span class="line">#</span><br><span class="line">########################################################################</span><br><span class="line">#</span><br><span class="line">for i in Quercetin-induced_apoptosis_prevents_EBV_infection.pdf \</span><br><span class="line">Cordycepin_is_a_novel_chemical_suppressor_of_EBV.pdf \</span><br><span class="line">Genipin_as_a_novel_chemical_activator_of_EBV_lytic_cycle.pdf \</span><br><span class="line">Rapamycin_enhances_lytic_replication_of_EBV_in_gastric_carcinoma_cells_by_increasing_the_transcriptional_of_lytic_promoters.pdf \</span><br><span class="line">Emodin_Inhibits_EBV_Reactivation_and_Represses_NPC_Tumorigenesis.pdf \</span><br><span class="line">The_mTOR_Inhibitor_Manassantin_B_Reveals-A_Crucial_Role_of_mTORC2_Signaling_in_EBV_Reactivation.pdf \</span><br><span class="line">Inhibition_of_class_I_histone_deacetylases_by_romidepsin_potently_induces_EBV_lytic_cycle_ganciclovir.pdf \</span><br><span class="line">Pharmacologic_Activation_of_Lytic_EBV_Gene_Expression_without_Virion_Production.pdf \</span><br><span class="line">Chaetocin_reactivates_the_lytic_replication_of_EBV_from_latency_via_reactive_oxygen_species.pdf \</span><br><span class="line">EBV-encoded_EBNA1_regulates_cell_viability_by_modulating_miR34a-NOX2-ROS.pdf; do find &#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;NFE&#x2F;B95-8&#x2F;Ref&#x2F; -type f | ag $&#123;i&#125;; done</span><br><span class="line">#</span><br><span class="line">227_Quercetin-induced_apoptosis_prevents_EBV_infection.pdf</span><br><span class="line">226_Cordycepin_is_a_novel_chemical_suppressor_of_EBV.pdf</span><br><span class="line">225_Genipin_as_a_novel_chemical_activator_of_EBV_lytic_cycle.pdf</span><br><span class="line">222_Rapamycin_enhances_lytic_replication_of_EBV_in_gastric_carcinoma_cells_by_increasing_the_transcriptional_of_lytic_promoters.pdf</span><br><span class="line">220_Emodin_Inhibits_EBV_Reactivation_and_Represses_NPC_Tumorigenesis.pdf</span><br><span class="line">203_The_mTOR_Inhibitor_Manassantin_B_Reveals-A_Crucial_Role_of_mTORC2_Signaling_in_EBV_Reactivation.pdf</span><br><span class="line">186_Inhibition_of_class_I_histone_deacetylases_by_romidepsin_potently_induces_EBV_lytic_cycle_ganciclovir.pdf</span><br><span class="line">184_Pharmacologic_Activation_of_Lytic_EBV_Gene_Expression_without_Virion_Production.pdf</span><br><span class="line">101_Chaetocin_reactivates_the_lytic_replication_of_EBV_from_latency_via_reactive_oxygen_species.pdf</span><br><span class="line">243_EBV-encoded_EBNA1_regulates_cell_viability_by_modulating_miR34a-NOX2-ROS.pdf</span><br><span class="line">#</span><br><span class="line">removed &#39;.&#x2F;20210414&#x2F;EBV&#x2F;EBV_Ref&#x2F;227_Quercetin-induced_apoptosis_prevents_EBV_infection.pdf&#39;</span><br><span class="line">removed &#39;.&#x2F;20210414&#x2F;EBV&#x2F;EBV_Ref&#x2F;226_Cordycepin_is_a_novel_chemical_suppressor_of_EBV.pdf&#39;</span><br><span class="line">removed &#39;.&#x2F;20210414&#x2F;EBV&#x2F;EBV_Ref&#x2F;225_Genipin_as_a_novel_chemical_activator_of_EBV_lytic_cycle.pdf&#39;</span><br><span class="line">removed &#39;.&#x2F;20210414&#x2F;EBV&#x2F;EBV_Ref&#x2F;222_Rapamycin_enhances_lytic_replication_of_EBV_in_gastric_carcinoma_cells_by_increasing_the_transcriptional_of_lytic_promoters.pdf&#39;</span><br><span class="line">removed &#39;.&#x2F;20210414&#x2F;EBV&#x2F;EBV_Ref&#x2F;220_Emodin_Inhibits_EBV_Reactivation_and_Represses_NPC_Tumorigenesis.pdf&#39;</span><br><span class="line">removed &#39;.&#x2F;20210414&#x2F;EBV&#x2F;EBV_Ref&#x2F;203_The_mTOR_Inhibitor_Manassantin_B_Reveals-A_Crucial_Role_of_mTORC2_Signaling_in_EBV_Reactivation.pdf&#39;</span><br><span class="line">removed &#39;.&#x2F;20210414&#x2F;EBV&#x2F;EBV_Ref&#x2F;186_Inhibition_of_class_I_histone_deacetylases_by_romidepsin_potently_induces_EBV_lytic_cycle_ganciclovir.pdf&#39;</span><br><span class="line">removed &#39;.&#x2F;20210414&#x2F;EBV&#x2F;EBV_Ref&#x2F;184_Pharmacologic_Activation_of_Lytic_EBV_Gene_Expression_without_Virion_Production.pdf&#39;</span><br><span class="line">removed &#39;.&#x2F;20210414&#x2F;EBV&#x2F;EBV_Ref&#x2F;101_Chaetocin_reactivates_the_lytic_replication_of_EBV_from_latency_via_reactive_oxygen_species.pdf&#39;</span><br><span class="line">removed &#39;.&#x2F;20210414&#x2F;EBV&#x2F;EBV_Ref&#x2F;243_EBV-encoded_EBNA1_regulates_cell_viability_by_modulating_miR34a-NOX2-ROS.pdf&#39;</span><br></pre></td></tr></table></figure>
<p>It took me four hours for update in X1.</p>
<h1 id="Hexo"><a href="#Hexo" class="headerlink" title="Hexo"></a>Hexo</h1><h2 id="TOC"><a href="#TOC" class="headerlink" title="TOC"></a>TOC</h2><ul>
<li><a target="_blank" rel="noopener" href="https://github.com/bubkoo/hexo-toc">bubkoo/hexo-toc: 📖 Insert a markdown TOC before posts be rendered.</a></li>
</ul>
<h2 id="免密上传"><a href="#免密上传" class="headerlink" title="免密上传"></a>免密上传</h2><p>[[Hexo#Password free upload|免密上传]]</p>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br><span class="line">13</span><br><span class="line">14</span><br><span class="line">15</span><br><span class="line">16</span><br><span class="line">17</span><br><span class="line">18</span><br><span class="line">19</span><br><span class="line">20</span><br><span class="line">21</span><br><span class="line">22</span><br><span class="line">23</span><br></pre></td><td class="code"><pre><span class="line">sudo hexo init</span><br><span class="line">hexo s # INFO  Hexo is running at http:&#x2F;&#x2F;localhost:4000 . Press Ctrl+C to stop.</span><br><span class="line">#</span><br><span class="line"># hexo n &quot;My Blog Name&quot;</span><br><span class="line">hexo n &#96;Date&#96; # add a new blog</span><br><span class="line">hexo s # check the new blog</span><br><span class="line">#</span><br><span class="line">hexo clean; hexo g</span><br><span class="line">#</span><br><span class="line">nickname.github.io</span><br><span class="line">https:&#x2F;&#x2F;github.com&#x2F;hotliu&#x2F;hotliu.github.io.git</span><br><span class="line">#</span><br><span class="line">cnpm install --save hexo-deployer-git</span><br><span class="line">#</span><br><span class="line">vi _config.yml</span><br><span class="line">hexo d</span><br><span class="line">#######################################</span><br><span class="line"># Theme selection</span><br><span class="line">#######################################</span><br><span class="line">#</span><br><span class="line">https:&#x2F;&#x2F;github.com&#x2F;litten&#x2F;hexo-theme-yilia</span><br><span class="line">#</span><br><span class="line">git clone https:&#x2F;&#x2F;github.com&#x2F;litten&#x2F;hexo-theme-yilia.git themes&#x2F;yilia # Time, 24&#39;00&quot;</span><br></pre></td></tr></table></figure>
<ul>
<li><a target="_blank" rel="noopener" href="https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh">Connecting to GitHub with SSH - GitHub Docs</a></li>
<li><a target="_blank" rel="noopener" href="https://github.com/hotliu/hotliu.github.io">hotliu/hotliu.github.io: Hexo Blog</a></li>
<li><a target="_blank" rel="noopener" href="https://github.com/settings/keys">SSH and GPG keys</a></li>
<li><a target="_blank" rel="noopener" href="https://docs.github.com/cn/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account">新增 SSH 密钥到 GitHub 帐户 - GitHub Docs</a></li>
<li><a target="_blank" rel="noopener" href="https://docs.github.com/cn/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account">新增 SSH 密钥到 GitHub 帐户 - GitHub Docs</a></li>
</ul>
<h1 id="Proteomics"><a href="#Proteomics" class="headerlink" title="Proteomics"></a>Proteomics</h1><ul>
<li><a target="_blank" rel="noopener" href="https://www.bioinfo-scrounger.com/categories/Bioinformatics-Notes/Proteomics/">Category: Proteomics | KeapNotes blog</a></li>
<li><a target="_blank" rel="noopener" href="http://blog.sciencenet.cn/blog-3402731-1147389.html">科学网—干货篇：蛋白质谱之“搜库” - 崔蕴博的博文</a></li>
<li><a target="_blank" rel="noopener" href="http://www.pibb.ac.cn/html/2015/2/20140078.htm">“自顶向下(top-down)”的蛋白质组学—蛋白质变体的规模化鉴定</a></li>
</ul>
<h1 id="B95-8-Nrf2-Project"><a href="#B95-8-Nrf2-Project" class="headerlink" title="B95-8_Nrf2 Project"></a>B95-8_Nrf2 Project</h1><ul>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/001_EBV_blocks_the_autophagic_flux_n_appropriates_the_autophagic_machinery_to_enhance_viral_replication.pdf">001_EBV_blocks_the_autophagic_flux_n_appropriates_the_autophagic_machinery_to_enhance_viral_replication.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/002_EBV_reduces_autophagy_intracellular_ROS_n_mitochondria_to_impair_monocyte_survival_n_differentiation.pdf">002_EBV_reduces_autophagy_intracellular_ROS_n_mitochondria_to_impair_monocyte_survival_n_differentiation.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/002_Nrf2_Is_a_Direct_PERK_Substrate_n_Effector_of_PERK-Dependent_Cell_Survival.pdf">002_Nrf2_Is_a_Direct_PERK_Substrate_n_Effector_of_PERK-Dependent_Cell_Survival.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/003_Regulation_of_Autophagic_Activation_by_Rta_of_EBV_via_the_Extracellular_Signal-Regulated_Kinase_Pathway.pdf">003_Regulation_of_Autophagic_Activation_by_Rta_of_EBV_via_the_Extracellular_Signal-Regulated_Kinase_Pathway.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/004_p62-mediated_Selective_autophagy_endows_virus-transformed_cells_with_insusceptibility_to_DNA_damage_under_oxidative_stress.pdf">004_p62-mediated_Selective_autophagy_endows_virus-transformed_cells_with_insusceptibility_to_DNA_damage_under_oxidative_stress.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/005_Stimulus_duration_n_response_time_independently_influence_the_kinetics_of_lytic_cycle_reactivation_of_EBV.pdf">005_Stimulus_duration_n_response_time_independently_influence_the_kinetics_of_lytic_cycle_reactivation_of_EBV.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/006_LMP1_n_2A_Induce_the_Expression_of_Nrf2_Through_Akt_Signaling_Pathway_in_EBV-Transformed_B_Cells.pdf">006_LMP1_n_2A_Induce_the_Expression_of_Nrf2_Through_Akt_Signaling_Pathway_in_EBV-Transformed_B_Cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/007_Bortezomib_promotes_KHSV_n_EBV_lytic_cycle_by_activating_JNK_n_autophagy.pdf">007_Bortezomib_promotes_KHSV_n_EBV_lytic_cycle_by_activating_JNK_n_autophagy.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/008_EBV_n_KSHV_Infection_Dysregulates_Autophagy_to_Optimize_Viral_Replication_Prevent_Immune_Recognition_n_Promote_Tumorigenesis.pdf">008_EBV_n_KSHV_Infection_Dysregulates_Autophagy_to_Optimize_Viral_Replication_Prevent_Immune_Recognition_n_Promote_Tumorigenesis.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/009_PKC_theta_n_p38_MAPK_activate_the_EBV_lytic_cycle_through_autophagy_induction.pdf">009_PKC_theta_n_p38_MAPK_activate_the_EBV_lytic_cycle_through_autophagy_induction.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/010_Protein_Kinase_C-Independent_Activation_of_the_EBV_Lytic_Cycle.pdf">010_Protein_Kinase_C-Independent_Activation_of_the_EBV_Lytic_Cycle.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/011_Regulation_of_the_ARE_by_PKC-mediated_phosphorylation_of_Nrf2.pdf">011_Regulation_of_the_ARE_by_PKC-mediated_phosphorylation_of_Nrf2.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/012_Stress-sensing_mechanisms_n_the_physiological_roles_of_the_Keap1%E2%80%93Nrf2_system_during_cellular_stress.pdf">012_Stress-sensing_mechanisms_n_the_physiological_roles_of_the_Keap1–Nrf2_system_during_cellular_stress.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/013_NRF2_in_regulating_cancer_metabolism.pdf">013_NRF2_in_regulating_cancer_metabolism.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/014_NRF2_n_the_Hallmarks_of_Cancer.pdf">014_NRF2_n_the_Hallmarks_of_Cancer.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/015_NRF2_addiction_in_cancer_cells.pdf">015_NRF2_addiction_in_cancer_cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/016_The_Dual_Roles_of_NRF2_in_Cancer.pdf">016_The_Dual_Roles_of_NRF2_in_Cancer.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/017_The_Nrf2_Activator_tBHQ_Differentially_Affects_Early_Events.pdf">017_The_Nrf2_Activator_tBHQ_Differentially_Affects_Early_Events.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/018_Stabilization_of_Nrf2_by_tBHQ.pdf">018_Stabilization_of_Nrf2_by_tBHQ.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/019_The_B95-8_Isolate_of_EBV_Arose_from_an_Isolate_with_a_Standard_Genome.pdf">019_The_B95-8_Isolate_of_EBV_Arose_from_an_Isolate_with_a_Standard_Genome.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/020_Transcription_n_DNA_sequence_of_the_BamHI_L_fragment_of_B95-8_EBV.pdf">020_Transcription_n_DNA_sequence_of_the_BamHI_L_fragment_of_B95-8_EBV.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/021_Growth_of_B95-8_cells_n_expression_of_EBV_lytic_phase_in_serum-free_medium.pdf">021_Growth_of_B95-8_cells_n_expression_of_EBV_lytic_phase_in_serum-free_medium.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/022_Growth_of_B95-8_Cell_line_n_production_of_human_EBV_virus_2010.pdf">022_Growth_of_B95-8_Cell_line_n_production_of_human_EBV_virus_2010.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/023_Growth_of_B95-8_Cell_line_n_production_of_human_EBV_virus_2016.pdf">023_Growth_of_B95-8_Cell_line_n_production_of_human_EBV_virus_2016.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/024_EBV_Turns_50.pdf">024_EBV_Turns_50.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/025_The_transforming_prototype_of_EBV_B95-8_is_also_a_lytic_virus.pdf">025_The_transforming_prototype_of_EBV_B95-8_is_also_a_lytic_virus.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/026_Optimizing_osmotic_pressure_removes_EBV_particles_from_B95-8_host_cells_while_maintaining_normal_activity.pdf">026_Optimizing_osmotic_pressure_removes_EBV_particles_from_B95-8_host_cells_while_maintaining_normal_activity.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/027_Enhancement_of_spontaneous_VCA_n_EA_induction_in_B95-8_cells_n_EA_induction_in_Raji_cells.pdf">027_Enhancement_of_spontaneous_VCA_n_EA_induction_in_B95-8_cells_n_EA_induction_in_Raji_cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/028_Molecular_cytogenetic_characterization_of_the_EBV-producing_cell_line_B95-8_by_chromosome_sorting_n_painting.pdf">028_Molecular_cytogenetic_characterization_of_the_EBV-producing_cell_line_B95-8_by_chromosome_sorting_n_painting.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/029_Release_of_Infectious_EBV_by_Transformed_Marmoset_Leukocytes.pdf">029_Release_of_Infectious_EBV_by_Transformed_Marmoset_Leukocytes.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/030_EBV_Transformation_Cytopathic_Changes_n_Viral_Antigens_in_Squirrel_Monkey_n_Marmoset_Leukocytes.pdf">030_EBV_Transformation_Cytopathic_Changes_n_Viral_Antigens_in_Squirrel_Monkey_n_Marmoset_Leukocytes.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/031_EBV_latent_genes.pdf">031_EBV_latent_genes.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/032_Encyclopedia_of_EBV-Encoded_Lytic_Genes_An_Update.pdf">032_Encyclopedia_of_EBV-Encoded_Lytic_Genes_An_Update.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/033_TPA_Oncogenic_KRAS_confers_chemoresistance_by_upregulating_NRF2.pdf">033_TPA_Oncogenic_KRAS_confers_chemoresistance_by_upregulating_NRF2.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/034_NaB_Activation_of_Lytic_EBV_Infection_by_Radiation_n_NaB_in_Vitro_n_in_Vivo.pdf">034_NaB_Activation_of_Lytic_EBV_Infection_by_Radiation_n_NaB_in_Vitro_n_in_Vivo.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/035_NaB_AMPK_synergizes_with_the_combined_treatment_of_ACA_n_NaB_to_upregulate_phase_II_detoxifying_enzyme_activities.pdf">035_NaB_AMPK_synergizes_with_the_combined_treatment_of_ACA_n_NaB_to_upregulate_phase_II_detoxifying_enzyme_activities.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/036_NaB_Butyrate-producing_probiotics_reduce_nonalcoholic_fatty_liver_disease_progression_in_rats.pdf">036_NaB_Butyrate-producing_probiotics_reduce_nonalcoholic_fatty_liver_disease_progression_in_rats.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/037_NaB_NaB_activates_NRF2_to_ameliorate_diabetic_nephropathy_possibly_via_inhibition_of_HDAC.pdf">037_NaB_NaB_activates_NRF2_to_ameliorate_diabetic_nephropathy_possibly_via_inhibition_of_HDAC.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/038_NaB_NaB_improves_antioxidant_stability_in_sub-acute_ruminal_acidosis_in_dairy_goats.pdf">038_NaB_NaB_improves_antioxidant_stability_in_sub-acute_ruminal_acidosis_in_dairy_goats.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/039_NaB_NaB_protects_against_high-fat_diet-induced_oxidative_stress_in_rat_liver_by_promoting_expression_of_Nrf2.pdf">039_NaB_NaB_protects_against_high-fat_diet-induced_oxidative_stress_in_rat_liver_by_promoting_expression_of_Nrf2.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/040_BZ_induction_of_CEBP%CE%B2_mediates_EBV_lytic_activation_in_Burkitt_lymphoma.pdf">040_BZ_induction_of_CEBPβ_mediates_EBV_lytic_activation_in_Burkitt_lymphoma.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/041_NaB_The_enhancement_of_phase_2_enzyme_activities_by_NaB_in_normal_intestinal_epithelial_cells_is_associated_with_Nrf2_n_p53.pdf">041_NaB_The_enhancement_of_phase_2_enzyme_activities_by_NaB_in_normal_intestinal_epithelial_cells_is_associated_with_Nrf2_n_p53.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/042_NaB_NaB_attenuates_diabetes-induced_aortic_endothelial_dysfunction_via_P300-mediated_transcriptional_activation_of_Nrf2.pdf">042_NaB_NaB_attenuates_diabetes-induced_aortic_endothelial_dysfunction_via_P300-mediated_transcriptional_activation_of_Nrf2.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/043_BZ_High_basal_nuclear_levels_of_Nrf2_in_acute_myeloid_leukemia_reduces_sensitivity_to_proteasome_inhibitors.pdf">043_BZ_High_basal_nuclear_levels_of_Nrf2_in_acute_myeloid_leukemia_reduces_sensitivity_to_proteasome_inhibitors.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/044_BZ_HO-1_up-regulation_A_key_point_in_high-risk_neuroblastoma_resistance.pdf">044_BZ_HO-1_up-regulation_A_key_point_in_high-risk_neuroblastoma_resistance.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/046_BZ_Treatment-induced_oxidative_stress_n_cellular_antioxidant_capacity_determine_response_to_bortezomib.pdf">046_BZ_Treatment-induced_oxidative_stress_n_cellular_antioxidant_capacity_determine_response_to_bortezomib.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/047_BZ_Bortezomib_induces_HO-1_expression_in_multiple_myeloma.pdf">047_BZ_Bortezomib_induces_HO-1_expression_in_multiple_myeloma.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/048_TG_Regulation_of_Nrf2_by_XBP1_in_Retinal_Pigment_Epithelium.pdf">048_TG_Regulation_of_Nrf2_by_XBP1_in_Retinal_Pigment_Epithelium.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/049_TG_Transcriptional_Regulation_of_ATF4_under_Oxidative_Stress.pdf">049_TG_Transcriptional_Regulation_of_ATF4_under_Oxidative_Stress.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/050_KSHV_Induces_Nrf2_Activation_in_Latently_Infected_Endothelial_Cells_through_SQSTM1_Phosphorylation_n_Interaction_with_Keap1.pdf">050_KSHV_Induces_Nrf2_Activation_in_Latently_Infected_Endothelial_Cells_through_SQSTM1_Phosphorylation_n_Interaction_with_Keap1.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/051_KSHV_Cytotoxic_Drugs_Activate_KSHV_Lytic_Cycle_in_Latently_Infected_PEL_Cells_by_Inducing_a_Moderate_ROS_Increase_Controlled_by_HSF1_NRF2_n_p62.pdf">051_KSHV_Cytotoxic_Drugs_Activate_KSHV_Lytic_Cycle_in_Latently_Infected_PEL_Cells_by_Inducing_a_Moderate_ROS_Increase_Controlled_by_HSF1_NRF2_n_p62.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/052_KSHV_Activated_Nrf2_Interacts_with_KSHV_Latency_Protein_LANA-1_n_Host_Protein_KAP1_To_Mediate_Global_Lytic_Gene_Repression.pdf">052_KSHV_Activated_Nrf2_Interacts_with_KSHV_Latency_Protein_LANA-1_n_Host_Protein_KAP1_To_Mediate_Global_Lytic_Gene_Repression.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/053_TG_Promising_therapeutic_effect_of_TG_nanoparticles_on_chronic_kidney_disease_through_the_activation_of_Nrf2_n_FoxO1.pdf">053_TG_Promising_therapeutic_effect_of_TG_nanoparticles_on_chronic_kidney_disease_through_the_activation_of_Nrf2_n_FoxO1.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/054_Mutant_p53_Stabilized_by_Its_Interplay_with_HSP90_Activates_a_Positive_Feed-Back_Loop_Between_NRF2_n_p62.pdf">054_Mutant_p53_Stabilized_by_Its_Interplay_with_HSP90_Activates_a_Positive_Feed-Back_Loop_Between_NRF2_n_p62.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/055_Constitutive_autophagy_contributes_to_resistance_to_TP53-mediated_apoptosis_in_EBV-positive_latency_III.pdf">055_Constitutive_autophagy_contributes_to_resistance_to_TP53-mediated_apoptosis_in_EBV-positive_latency_III.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/056_Monitoring_Keap1%E2%80%93Nrf2_interactions_in_single_live_cells.pdf">056_Monitoring_Keap1–Nrf2_interactions_in_single_live_cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/057_Role_of_p62_SQSTM1_beyond_autophagy.pdf">057_Role_of_p62_SQSTM1_beyond_autophagy.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/058_Contribution_of_NAD_P_H_quinone_oxidoreductase_1_to_protection_against_carcinogenesis_n_regulation_of_its_gene_by_Nrf2.pdf">058_Contribution_of_NAD_P_H_quinone_oxidoreductase_1_to_protection_against_carcinogenesis_n_regulation_of_its_gene_by_Nrf2.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/059_When_autophagy_meets_cancer_through_p62_SQSTM1.pdf">059_When_autophagy_meets_cancer_through_p62_SQSTM1.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/060_The_Role_of_the_Selective_Adaptor_p62_n_Ubiquitin-LikeProteins_in_Autophagy.pdf">060_The_Role_of_the_Selective_Adaptor_p62_n_Ubiquitin-LikeProteins_in_Autophagy.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/061_Small_Maf_proteins_History_structure_n_function.pdf">061_Small_Maf_proteins_History_structure_n_function.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/062_Crosstalk_between_Nrf2_n_Notch_signaling.pdf">062_Crosstalk_between_Nrf2_n_Notch_signaling.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/063_Regulation_of_cell_proliferation_n_migration_by_p62_through_stabilization_of_Twist1.pdf">063_Regulation_of_cell_proliferation_n_migration_by_p62_through_stabilization_of_Twist1.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/064_Nrf2_negatively_regulates_STING_indicating_a_link_between_antiviral_sensing_n_metabolic_reprogramming.pdf">064_Nrf2_negatively_regulates_STING_indicating_a_link_between_antiviral_sensing_n_metabolic_reprogramming.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/065_Mutant_p53_n_Cellular_Stress_Pathways_A_Criminal_Alliance_That_Promotes_Cancer_Progression.pdf">065_Mutant_p53_n_Cellular_Stress_Pathways_A_Criminal_Alliance_That_Promotes_Cancer_Progression.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/066_Overview_of_Nrf2_as_Therapeutic_Target_in_Epilepsy.pdf">066_Overview_of_Nrf2_as_Therapeutic_Target_in_Epilepsy.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/067_P62_An_emerging_oncotarget_for_osteolytic_metastas.pdf">067_P62_An_emerging_oncotarget_for_osteolytic_metastas.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/068_Genetic_Variations_n_mRNA_Expression_of_NRF2_in_Parkinsons_Disease.pdf">068_Genetic_Variations_n_mRNA_Expression_of_NRF2_in_Parkinsons_Disease.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/069_Phorbol_Ester-dependent_Activation_of_Peroxiredoxin_I_Gene_Expression_TPA.pdf">069_Phorbol_Ester-dependent_Activation_of_Peroxiredoxin_I_Gene_Expression_TPA.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/070_Cellular_Differentiation_Regulator_BLIMP1_Induces_EBV_Lytic_Reactivation_in_Epithelial_n_B_Cells.pdf">070_Cellular_Differentiation_Regulator_BLIMP1_Induces_EBV_Lytic_Reactivation_in_Epithelial_n_B_Cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/071_Inactivation_of_NF-kB_by_EBV_BZLF-1-Encoded_ZEBRA_Protein_in_Human_T_Cells.pdf">071_Inactivation_of_NF-kB_by_EBV_BZLF-1-Encoded_ZEBRA_Protein_in_Human_T_Cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/072_EBNA-1_induces_expression_of_the_cellular_microRNA_hsa-miR-127_n_impairing_B-cell_differentiation.pdf">072_EBNA-1_induces_expression_of_the_cellular_microRNA_hsa-miR-127_n_impairing_B-cell_differentiation.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/073_Detection_of_IgG_n_IgA_Ab_to_EBV_membrane_Ag_in_sera_from_patients_with_NPC_n_from_normal_individuals.pdf">073_Detection_of_IgG_n_IgA_Ab_to_EBV_membrane_Ag_in_sera_from_patients_with_NPC_n_from_normal_individuals.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/074_Expression_of_EBV_antibody_EA-IgA_Rta-IgG_n_VCA-IgA_n_SA_in_serum.pdf">074_Expression_of_EBV_antibody_EA-IgA_Rta-IgG_n_VCA-IgA_n_SA_in_serum.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/075_Differentiation-Dependent_KLF4_Expression_Promotes_Lytic_EBV_Infection_in_Epithelial_Cells.pdf">075_Differentiation-Dependent_KLF4_Expression_Promotes_Lytic_EBV_Infection_in_Epithelial_Cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/076_EBV_infection_induces_aberrant_TLR_activation_pathway_n_fibroblast-myofibroblast_conversion_in_scleroderma.pdf">076_EBV_infection_induces_aberrant_TLR_activation_pathway_n_fibroblast-myofibroblast_conversion_in_scleroderma.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/077_quercitrin_Inhibition_of_AP-1_n_MAPK_signaling_n_activation_of_Nrf2_ARE_pathway_by_quercitrin.pdf">077_quercitrin_Inhibition_of_AP-1_n_MAPK_signaling_n_activation_of_Nrf2_ARE_pathway_by_quercitrin.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/078_Quercetin_Interrupts_the_Positive_Feedback_Loop_Between_STAT3_n_IL-6_Promotes_Autophagy_n_Reduces_ROS.pdf">078_Quercetin_Interrupts_the_Positive_Feedback_Loop_Between_STAT3_n_IL-6_Promotes_Autophagy_n_Reduces_ROS.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/079_Combination_of_betulinic_acid_n_chidamide_synergistically_inhibits_EBV_replication_through_over-generation_of_ROS.pdf">079_Combination_of_betulinic_acid_n_chidamide_synergistically_inhibits_EBV_replication_through_over-generation_of_ROS.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/080_Chloroquine_Combination_Therapy_with_a_PI3KmTOR_Dual_Inhibitor_n_Chloroquine_Enhances_Synergistic_Apoptotic_Cell_Death_in_EBV-Infected.pdf">080_Chloroquine_Combination_Therapy_with_a_PI3KmTOR_Dual_Inhibitor_n_Chloroquine_Enhances_Synergistic_Apoptotic_Cell_Death_in_EBV-Infected.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/081_Chloroquine_triggers_EBV_replication_through_phosphorylation_of_KAP1_TRIM28_in_Burkitt_lymphoma_cells.pdf">081_Chloroquine_triggers_EBV_replication_through_phosphorylation_of_KAP1_TRIM28_in_Burkitt_lymphoma_cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/082_Deletion_of_EBV_BFLF2_leads_to_impaired_viral_DNA_packaging_n_primary_egress_as_well_as_to_the_production_of_defective_viral_particles.pdf">082_Deletion_of_EBV_BFLF2_leads_to_impaired_viral_DNA_packaging_n_primary_egress_as_well_as_to_the_production_of_defective_viral_particles.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/083_EBV_LF2_Protein_Regulates_Viral_Replication_by_Altering_Rta_Subcellular_Localization.pdf">083_EBV_LF2_Protein_Regulates_Viral_Replication_by_Altering_Rta_Subcellular_Localization.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/084_RNAs_in_EBV_control_early_steps_of_infection.pdf">084_RNAs_in_EBV_control_early_steps_of_infection.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/085_The_EBV_EBNA-2_Gene_in_Oral_Hairy_Leukoplakia.pdf">085_The_EBV_EBNA-2_Gene_in_Oral_Hairy_Leukoplakia.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/086_TPA_Inhibition_of_Activator_Protein-1_NF-%CE%BAB_n_MAPKs_n_Induction_of_Phase_2_Detoxifying_Enzyme_Activity.pdf">086_TPA_Inhibition_of_Activator_Protein-1_NF-κB_n_MAPKs_n_Induction_of_Phase_2_Detoxifying_Enzyme_Activity.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/087_Atypical_PKC_mediates_activation_of_Nrf2_in_response_to_oxidative_stress.pdf">087_Atypical_PKC_mediates_activation_of_Nrf2_in_response_to_oxidative_stress.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/088_Latent_n_lytic_cycle_promoters_of_EBV.pdf">088_Latent_n_lytic_cycle_promoters_of_EBV.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/089_Switches_of_Oncogenic_Human_Gammaherpesviruses1.pdf">089_Switches_of_Oncogenic_Human_Gammaherpesviruses1.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/090_TPA_induction_of_EBV_early_antigens_in_Raji_cells_is_blocked_by_selective_PKC_inhibitors.pdf">090_TPA_induction_of_EBV_early_antigens_in_Raji_cells_is_blocked_by_selective_PKC_inhibitors.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/091_Bortezomib-induced_EBV_n_KSHV_lytic_gene_expression_oncolytic_strategies.pdf">091_Bortezomib-induced_EBV_n_KSHV_lytic_gene_expression_oncolytic_strategies.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/092_bZIP_proteins_of_human_gammaherpesviruses.pdf">092_bZIP_proteins_of_human_gammaherpesviruses.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/093_Regulation_of_cellular_n_viral_protein_expression_by_the_EBV_transcriptional_regulator_Zta_Implications_for_therapy_of_EBV.pdf">093_Regulation_of_cellular_n_viral_protein_expression_by_the_EBV_transcriptional_regulator_Zta_Implications_for_therapy_of_EBV.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/094_Structural_basis_of_lytic_cycle_activation_by_the_EBV_ZEBRA_protein.pdf">094_Structural_basis_of_lytic_cycle_activation_by_the_EBV_ZEBRA_protein.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/095_TPA_induces_EBV_reactivation_via_NF-kappaB_n_AP-1_as_regulated_by_PKC_n_mitogen-activated_protein_kinase.pdf">095_TPA_induces_EBV_reactivation_via_NF-kappaB_n_AP-1_as_regulated_by_PKC_n_mitogen-activated_protein_kinase.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/096_Phosphorylation_of_Nrf2_at_Ser-40_by_PKC_regulates_ARE-mediated_transcription.pdf">096_Phosphorylation_of_Nrf2_at_Ser-40_by_PKC_regulates_ARE-mediated_transcription.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/097_Induction_of_EBV_lytic_cycle_in_vitro_causes_oxidative_stress_in_lymphoblastoid_B_cell_lines.pdf">097_Induction_of_EBV_lytic_cycle_in_vitro_causes_oxidative_stress_in_lymphoblastoid_B_cell_lines.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/098_Induction_of_EBV_Lytic_Cycle_in_Vitro_Causes_Lipid_Peroxidation_Protein_Oxidation_n_DNA_Damage_in_Lymphoblastoid_B_Cell_Lines.pdf">098_Induction_of_EBV_Lytic_Cycle_in_Vitro_Causes_Lipid_Peroxidation_Protein_Oxidation_n_DNA_Damage_in_Lymphoblastoid_B_Cell_Lines.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/099_Reactive_oxygen_species_production_n_antioxidant_enzyme_expression_after_EBV_lytic_cycle_induction_in_Raji_cell_line.pdf">099_Reactive_oxygen_species_production_n_antioxidant_enzyme_expression_after_EBV_lytic_cycle_induction_in_Raji_cell_line.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/102_EBV_induces_an_oxidative_stress_during_the_early_stages_of_infection_in_B_lymphocytes_epithelial_n_lymphoblastoid_cell_lines.pdf">102_EBV_induces_an_oxidative_stress_during_the_early_stages_of_infection_in_B_lymphocytes_epithelial_n_lymphoblastoid_cell_lines.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/103_The_role_of_oxidative_stress_in_EBV_lytic_reactivation_radioresistance_n_the_potential_preventive_n_therapeutic_implications.pdf">103_The_role_of_oxidative_stress_in_EBV_lytic_reactivation_radioresistance_n_the_potential_preventive_n_therapeutic_implications.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/104_Transcription_of_the_EBV_lytic_cycle_activator_BZLF-1_during_oxidative_stress_induction.pdf">104_Transcription_of_the_EBV_lytic_cycle_activator_BZLF-1_during_oxidative_stress_induction.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/105_Silver_nanoparticles_selectively_induce_human_oncogenic_%CE%B3-herpesvirus-related_cancer_cell_death.pdf">105_Silver_nanoparticles_selectively_induce_human_oncogenic_γ-herpesvirus-related_cancer_cell_death.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/106_Reactive_oxygen_species_drive_herpes_simplex_virus_HSV_-1-induced_proinflammatory_cytokine_production_by_murine_microglia.pdf">106_Reactive_oxygen_species_drive_herpes_simplex_virus_HSV_-1-induced_proinflammatory_cytokine_production_by_murine_microglia.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/107_Reactive_oxygen_signaling_n_MAPK_activation_distinguish_EBV-positive_versus_EBV-negative_Burkitt's_lymphoma.pdf">107_Reactive_oxygen_signaling_n_MAPK_activation_distinguish_EBV-positive_versus_EBV-negative_Burkitt’s_lymphoma.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/108_Combination_of_betulinic_acid_n_chidamide_synergistically_inhibits_EBV_replication_through_over-generation_of_ROS.pdf">108_Combination_of_betulinic_acid_n_chidamide_synergistically_inhibits_EBV_replication_through_over-generation_of_ROS.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/109_Bortezomib_n_SAHA_synergistically_induce_ROS-driven_caspase-dependent_apoptosis_of_NPC_n_block_replication_of_EBV.pdf">109_Bortezomib_n_SAHA_synergistically_induce_ROS-driven_caspase-dependent_apoptosis_of_NPC_n_block_replication_of_EBV.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/110_Calcium_mobilization_is_responsible_for_Thapsigargin_induced_EBV_lytic_reactivation_in_in_vitro_immortalized_lymphoblstoid_cell_lines.pdf">110_Calcium_mobilization_is_responsible_for_Thapsigargin_induced_EBV_lytic_reactivation_in_in_vitro_immortalized_lymphoblstoid_cell_lines.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/111_EBV_post-transplant_lymphoproliferative_disease_n_virus-specific_therapy-pharmacological_re-activation_of_viral_target_genes_with_arginine_butyrate.pdf">111_EBV_post-transplant_lymphoproliferative_disease_n_virus-specific_therapy-pharmacological_re-activation_of_viral_target_genes_with_arginine_butyrate.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/112_EBV_reactivation_n_therapeutic_inhibitors.pdf">112_EBV_reactivation_n_therapeutic_inhibitors.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/113_EBV-targeted_therapy_in_nasopharyngeal_carcinoma.pdf">113_EBV-targeted_therapy_in_nasopharyngeal_carcinoma.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/114_Contribution_of_the_KSHV_n_EBV_lytic_cycles_to_tumourigenesis.pdf">114_Contribution_of_the_KSHV_n_EBV_lytic_cycles_to_tumourigenesis.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/115_EBV_lytic_reactivation_regulation_n_its_pathogenic_role_in_carcinogenesis.pdf">115_EBV_lytic_reactivation_regulation_n_its_pathogenic_role_in_carcinogenesis.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/116_Detection_n_characterization_of_EBV_in_clinical_specimens.pdf">116_Detection_n_characterization_of_EBV_in_clinical_specimens.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/117_EBV_Lytic_Cycle_Reactivation.pdf">117_EBV_Lytic_Cycle_Reactivation.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/118_Lytic_cycle_switches_of_oncogenic_human_gammaherpesviruses.pdf">118_Lytic_cycle_switches_of_oncogenic_human_gammaherpesviruses.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/119_Regulation_n_dysregulation_of_EBV_latency.pdf">119_Regulation_n_dysregulation_of_EBV_latency.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/120_EBV-driven_B-cell_lymphoproliferative_disorders.pdf">120_EBV-driven_B-cell_lymphoproliferative_disorders.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/121_The_EBV_in_T_Cell_n_NK_Cell_Lymphomas-Time_for_a_Reassessment.pdf">121_The_EBV_in_T_Cell_n_NK_Cell_Lymphomas-Time_for_a_Reassessment.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/122_Endoplasmic_reticulum_stress_causes_EBV_lytic_replication.pdf">122_Endoplasmic_reticulum_stress_causes_EBV_lytic_replication.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/123_p62_SQSTM1-Jack_of_All_Trades_in_Health_n_Cancer.pdf">123_p62_SQSTM1-Jack_of_All_Trades_in_Health_n_Cancer.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/124_Activation_of_SQSTM1-Keap1-Nrf2_Pathway_in_Cancer.pdf">124_Activation_of_SQSTM1-Keap1-Nrf2_Pathway_in_Cancer.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/125_Canonical_n_non-canonical_mechanisms_of_Nrf2_activation.pdf">125_Canonical_n_non-canonical_mechanisms_of_Nrf2_activation.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/126_Upregulation_of_STAT3_marks_Burkitt_lymphoma_cells_refractory_to_EBV_lytic_cycle_induction_by_HDAC_inhibitors.pdf">126_Upregulation_of_STAT3_marks_Burkitt_lymphoma_cells_refractory_to_EBV_lytic_cycle_induction_by_HDAC_inhibitors.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/127_Valproic_Acid_Antagonizes_the_Capacity_of_Other_HDAC_Inhibitors_To_Activate_the_EBV_Lytic_Cycle.pdf">127_Valproic_Acid_Antagonizes_the_Capacity_of_Other_HDAC_Inhibitors_To_Activate_the_EBV_Lytic_Cycle.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/128_Nitric_Oxide_Down-Regulates_EBV_Reactivation_in_Epithelial_Cell_Lines.pdf">128_Nitric_Oxide_Down-Regulates_EBV_Reactivation_in_Epithelial_Cell_Lines.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/129_EBV_lytic_transactivator_Zta_enhances_chemotactic_activity_through_induction_of_interleukin-8_in_NPC_cells.pdf">129_EBV_lytic_transactivator_Zta_enhances_chemotactic_activity_through_induction_of_interleukin-8_in_NPC_cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/130_Changes_in_the_nasopharyngeal_carcinoma_nuclear_proteome_induced_by_the_EBNA1_reveal_potential_roles_in_metastasis_n_oxidative_stress.pdf">130_Changes_in_the_nasopharyngeal_carcinoma_nuclear_proteome_induced_by_the_EBNA1_reveal_potential_roles_in_metastasis_n_oxidative_stress.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/131_0810619106SI.pdf">131_0810619106SI.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/131_The_EBV_nuclear_antigen-1_promotes_genomic_instability_via_induction_of_ROS.pdf">131_The_EBV_nuclear_antigen-1_promotes_genomic_instability_via_induction_of_ROS.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/132_The_EBV_nuclear_antigen-1_promotes_telomere_dysfunction_via_induction_of_oxidative_stress.pdf">132_The_EBV_nuclear_antigen-1_promotes_telomere_dysfunction_via_induction_of_oxidative_stress.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/133_EBV-encoded_EBNA1_regulates_cell_viability_by_modulating_miR34a-NOX2-ROS_signaling_in_gastric_cancer_cells.pdf">133_EBV-encoded_EBNA1_regulates_cell_viability_by_modulating_miR34a-NOX2-ROS_signaling_in_gastric_cancer_cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/134_Crosstalk_between_the_mTOR_n_Nrf2-ARE_signaling_pathways_as_a_target_in_the_improvement_of_long-term_potentiation.pdf">134_Crosstalk_between_the_mTOR_n_Nrf2-ARE_signaling_pathways_as_a_target_in_the_improvement_of_long-term_potentiation.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/135_Niclosamide_inhibits_lytic_replication_of_EBV_by_disrupting_mTOR_activation2.pdf">135_Niclosamide_inhibits_lytic_replication_of_EBV_by_disrupting_mTOR_activation2.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/135_Niclosamide_inhibits_lytic_replication_of_EBV_by_disrupting_mTOR_activation.pdf">135_Niclosamide_inhibits_lytic_replication_of_EBV_by_disrupting_mTOR_activation.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/136_Inhibition_of_mTORC1_inhibits_lytic_replication_of_EBV_in_a_cell-type_specific_manner.pdf">136_Inhibition_of_mTORC1_inhibits_lytic_replication_of_EBV_in_a_cell-type_specific_manner.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/137_Nrf2_Transcription_Factor_Can_Directly_Regulate_mTOR.pdf">137_Nrf2_Transcription_Factor_Can_Directly_Regulate_mTOR.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/138_Chloroquine_inhibits_lytic_replication_of_KSHV_by_disrupting_mTOR_n_p38-MAPK_activation.pdf">138_Chloroquine_inhibits_lytic_replication_of_KSHV_by_disrupting_mTOR_n_p38-MAPK_activation.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/139_mTOR_inhibitors_induce_cell-cycle_arrest_n_inhibit_tumor_growth_in_EBV-associated_T_n_natural_killer_cell_lymphoma_cells.pdf">139_mTOR_inhibitors_induce_cell-cycle_arrest_n_inhibit_tumor_growth_in_EBV-associated_T_n_natural_killer_cell_lymphoma_cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/140_mTOR-Mediated_Antioxidant_Activation_in_Solid_Tumor_Radioresistance.pdf">140_mTOR-Mediated_Antioxidant_Activation_in_Solid_Tumor_Radioresistance.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/141_Transcriptional_regulation_of_the_stress_response_by_mTOR.pdf">141_Transcriptional_regulation_of_the_stress_response_by_mTOR.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/143_Molecular_mechanisms_of_mTOR_regulation_by_stress.pdf">143_Molecular_mechanisms_of_mTOR_regulation_by_stress.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/144_Mazes_of_Nrf2_Regulation.pdf">144_Mazes_of_Nrf2_Regulation.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/145_Global_downstream_pathway_analysis_reveals_a_dependence_of_oncogenic_Nrf2_mutation_on_the_mTOR_growth_signaling_pathway.pdf">145_Global_downstream_pathway_analysis_reveals_a_dependence_of_oncogenic_Nrf2_mutation_on_the_mTOR_growth_signaling_pathway.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/146_Inhibition_of_PI3KAKTmTOR_axis_disrupts_oxidative_stress-mediated_survival_of_melanoma_cells.pdf">146_Inhibition_of_PI3KAKTmTOR_axis_disrupts_oxidative_stress-mediated_survival_of_melanoma_cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/147_Insights_for_Oxidative_Stress_n_mTOR_Signaling_in_Myocardial_Ischemia-Reperfusion_Injury_under_Diabetes.pdf">147_Insights_for_Oxidative_Stress_n_mTOR_Signaling_in_Myocardial_Ischemia-Reperfusion_Injury_under_Diabetes.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/148_ROS_signaling_under_metabolic_stress_cross-talk_between_AMPK_n_AKT_pathway.pdf">148_ROS_signaling_under_metabolic_stress_cross-talk_between_AMPK_n_AKT_pathway.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/149_Expression_of_EBV_latent_antigens_mTOR_n_tumor_suppression_genes_in_EBV-positive_smooth_muscle_tumors.pdf">149_Expression_of_EBV_latent_antigens_mTOR_n_tumor_suppression_genes_in_EBV-positive_smooth_muscle_tumors.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/150_EBV_encoded_LMP1_regulates_mTOR_signaling_pathway_genes_which_predict_poor_prognosis_of_nasopharyngeal_carcinoma.pdf">150_EBV_encoded_LMP1_regulates_mTOR_signaling_pathway_genes_which_predict_poor_prognosis_of_nasopharyngeal_carcinoma.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/151_Latent_membrane_protein_1_of_EBV_activates_phosphatidylinositol_3-kinase_to_induce_production_of_IL-10.pdf">151_Latent_membrane_protein_1_of_EBV_activates_phosphatidylinositol_3-kinase_to_induce_production_of_IL-10.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/152_Modulation_of_the_cell_growth_regulator_mTOR_by_EBV_LMP2A.pdf">152_Modulation_of_the_cell_growth_regulator_mTOR_by_EBV_LMP2A.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/153_Oncogene-induced_Nrf2_transcription_promotes_ROS_detoxification_n_tumorigenesis_NIHMS313089-supplement-1.pdf">153_Oncogene-induced_Nrf2_transcription_promotes_ROS_detoxification_n_tumorigenesis_NIHMS313089-supplement-1.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/153_Oncogene-induced_Nrf2_transcription_promotes_ROS_detoxification_n_tumorigenesis.pdf">153_Oncogene-induced_Nrf2_transcription_promotes_ROS_detoxification_n_tumorigenesis.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/154_MNNG_induces_n_cooperates_with_TB_to_enhance_EBV_reactivation_n_genome_instability_in_nasopharyngeal_carcinoma_cells.pdf">154_MNNG_induces_n_cooperates_with_TB_to_enhance_EBV_reactivation_n_genome_instability_in_nasopharyngeal_carcinoma_cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/155_The_Keap1-Nrf2-antioxidant_response_element_pathway-a_review_of_its_regulation_by_melatonin_n_the_proteasome.pdf">155_The_Keap1-Nrf2-antioxidant_response_element_pathway-a_review_of_its_regulation_by_melatonin_n_the_proteasome.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/156_mTOR_signaling_pathway_n_mTOR_inhibitors_in_cancer_progress_n_challenges.pdf">156_mTOR_signaling_pathway_n_mTOR_inhibitors_in_cancer_progress_n_challenges.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/157_HDACi_chidamide_induces_growth_inhibition_n_apoptosis_in_NK-T_lymphoma_cells_through_ATM-Chk2-p53-p21_signalling_pathway.pdf">157_HDACi_chidamide_induces_growth_inhibition_n_apoptosis_in_NK-T_lymphoma_cells_through_ATM-Chk2-p53-p21_signalling_pathway.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/158_Therapies_based_on_targeting_EBV_lytic_replication_for_EBV-associated_malignancies.pdf">158_Therapies_based_on_targeting_EBV_lytic_replication_for_EBV-associated_malignancies.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/159_Activation_of_p62-Keap1-Nrf2_Pathway_in_Cancer.pdf">159_Activation_of_p62-Keap1-Nrf2_Pathway_in_Cancer.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/160_p62_as_a_therapeutic_target_for_tumor.pdf">160_p62_as_a_therapeutic_target_for_tumor.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/161_p62_is_a_key_regulator_of_nutrient_sensing_in_the_mTORC1_pathway.pdf">161_p62_is_a_key_regulator_of_nutrient_sensing_in_the_mTORC1_pathway.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/162_p62-related_pathways_as_therapeutic_targets_in_hepatocellular_carcinoma.pdf">162_p62-related_pathways_as_therapeutic_targets_in_hepatocellular_carcinoma.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/163_Autophagy_n_its_role_in_gastric_cancer.pdf">163_Autophagy_n_its_role_in_gastric_cancer.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/164_Autophagy_pathway_Cellular_n_molecular_mechanisms.pdf">164_Autophagy_pathway_Cellular_n_molecular_mechanisms.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/165_Role_of_p62_in_liver_physiology_n_pathogenesis.pdf">165_Role_of_p62_in_liver_physiology_n_pathogenesis.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/166_Targeting_autophagy_in_cancer.pdf">166_Targeting_autophagy_in_cancer.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/167_Autophagy_n_disease-unanswered_questions.pdf">167_Autophagy_n_disease-unanswered_questions.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/168_Autophagy_in_cancer-moving_from_understanding_mechanism_to_improving_therapy_responses_in_patients.pdf">168_Autophagy_in_cancer-moving_from_understanding_mechanism_to_improving_therapy_responses_in_patients.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/169_Guidelines_for_the_use_n_interpretation_of_assays_for_monitoring_autophagy_3rd_edition.pdf">169_Guidelines_for_the_use_n_interpretation_of_assays_for_monitoring_autophagy_3rd_edition.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/170_Targeting_autophagy_in_cancer_2018.pdf">170_Targeting_autophagy_in_cancer_2018.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/171_Cisplatin_induces_autophagy_to_enhance_hepatitis_B_virus_replication_via_activation_of_ROS_JNK_n_inhibition_of_the_AktmTOR_pathway.pdf">171_Cisplatin_induces_autophagy_to_enhance_hepatitis_B_virus_replication_via_activation_of_ROS_JNK_n_inhibition_of_the_AktmTOR_pathway.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/172_Autophagy_during_viral_infection%E2%80%94_a_double-edged_sword.pdf">172_Autophagy_during_viral_infection—_a_double-edged_sword.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/173_KSHV_infection_inhibits_macrophage_formation_n_survival_by_counteracting_M-CSF-induced_increase_of_ROS_c-JNK_phosphorylation_n_autophagy.pdf">173_KSHV_infection_inhibits_macrophage_formation_n_survival_by_counteracting_M-CSF-induced_increase_of_ROS_c-JNK_phosphorylation_n_autophagy.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/174_FGF9-induced_changes_in_cellular_redox_status_n_HO-1_upregulation_are_FGFR-dependent_n_proceed_through_both_ERK_n_AKT_to_induce_CREB_n_Nrf2_activation.pdf">174_FGF9-induced_changes_in_cellular_redox_status_n_HO-1_upregulation_are_FGFR-dependent_n_proceed_through_both_ERK_n_AKT_to_induce_CREB_n_Nrf2_activation.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/175_Essential_roles_of_the_PI3_kinase-Akt_pathway_in_regulating_Nrf2-dependent_antioxidant_functions_in_the_RPE.pdf">175_Essential_roles_of_the_PI3_kinase-Akt_pathway_in_regulating_Nrf2-dependent_antioxidant_functions_in_the_RPE.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/176_Maltol_Slows_d-Galactose-Induced_Brain_Aging_Process_by_Damping_the_Nrf2HO-1-Mediated_Oxidative_Stress_in_Mice.pdf">176_Maltol_Slows_d-Galactose-Induced_Brain_Aging_Process_by_Damping_the_Nrf2HO-1-Mediated_Oxidative_Stress_in_Mice.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/177_EBV_Associated_Lymphomas_n_Epithelia_Cancers_in_Humans.pdf">177_EBV_Associated_Lymphomas_n_Epithelia_Cancers_in_Humans.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/178_Cytolytic_virus_activation_therapy_for_EBV-driven_tumors.pdf">178_Cytolytic_virus_activation_therapy_for_EBV-driven_tumors.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/179_Activation_of_lytic_cycle_of_EBV_by_suberoylanilide_hydroxamic_acid_leads_to_apoptosis_n_tumor_suppression_of_NPC.pdf">179_Activation_of_lytic_cycle_of_EBV_by_suberoylanilide_hydroxamic_acid_leads_to_apoptosis_n_tumor_suppression_of_NPC.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/180_Recurrent_chemical_reactivations_of_EBV_promotes_genome_instability_n_enhances_tumor_progression_of_nasopharyngeal_carcinoma_cells.pdf">180_Recurrent_chemical_reactivations_of_EBV_promotes_genome_instability_n_enhances_tumor_progression_of_nasopharyngeal_carcinoma_cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/181_Identification_of_Novel_Small_Organic_Compounds_with_Diverse_Structures_for_the_Induction_of_EBV_Lytic_Cycle.pdf">181_Identification_of_Novel_Small_Organic_Compounds_with_Diverse_Structures_for_the_Induction_of_EBV_Lytic_Cycle.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/182_Intracellular_Iron_Chelation_by_a_Novel_Compound_C7_Reactivates_EBV_Lytic_Cycle_via_the_ERK-Autophagy_Axis.pdf">182_Intracellular_Iron_Chelation_by_a_Novel_Compound_C7_Reactivates_EBV_Lytic_Cycle_via_the_ERK-Autophagy_Axis.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/183_Autophagy-Dependent_Reactivation_of_EBV_Lytic_Cycle_n_Combinatorial_Effects_of_Autophagy-Dependent_n_Independent_Lytic_Inducers.pdf">183_Autophagy-Dependent_Reactivation_of_EBV_Lytic_Cycle_n_Combinatorial_Effects_of_Autophagy-Dependent_n_Independent_Lytic_Inducers.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/185_Targeting_EBV_in_Nasopharyngeal_Carcinoma.pdf">185_Targeting_EBV_in_Nasopharyngeal_Carcinoma.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/187_Therapeutic_Strategies_against_EBV-Associated_Cancers_Using_Proteasome_Inhibitors.pdf">187_Therapeutic_Strategies_against_EBV-Associated_Cancers_Using_Proteasome_Inhibitors.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/188_TIPE1-mediated_autophagy_suppression_promotes_NPC_cell_proliferation_via_the_AMPK-mTOR_signalling_pathway.pdf">188_TIPE1-mediated_autophagy_suppression_promotes_NPC_cell_proliferation_via_the_AMPK-mTOR_signalling_pathway.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/189_Contribution_of_EBV_Reactivation_to_the_Carcinogenicity_of_Nasopharyngeal_Cancer_Cell.pdf">189_Contribution_of_EBV_Reactivation_to_the_Carcinogenicity_of_Nasopharyngeal_Cancer_Cell.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/190_Development_of_a_novel_inducer_for_EBV_lytic_therapy.pdf">190_Development_of_a_novel_inducer_for_EBV_lytic_therapy.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/191_The_PERK-Nrf2_Pathway_Mediates_Endoplasmic_Reticulum_Stres.pdf">191_The_PERK-Nrf2_Pathway_Mediates_Endoplasmic_Reticulum_Stres.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/192_The_NRF2-KEAP1_Pathway_Modulates_NPC_Cell_Radiosensitivity_via_ROS_Elimination.pdf">192_The_NRF2-KEAP1_Pathway_Modulates_NPC_Cell_Radiosensitivity_via_ROS_Elimination.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/193_BDH2_triggers_ROS-induced_cell_death_n_autophagy_by_promoting_Nrf2_ubiquitination_in_gastric_cancer.pdf">193_BDH2_triggers_ROS-induced_cell_death_n_autophagy_by_promoting_Nrf2_ubiquitination_in_gastric_cancer.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/194_Berbamine_induced_AMPK_activation_regulates_mTOR-SREBP-1c_axis_n_Nrf2-ARE_pathway_in_steatotic_HepG2_cells.pdf">194_Berbamine_induced_AMPK_activation_regulates_mTOR-SREBP-1c_axis_n_Nrf2-ARE_pathway_in_steatotic_HepG2_cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/195_Effects_of_a_novel_biflavonoid_of_Lonicera_japonica_flower_buds_on_modulating_apoptosis_in_hepatoma_cells.pdf">195_Effects_of_a_novel_biflavonoid_of_Lonicera_japonica_flower_buds_on_modulating_apoptosis_in_hepatoma_cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/196_Lanthanum_chloride_induces_autophagy_in_rat_hippocampus_through_ROS-mediated_JNK_n_AKTmTOR_signaling_pathways.pdf">196_Lanthanum_chloride_induces_autophagy_in_rat_hippocampus_through_ROS-mediated_JNK_n_AKTmTOR_signaling_pathways.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/197_Neferine_prevents_NF-%CE%BAB_translocation_n_protects_muscle_cells_from_oxidative_stress_n_apoptosis_induced_by_hypoxia.pdf">197_Neferine_prevents_NF-κB_translocation_n_protects_muscle_cells_from_oxidative_stress_n_apoptosis_induced_by_hypoxia.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/198_Neferine_prevents_autophagy_induced_by_hypoxia_through_activation_of_Akt-mTOR_pathway_n_Nrf2_in_muscle_cells.pdf">198_Neferine_prevents_autophagy_induced_by_hypoxia_through_activation_of_Akt-mTOR_pathway_n_Nrf2_in_muscle_cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/199_Scoparone_improves_hepatic_inflammation_n_autophagy_in_mice_with_nonalcoholic_steatohepatitis_macrophages.pdf">199_Scoparone_improves_hepatic_inflammation_n_autophagy_in_mice_with_nonalcoholic_steatohepatitis_macrophages.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/200_Synthetic_oleanane_triterpenoids-multifunctional_drugs_with_applications_for_prevention_n_treatment_of_chronic_disease.pdf">200_Synthetic_oleanane_triterpenoids-multifunctional_drugs_with_applications_for_prevention_n_treatment_of_chronic_disease.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/201_ROS-mediated_inactivation_of_the_PI3K-AKT_pathway_is_involved_in_the_antigastric_cancer_effects_of_chaetocin.pdf">201_ROS-mediated_inactivation_of_the_PI3K-AKT_pathway_is_involved_in_the_antigastric_cancer_effects_of_chaetocin.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/202_Baicalein_enhanced_cisplatin_sensitivity_of_gastric_cancer_by_inducing_apoptosis_n_autophagy.pdf">202_Baicalein_enhanced_cisplatin_sensitivity_of_gastric_cancer_by_inducing_apoptosis_n_autophagy.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/204_mTORC2_facilitates_endothelial_cell_senescence_by_suppressing_Nrf2_expression_via_Akt-GSK-3%CE%B2-C-EBP%CE%B1_pathway.pdf">204_mTORC2_facilitates_endothelial_cell_senescence_by_suppressing_Nrf2_expression_via_Akt-GSK-3β-C-EBPα_pathway.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/205_TORC2-SGK-1_signaling_integrates_external_signals_to_regulate_autophagic_turnover_of_mitochondria_via_mtROS.pdf">205_TORC2-SGK-1_signaling_integrates_external_signals_to_regulate_autophagic_turnover_of_mitochondria_via_mtROS.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/206_Activation_of_KGFR-Akt-mTOR-Nrf2_signaling_protects_human_retinal_pigment_epithelium_cells_from_Ultra-violet.pdf">206_Activation_of_KGFR-Akt-mTOR-Nrf2_signaling_protects_human_retinal_pigment_epithelium_cells_from_Ultra-violet.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/207_PF-4708671_activates_Nrf2_by_promoting_p62-dependent_autophagic_degradation_of_Keap1.pdf">207_PF-4708671_activates_Nrf2_by_promoting_p62-dependent_autophagic_degradation_of_Keap1.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/208_The_antioxidant_function_of_sestrins_is_mediated_by_promotion_of_autophagic_degradation_of_Keap1_n_Nrf2_activation.pdf">208_The_antioxidant_function_of_sestrins_is_mediated_by_promotion_of_autophagic_degradation_of_Keap1_n_Nrf2_activation.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/209_Regulation_of_mTORC2_Signaling.pdf">209_Regulation_of_mTORC2_Signaling.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/210_Research_progress_of_mTOR_inhibitors.pdf">210_Research_progress_of_mTOR_inhibitors.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/211_Mitochondrial_Perturbations_Couple_mTORC2_to_Autophagy_in_C._elegans.pdf">211_Mitochondrial_Perturbations_Couple_mTORC2_to_Autophagy_in_C._elegans.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/212_mTORC2_controls_hydrophobic_motif_phosphorylation_n_activation_of_SGK1.pdf">212_mTORC2_controls_hydrophobic_motif_phosphorylation_n_activation_of_SGK1.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/213_Cellular_differentiation_regulator_BLIMP1_induces_EBV_lytic_reactivation_by_activating_transcription_R_n_Z_promoters.pdf">213_Cellular_differentiation_regulator_BLIMP1_induces_EBV_lytic_reactivation_by_activating_transcription_R_n_Z_promoters.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/214_Differentiation-Dependent_LMP1_Expression_Is_Required_for_Efficient_Lytic_EBV_Reactivation_in_Epithelial_Cells.pdf">214_Differentiation-Dependent_LMP1_Expression_Is_Required_for_Efficient_Lytic_EBV_Reactivation_in_Epithelial_Cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/215_Inhibition_of_EBV_Replication_in_HPV-Immortalized_Keratinocytes.pdf">215_Inhibition_of_EBV_Replication_in_HPV-Immortalized_Keratinocytes.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/216_The_p65_Isoform_of_Nrf1_Is_a_Dominant_Negative_Inhibitor_of_ARE-mediated_Transcription.pdf">216_The_p65_Isoform_of_Nrf1_Is_a_Dominant_Negative_Inhibitor_of_ARE-mediated_Transcription.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/217_Activation_of_lytic_EBV_infection_by_radiation_n_NaB_for_treating_EBV-positive_malignancies.pdf">217_Activation_of_lytic_EBV_infection_by_radiation_n_NaB_for_treating_EBV-positive_malignancies.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/218_Azacitidine_induces_demethylation_of_the_EBV_genome_in_tumors.pdf">218_Azacitidine_induces_demethylation_of_the_EBV_genome_in_tumors.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/219_Bortezomib-induced_enzyme-targeted_radiation_therapy_in_herpesvirus-associated_tumors.pdf">219_Bortezomib-induced_enzyme-targeted_radiation_therapy_in_herpesvirus-associated_tumors.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/221_Lytic_Induction_Therapy_against_EBV-Associated_Malignancies-Past_Present_n_Future.pdf">221_Lytic_Induction_Therapy_against_EBV-Associated_Malignancies-Past_Present_n_Future.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/223_Viral-Targeted_Strategies_Against_EBV-Associated_Lymphoproliferative_Diseases.pdf">223_Viral-Targeted_Strategies_Against_EBV-Associated_Lymphoproliferative_Diseases.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/224_Curcuminoids_as_EBV_Lytic_Activators_for_Adjuvant_Treatment_in_EBV-Positive_Carcinomas.pdf">224_Curcuminoids_as_EBV_Lytic_Activators_for_Adjuvant_Treatment_in_EBV-Positive_Carcinomas.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/228_Targeting_EBV_LMP1-mediated_high_oxidative_stress_suppresses_EBV_lytic_reactivation_n_sensitizes_tumors_to_radiation_therapy.pdf">228_Targeting_EBV_LMP1-mediated_high_oxidative_stress_suppresses_EBV_lytic_reactivation_n_sensitizes_tumors_to_radiation_therapy.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/228_Targeting_EBV_LMP1-mediated_high_oxidative_stress_suppresses_EBV_lytic_reactivation_n_sensitizes_tumors_to_radiation_therapy_supp.pdf">228_Targeting_EBV_LMP1-mediated_high_oxidative_stress_suppresses_EBV_lytic_reactivation_n_sensitizes_tumors_to_radiation_therapy_supp.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/229_Alterations_in_cellular_expression_in_EBV_infected_epithelial_cell_lines_n_tumors.pdf">229_Alterations_in_cellular_expression_in_EBV_infected_epithelial_cell_lines_n_tumors.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/230_CCAATEnhancer_Binding_Protein_%CE%B1_Interacts_with_ZTA_n_Mediates_ZTA-Induced_p21CIP-1_Accumulation.pdf">230_CCAATEnhancer_Binding_Protein_α_Interacts_with_ZTA_n_Mediates_ZTA-Induced_p21CIP-1_Accumulation.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/231_Two_21-kilodalton_components_of_the_EBV_capsid_antigen_complex_n_their_relationship_to_ZEBRA-associated_protein_p21_ZAP21.pdf">231_Two_21-kilodalton_components_of_the_EBV_capsid_antigen_complex_n_their_relationship_to_ZEBRA-associated_protein_p21_ZAP21.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/232_Direct_Interaction_between_Nrf2_n_p21_Upregulates_the_Nrf2-Mediated_Antioxidant_Response.pdf">232_Direct_Interaction_between_Nrf2_n_p21_Upregulates_the_Nrf2-Mediated_Antioxidant_Response.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/233_Functions_of_the_EBV_EBNA1_protein_in_viral_reactivation_n_lytic_infection.pdf">233_Functions_of_the_EBV_EBNA1_protein_in_viral_reactivation_n_lytic_infection.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/234_Proteomic_profiling_of_EBNA1-host_protein_interactions_in_latent_n_lytic_EBV_infections.pdf">234_Proteomic_profiling_of_EBNA1-host_protein_interactions_in_latent_n_lytic_EBV_infections.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/235_EBNA1.pdf">235_EBNA1.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/236_EBV_EBNA1_Protein_Regulates_Viral_Latency_through_Effects_on_let-7_MicroRNA_n_Dicer.pdf">236_EBV_EBNA1_Protein_Regulates_Viral_Latency_through_Effects_on_let-7_MicroRNA_n_Dicer.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/237_EBV-encoded_EBNA1_n_ZEBRA.pdf">237_EBV-encoded_EBNA1_n_ZEBRA.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/238_LMP1_Is_a_Novel_Determinant_of_EBV_Genome_Persistence_n_Reactivation.pdf">238_LMP1_Is_a_Novel_Determinant_of_EBV_Genome_Persistence_n_Reactivation.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/239_Proteasomal_inhibition_triggers_viral_oncoprotein_degradation_via_autophagy-lysosomal_pathway.pdf">239_Proteasomal_inhibition_triggers_viral_oncoprotein_degradation_via_autophagy-lysosomal_pathway.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/240_Carcinoma-risk_variant_of_EBNA1_deregulates_EBV_episomal_latency.pdf">240_Carcinoma-risk_variant_of_EBNA1_deregulates_EBV_episomal_latency.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/241_Curcumin_Inhibits_Proliferation_of_EBV-Associated_Human_NPC_by_Inhibiting_EBV_EBNA1_Expression.pdf">241_Curcumin_Inhibits_Proliferation_of_EBV-Associated_Human_NPC_by_Inhibiting_EBV_EBNA1_Expression.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/242_In_Vivo_Dynamics_of_EBNA1-oriP_Interaction_during_Latent_n_Lytic_Replication_of_EBV.pdf">242_In_Vivo_Dynamics_of_EBNA1-oriP_Interaction_during_Latent_n_Lytic_Replication_of_EBV.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/244_STUB1_is_targeted_by_the_SUMO-interacting_motif_of_EBNA1_to_maintain_EBV_latency.pdf">244_STUB1_is_targeted_by_the_SUMO-interacting_motif_of_EBNA1_to_maintain_EBV_latency.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/245_EBV_EBNA3A_Is_Critical_for_Maintaining_Lymphoblastoid_Cell_Line_Growth.pdf">245_EBV_EBNA3A_Is_Critical_for_Maintaining_Lymphoblastoid_Cell_Line_Growth.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/246_EBV_BNA3C_is_required_for_cell_cycle_progression_n_growth_maintenance_of_lymphoblastoid_cells.pdf">246_EBV_BNA3C_is_required_for_cell_cycle_progression_n_growth_maintenance_of_lymphoblastoid_cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/247_Mechanisms_of_B-Cell_Oncogenesis_Induced_by_EBV.pdf">247_Mechanisms_of_B-Cell_Oncogenesis_Induced_by_EBV.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/248_EBNA3B-deficient_EBV_promotes_B_cell_lymphomagenesis_in_humanized_mice_n_is_found_in_human_tumors.pdf">248_EBNA3B-deficient_EBV_promotes_B_cell_lymphomagenesis_in_humanized_mice_n_is_found_in_human_tumors.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/249_The_EBNA3_Family-Two_Oncoproteins_n_a_Tumour_Suppressor_that_Are_Central_to_the_Biology_of_EBV_in_B_Cells.pdf">249_The_EBNA3_Family-Two_Oncoproteins_n_a_Tumour_Suppressor_that_Are_Central_to_the_Biology_of_EBV_in_B_Cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/250_LMP2_blocks_reactivation_of_EBV_from_latency_following_surface_immunoglobulin_crosslinking.pdf">250_LMP2_blocks_reactivation_of_EBV_from_latency_following_surface_immunoglobulin_crosslinking.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/251_LPM2_of_EBV_regulates_reactivation_from_latency_through_dominant_negative_effects_on_protein-tyrosine_kinases.pdf">251_LPM2_of_EBV_regulates_reactivation_from_latency_through_dominant_negative_effects_on_protein-tyrosine_kinases.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/252_LMP2B_regulates_susceptibility_to_induction_of_lytic_EBV_infection.pdf">252_LMP2B_regulates_susceptibility_to_induction_of_lytic_EBV_infection.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/253_Analysis_of_EBV_gene_expression_upon_phorbol_ester_n_hydroxyurea_treatment_by_real-time_quantitative_PCR.pdf">253_Analysis_of_EBV_gene_expression_upon_phorbol_ester_n_hydroxyurea_treatment_by_real-time_quantitative_PCR.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/254_Effect_of_the_diterpene_ester_TPA_on_EBV_antigen-_n_DNA_synthesis_in_producer_n_nonproducer_cell_lines.pdf">254_Effect_of_the_diterpene_ester_TPA_on_EBV_antigen-_n_DNA_synthesis_in_producer_n_nonproducer_cell_lines.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/255_Histone_Hyperacetylation_Occurs_on_Promoters_of_Lytic_Cycle_Regulatory_Genes_in_EBV-Infected_Cell_Lines.pdf">255_Histone_Hyperacetylation_Occurs_on_Promoters_of_Lytic_Cycle_Regulatory_Genes_in_EBV-Infected_Cell_Lines.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/256_Suberoylanilide_hydroxamic_acid_induces_viral_lytic_cycle_in_EBV-positive_epithelial_malignancies_n_mediates_enhanced_cell_death.pdf">256_Suberoylanilide_hydroxamic_acid_induces_viral_lytic_cycle_in_EBV-positive_epithelial_malignancies_n_mediates_enhanced_cell_death.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/257_3H-1_2-dithiole-3-thione_protects_retinal_pigment_epithelium_cells_against_Ultra-violet_radiation_via_activation_of_Akt-mTORC1-dependent_Nrf2-HO-1_signaling.pdf">257_3H-1_2-dithiole-3-thione_protects_retinal_pigment_epithelium_cells_against_Ultra-violet_radiation_via_activation_of_Akt-mTORC1-dependent_Nrf2-HO-1_signaling.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/258_Endoplasmic_Reticulum_Stress-Mediated_Apoptosis_of_EBV-Transformed_B_Cells_by_Cross-Linking_of_CD70_Is_Dependent_upon_Generation_of_Reactive_Oxygen_Species_n_Activation_of_p38_MAPK_n_JNK_Pathway.pdf">258_Endoplasmic_Reticulum_Stress-Mediated_Apoptosis_of_EBV-Transformed_B_Cells_by_Cross-Linking_of_CD70_Is_Dependent_upon_Generation_of_Reactive_Oxygen_Species_n_Activation_of_p38_MAPK_n_JNK_Pathway.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/259_A_Noncanonical_Basic_Motif_of_EBV_ZEBRA_Protein_Facilitates_Recognition_of_Methylated_DNA_HighAffinity_DNA_Binding_n_Lytic_Activation.pdf">259_A_Noncanonical_Basic_Motif_of_EBV_ZEBRA_Protein_Facilitates_Recognition_of_Methylated_DNA_HighAffinity_DNA_Binding_n_Lytic_Activation.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/259_Copy_number_of_EBV_latent_genome_correlates_with_the_oncogenicity_by_the_activation_level_of_LMP1_n_NF-%CE%BAB.pdf">259_Copy_number_of_EBV_latent_genome_correlates_with_the_oncogenicity_by_the_activation_level_of_LMP1_n_NF-κB.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/260_Bortezomib_induces_heme_oxygenase-1_expression_in_multiple_myeloma.pdf">260_Bortezomib_induces_heme_oxygenase-1_expression_in_multiple_myeloma.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/260_Early_EBV_Genomic_Diversity_n_Convergence_toward_the_B95.8_Genome_in_Primary_Infection.pdf">260_Early_EBV_Genomic_Diversity_n_Convergence_toward_the_B95.8_Genome_in_Primary_Infection.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/261_Combined_inhibition_of_autophagy_n_Nrf2_signaling_augments_bortezomib-induced_apoptosis_by_increasing_ROS_production_n_ER_stress_in_pancreatic_cancer_cells.pdf">261_Combined_inhibition_of_autophagy_n_Nrf2_signaling_augments_bortezomib-induced_apoptosis_by_increasing_ROS_production_n_ER_stress_in_pancreatic_cancer_cells.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/261_High_EBV_Load_n_Genomic_Diversity_Are_Associated_with_Generation_of_gp350-Specific_Neutralizing_Antibodies_following_Acute_Infectious_Mononucleosis.pdf">261_High_EBV_Load_n_Genomic_Diversity_Are_Associated_with_Generation_of_gp350-Specific_Neutralizing_Antibodies_following_Acute_Infectious_Mononucleosis.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/262_Degradation_of_p53_Can_Be_Targeted_by_HPV_E6_Sequences_Distinct_from_Those_Required_for_p53_Binding_n_Trans_Activation.pdf">262_Degradation_of_p53_Can_Be_Targeted_by_HPV_E6_Sequences_Distinct_from_Those_Required_for_p53_Binding_n_Trans_Activation.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/262_HLA_alleles_modulate_EBV_viral_load_in_multiple_sclerosis.pdf">262_HLA_alleles_modulate_EBV_viral_load_in_multiple_sclerosis.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/263_EBV_PCR_Sum.pdf">263_EBV_PCR_Sum.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/263_Rapid_determination_of_EBV_latent_or_lytic_infection_in_single_human_cells_using_in_situ_hybridization.pdf">263_Rapid_determination_of_EBV_latent_or_lytic_infection_in_single_human_cells_using_in_situ_hybridization.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/264_Enzyme_immunoassays_for_the_diagnosis_of_EBV_infection.pdf">264_Enzyme_immunoassays_for_the_diagnosis_of_EBV_infection.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/265_Oncogenic_Properties_of_the_EBV_ZEBRA_Protein.pdf">265_Oncogenic_Properties_of_the_EBV_ZEBRA_Protein.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/266_Sodium_butyrate_activates_NRF2_to_ameliorate_diabetic_nephropathy_possibly_via_inhibition_of_HDAC-J_Endocrinology.pdf">266_Sodium_butyrate_activates_NRF2_to_ameliorate_diabetic_nephropathy_possibly_via_inhibition_of_HDAC-J_Endocrinology.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/267_Sodium_butyrate_inhibits_high_cholesterol-induced_neuronal_amyloidogenesis_by_modulating_NRF2_stabilization-mediated_ROS_levels-involvement_of_NOX2_n_SOD1.pdf">267_Sodium_butyrate_inhibits_high_cholesterol-induced_neuronal_amyloidogenesis_by_modulating_NRF2_stabilization-mediated_ROS_levels-involvement_of_NOX2_n_SOD1.pdf</a></li>
</ul>
<hr>
<h2 id="Viral-Load"><a href="#Viral-Load" class="headerlink" title="Viral Load"></a>Viral Load</h2><ul>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/EBV_DNA_load_RT-PCR/101_Chaetocin_reactivates_the_lytic_replication_of_EBV_from_latency_via_reactive_oxygen_species.pdf">101_Chaetocin_reactivates_the_lytic_replication_of_EBV_from_latency_via_reactive_oxygen_species.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/EBV_DNA_load_RT-PCR/184_Pharmacologic_Activation_of_Lytic_EBV_Gene_Expression_without_Virion_Production.pdf">184_Pharmacologic_Activation_of_Lytic_EBV_Gene_Expression_without_Virion_Production.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/EBV_DNA_load_RT-PCR/186_Inhibition_of_class_I_histone_deacetylases_by_romidepsin_potently_induces_EBV_lytic_cycle_ganciclovir.pdf">186_Inhibition_of_class_I_histone_deacetylases_by_romidepsin_potently_induces_EBV_lytic_cycle_ganciclovir.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/EBV_DNA_load_RT-PCR/203_The_mTOR_Inhibitor_Manassantin_B_Reveals-A_Crucial_Role_of_mTORC2_Signaling_in_EBV_Reactivation.pdf">203_The_mTOR_Inhibitor_Manassantin_B_Reveals-A_Crucial_Role_of_mTORC2_Signaling_in_EBV_Reactivation.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/EBV_DNA_load_RT-PCR/220_Emodin_Inhibits_EBV_Reactivation_and_Represses_NPC_Tumorigenesis.pdf">220_Emodin_Inhibits_EBV_Reactivation_and_Represses_NPC_Tumorigenesis.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/EBV_DNA_load_RT-PCR/222_Rapamycin_enhances_lytic_replication_of_EBV_in_gastric_carcinoma_cells_by_increasing_the_transcriptional_of_lytic_promoters.pdf">222_Rapamycin_enhances_lytic_replication_of_EBV_in_gastric_carcinoma_cells_by_increasing_the_transcriptional_of_lytic_promoters.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/EBV_DNA_load_RT-PCR/225_Genipin_as_a_novel_chemical_activator_of_EBV_lytic_cycle.pdf">225_Genipin_as_a_novel_chemical_activator_of_EBV_lytic_cycle.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/EBV_DNA_load_RT-PCR/226_Cordycepin_is_a_novel_chemical_suppressor_of_EBV.pdf">226_Cordycepin_is_a_novel_chemical_suppressor_of_EBV.pdf</a></li>
<li><a href="/media/ht/ht_5T/Work/Projects/NFE/B95-8/Ref/EBV_DNA_load_RT-PCR/227_Quercetin-induced_apoptosis_prevents_EBV_infection.pdf">227_Quercetin-induced_apoptosis_prevents_EBV_infection.pdf</a></li>
</ul>

      

      
    </div>
    <div class="article-info article-info-index">
      
      
	<div class="article-tag tagcloud">
		<i class="icon-price-tags icon"></i>
		<ul class="article-tag-list">
			 
        		<li class="article-tag-list-item">
        			<a href="javascript:void(0)" class="js-tag article-tag-list-link color5">Hexo</a>
        		</li>
      		 
        		<li class="article-tag-list-item">
        			<a href="javascript:void(0)" class="js-tag article-tag-list-link color1">Proteomics</a>
        		</li>
      		 
        		<li class="article-tag-list-item">
        			<a href="javascript:void(0)" class="js-tag article-tag-list-link color3">GanttProject</a>
        		</li>
      		
		</ul>
	</div>

      
	<div class="article-category tagcloud">
		<i class="icon-book icon"></i>
		<ul class="article-tag-list">
			 
        		<li class="article-tag-list-item">
        			<a href="/categories/Log//" class="article-tag-list-link color4">Log</a>
        		</li>
      		 
        		<li class="article-tag-list-item">
        			<a href="/categories/Log/Computing//" class="article-tag-list-link color5">Computing</a>
        		</li>
      		
		</ul>
	</div>


      
        <p class="article-more-link">
          <a class="article-more-a" href="/2021/04/18/20210418/">Expand all items >></a>
        </p>
      

      
      <div class="clearfix"></div>
    </div>
  </div>
</article>

<aside class="wrap-side-operation">
    <div class="mod-side-operation">
        
        <div class="jump-container" id="js-jump-container" style="display:none;">
            <a href="javascript:void(0)" class="mod-side-operation__jump-to-top">
                <i class="icon-font icon-back"></i>
            </a>
            <div id="js-jump-plan-container" class="jump-plan-container" style="top: -11px;">
                <i class="icon-font icon-plane jump-plane"></i>
            </div>
        </div>
        
        
    </div>
</aside>




  
    <article id="post-20210415" class="article article-type-post  article-index" itemscope itemprop="blogPost">
  <div class="article-inner">
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2021/04/15/20210415/">20210415</a>
    </h1>
  

        
        <a href="/2021/04/15/20210415/" class="archive-article-date">
  	<time datetime="2021-04-14T16:00:00.000Z" itemprop="datePublished"><i class="icon-calendar icon"></i>20210415</time>
</a>
        
      </header>
    
    <div class="article-entry" itemprop="articleBody">
    

      
<div id="toc" class="toc-article">
  <strong class="toc-title">Contents</strong>
  <ol class="toc"><li class="toc-item toc-level-1"><a class="toc-link" href="#Working-directories"><span class="toc-number">1.</span> <span class="toc-text">Working directories</span></a></li></ol>
</div>

        <h1 id="Working-directories"><a href="#Working-directories" class="headerlink" title="Working directories"></a>Working directories</h1><pre><code>i=&#39;/media/ht/ht_5T/Work/References/Tech/Protein_Tech/PPI_Prediction&#39;
cd $&#123;i&#125;
#</code></pre>

      

      
    </div>
    <div class="article-info article-info-index">
      
      
      
	<div class="article-category tagcloud">
		<i class="icon-book icon"></i>
		<ul class="article-tag-list">
			 
        		<li class="article-tag-list-item">
        			<a href="/categories/Log//" class="article-tag-list-link color4">Log</a>
        		</li>
      		
		</ul>
	</div>


      
        <p class="article-more-link">
          <a class="article-more-a" href="/2021/04/15/20210415/">Expand all items >></a>
        </p>
      

      
      <div class="clearfix"></div>
    </div>
  </div>
</article>

<aside class="wrap-side-operation">
    <div class="mod-side-operation">
        
        <div class="jump-container" id="js-jump-container" style="display:none;">
            <a href="javascript:void(0)" class="mod-side-operation__jump-to-top">
                <i class="icon-font icon-back"></i>
            </a>
            <div id="js-jump-plan-container" class="jump-plan-container" style="top: -11px;">
                <i class="icon-font icon-plane jump-plane"></i>
            </div>
        </div>
        
        
    </div>
</aside>




  
    <article id="post-20210414" class="article article-type-post  article-index" itemscope itemprop="blogPost">
  <div class="article-inner">
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2021/04/14/20210414/">20210414</a>
    </h1>
  

        
        <a href="/2021/04/14/20210414/" class="archive-article-date">
  	<time datetime="2021-04-13T16:00:00.000Z" itemprop="datePublished"><i class="icon-calendar icon"></i>20210414</time>
</a>
        
      </header>
    
    <div class="article-entry" itemprop="articleBody">
    

      
<div id="toc" class="toc-article">
  <strong class="toc-title">Contents</strong>
  <ol class="toc"><li class="toc-item toc-level-1"><a class="toc-link" href="#Working-directories"><span class="toc-number">1.</span> <span class="toc-text">Working directories</span></a></li></ol>
</div>

        <h1 id="Working-directories"><a href="#Working-directories" class="headerlink" title="Working directories"></a>Working directories</h1><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br></pre></td><td class="code"><pre><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;NFE&#x2F;B95-8&#x2F;Ref&#x2F;20210308&#39;</span><br><span class="line">cd $&#123;i&#125;</span><br><span class="line">#</span><br><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;NFE&#x2F;B95-8&#x2F;Ref&#x2F;20210414&#39;</span><br><span class="line">cd $&#123;i&#125;</span><br><span class="line">#</span><br><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;肿瘤室工作&#x2F;0_学生指导&#x2F;王娇&#x2F;Thesis&#39;; cd $&#123;i&#125;</span><br><span class="line">#</span><br><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;肿瘤室工作&#x2F;0_学生指导&#x2F;王娇&#x2F;Thesis&#x2F;Thesis_WJ_20210414_1630.docx&#39;</span><br><span class="line">setsid wps $&#123;i&#125;</span><br></pre></td></tr></table></figure>
<ul>
<li><a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2785934/">The Role of HPV E6 and E7 Oncoproteins in HPV-associated Cervical Carcinogenesis</a></li>
<li><a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6985034/">Human Papillomavirus E6 and E7: The Cervical Cancer Hallmarks and Targets for Therapy</a></li>
<li><a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5082385/">Functional Roles of E6 and E7 Oncoproteins in HPV-Induced Malignancies at Diverse Anatomical Sites</a></li>
</ul>

      

      
    </div>
    <div class="article-info article-info-index">
      
      
      
	<div class="article-category tagcloud">
		<i class="icon-book icon"></i>
		<ul class="article-tag-list">
			 
        		<li class="article-tag-list-item">
        			<a href="/categories/Log//" class="article-tag-list-link color4">Log</a>
        		</li>
      		
		</ul>
	</div>


      
        <p class="article-more-link">
          <a class="article-more-a" href="/2021/04/14/20210414/">Expand all items >></a>
        </p>
      

      
      <div class="clearfix"></div>
    </div>
  </div>
</article>

<aside class="wrap-side-operation">
    <div class="mod-side-operation">
        
        <div class="jump-container" id="js-jump-container" style="display:none;">
            <a href="javascript:void(0)" class="mod-side-operation__jump-to-top">
                <i class="icon-font icon-back"></i>
            </a>
            <div id="js-jump-plan-container" class="jump-plan-container" style="top: -11px;">
                <i class="icon-font icon-plane jump-plane"></i>
            </div>
        </div>
        
        
    </div>
</aside>




  
    <article id="post-20210408" class="article article-type-post  article-index" itemscope itemprop="blogPost">
  <div class="article-inner">
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2021/04/08/20210408/">20210408</a>
    </h1>
  

        
        <a href="/2021/04/08/20210408/" class="archive-article-date">
  	<time datetime="2021-04-07T16:00:00.000Z" itemprop="datePublished"><i class="icon-calendar icon"></i>20210408</time>
</a>
        
      </header>
    
    <div class="article-entry" itemprop="articleBody">
    

      
<div id="toc" class="toc-article">
  <strong class="toc-title">Contents</strong>
  <ol class="toc"><li class="toc-item toc-level-1"><a class="toc-link" href="#Working-directories"><span class="toc-number">1.</span> <span class="toc-text">Working directories</span></a></li><li class="toc-item toc-level-1"><a class="toc-link" href="#Error-calculation-in-R-CRAN"><span class="toc-number">2.</span> <span class="toc-text">Error calculation in R&#x2F;CRAN</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#MAE-MSE-RMSE-Coefficient-of-Determination-Adjusted-R-Squared-%E2%80%94-Which-Metric-is-Better"><span class="toc-number">2.1.</span> <span class="toc-text">MAE, MSE, RMSE, Coefficient of Determination, Adjusted R Squared — Which Metric is Better?</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#Differences-among-these-evaluation-metrics"><span class="toc-number">2.1.1.</span> <span class="toc-text">Differences among these evaluation metrics</span></a></li></ol></li></ol></li></ol>
</div>

        <h1 id="Working-directories"><a href="#Working-directories" class="headerlink" title="Working directories"></a>Working directories</h1><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br></pre></td><td class="code"><pre><span class="line">cd &#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;HIV&#x2F;Figures&#x2F;凉山 # 凉山地图下载</span><br><span class="line">#</span><br><span class="line">cd &#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;NFE&#x2F;Summary</span><br><span class="line">cd &#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;NFE&#x2F;B95-8&#x2F;Summary</span><br><span class="line">#</span><br><span class="line">cd &#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;References&#x2F;Tech&#x2F;ATCC&#x2F;Guidebooks</span><br><span class="line">wget https:&#x2F;&#x2F;www.atcc.org&#x2F;~&#x2F;media&#x2F;PDFs&#x2F;Culture%20Guides&#x2F;Virology_Guide.ashx</span><br><span class="line"></span><br></pre></td></tr></table></figure>
<p>记录的原则是</p>
<ol>
<li>可以是手写记录，但一定是在iPad Pro上。</li>
<li>其他的尽可能是文本记录。</li>
</ol>
<h1 id="Error-calculation-in-R-CRAN"><a href="#Error-calculation-in-R-CRAN" class="headerlink" title="Error calculation in R/CRAN"></a>Error calculation in R/CRAN</h1><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br></pre></td><td class="code"><pre><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Funding&#x2F;2017_重大专项&#x2F;Management&#x2F;Administration&#x2F;05_00_HFMD&#x2F;Publication&#x2F;Raw_Data&#39;; cd $&#123;i&#125;</span><br><span class="line">#</span><br><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Funding&#x2F;2017_重大专项&#x2F;Management&#x2F;Administration&#x2F;05_00_HFMD&#x2F;Publication&#x2F;README.md&#39;; subl $&#123;i&#125;</span><br><span class="line">#</span><br><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Funding&#x2F;2017_重大专项&#x2F;Management&#x2F;Administration&#x2F;05_00_HFMD&#x2F;Publication&#x2F;README.md&#39;; atom $&#123;i&#125;</span><br><span class="line">#</span><br><span class="line"># R</span><br><span class="line">i &lt;- &#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Funding&#x2F;2017_重大专项&#x2F;Management&#x2F;Administration&#x2F;05_00_HFMD&#x2F;MAE_Diff_20210408.xlsx&#39;</span><br></pre></td></tr></table></figure>

<ul>
<li><a target="_blank" rel="noopener" href="https://medium.com/analytics-vidhya/mae-mse-rmse-coefficient-of-determination-adjusted-r-squared-which-metric-is-better-cd0326a5697e">MAE, MSE, RMSE, Coefficient of Determination, Adjusted R Squared — Which Metric is Better? | by Akshita Chugh | Analytics Vidhya | Medium</a></li>
<li><a target="_blank" rel="noopener" href="https://www.rdocumentation.org/packages/DescTools/versions/0.99.40/topics/Measures%20of%20Accuracy">Measures of Accuracy function - RDocumentation</a></li>
<li><a target="_blank" rel="noopener" href="https://www.datatechnotes.com/2019/02/regression-model-accuracy-mae-mse-rmse.html">DataTechNotes: Regression Model Accuracy (MAE, MSE, RMSE, R-squared) Check in R</a></li>
<li><a target="_blank" rel="noopener" href="https://www.r-bloggers.com/2013/07/calculate-rmse-and-mae-in-r-and-sas/#:~:text=RMSE%20(root%20mean%20squared%20error,extra%20weight%20to%20large%20errors.">Calculate RMSE and MAE in R and SAS | R-bloggers</a></li>
</ul>
<h2 id="MAE-MSE-RMSE-Coefficient-of-Determination-Adjusted-R-Squared-—-Which-Metric-is-Better"><a href="#MAE-MSE-RMSE-Coefficient-of-Determination-Adjusted-R-Squared-—-Which-Metric-is-Better" class="headerlink" title="MAE, MSE, RMSE, Coefficient of Determination, Adjusted R Squared — Which Metric is Better?"></a>MAE, MSE, RMSE, Coefficient of Determination, Adjusted R Squared — Which Metric is Better?</h2><p><a target="_blank" rel="noopener" href="https://akshita-chugh024.medium.com/?source=post_page-----cd0326a5697e--------------------------------"><img src="https://miro.medium.com/fit/c/96/96/0*OmO3vZuubJAt95Gh.jpg" alt="Akshita Chugh"></a></p>
<p><a target="_blank" rel="noopener" href="https://akshita-chugh024.medium.com/?source=post_page-----cd0326a5697e--------------------------------">Akshita Chugh</a></p>
<p>Follow</p>
<p><a target="_blank" rel="noopener" href="https://medium.com/analytics-vidhya/mae-mse-rmse-coefficient-of-determination-adjusted-r-squared-which-metric-is-better-cd0326a5697e?source=post_page-----cd0326a5697e--------------------------------">Dec 8, 2020</a> · 4 min read</p>
<p><a target="_blank" rel="noopener" href="https://medium.com/m/signin?actionUrl=https://medium.com/_/bookmark/p/cd0326a5697e&operation=register&redirect=https://medium.com/analytics-vidhya/mae-mse-rmse-coefficient-of-determination-adjusted-r-squared-which-metric-is-better-cd0326a5697e&source=post_actions_header--------------------------bookmark_preview-----------"></a></p>
<p>The objective of Linear Regression is to find a line that minimizes the prediction error of all the data points.</p>
<p><img src="https://miro.medium.com/max/0/0*s6YGm5hBneEVNc7U.jpg"></p>
<p>The essential step in any machine learning model is to evaluate the accuracy of the model. The Mean Squared Error, Mean absolute error, Root Mean Squared Error, and R-Squared or Coefficient of determination metrics are used to evaluate the performance of the model in regression analysis.</p>
<ul>
<li>  The Mean absolute error represents the average of the absolute difference between the actual and predicted values in the dataset. It measures the average of the residuals in the dataset.</li>
</ul>
<p><img src="https://miro.medium.com/max/60/1*_mx6zD6Qw_1sYEvWszK5Nw.png?q=20"></p>
<p><img src="https://miro.medium.com/max/289/1*_mx6zD6Qw_1sYEvWszK5Nw.png"></p>
<p><img src="https://miro.medium.com/max/60/1*3DI_3QdQgyBWixpRxTgCHQ.png?q=20"></p>
<p><img src="https://miro.medium.com/max/374/1*3DI_3QdQgyBWixpRxTgCHQ.png"></p>
<ul>
<li>  Mean Squared Error represents the average of the squared difference between the original and predicted values in the data set. It measures the variance of the residuals.</li>
</ul>
<p><img src="https://miro.medium.com/max/60/1*1TV6T6hNP-iwJrtvfsvCRQ.png?q=20"></p>
<p><img src="https://miro.medium.com/max/266/1*1TV6T6hNP-iwJrtvfsvCRQ.png"></p>
<ul>
<li>  Root Mean Squared Error is the square root of Mean Squared error. It measures the standard deviation of residuals.</li>
</ul>
<p><img src="https://miro.medium.com/max/60/1*usaMSyi6jUT3f2bOMyiYdA.png?q=20"></p>
<p><img src="https://miro.medium.com/max/425/1*usaMSyi6jUT3f2bOMyiYdA.png"></p>
<ul>
<li>  The coefficient of determination or R-squared represents the proportion of the variance in the dependent variable which is explained by the linear regression model. It is a scale-free score i.e. irrespective of the values being small or large, the value of R square will be less than one.</li>
</ul>
<p><img src="https://miro.medium.com/max/60/1*X0_3mtDXwuhd3dl88xR4yA.png?q=20"></p>
<p><img src="https://miro.medium.com/max/274/1*X0_3mtDXwuhd3dl88xR4yA.png"></p>
<ul>
<li>  Adjusted R squared is a modified version of R square, and it is adjusted for the number of independent variables in the model, and it will always be less than or equal to R².In the formula below <strong>n</strong> is the number of observations in the data and <strong>k</strong> is the number of the independent variables in the data.</li>
</ul>
<p><img src="https://miro.medium.com/max/60/1*bBrMI54b85FGilV_ciAL-Q.png?q=20"></p>
<p><img src="https://miro.medium.com/max/262/1*bBrMI54b85FGilV_ciAL-Q.png"></p>
<h3 id="Differences-among-these-evaluation-metrics"><a href="#Differences-among-these-evaluation-metrics" class="headerlink" title="Differences among these evaluation metrics"></a>Differences among these evaluation metrics</h3><ul>
<li>  Mean Squared Error(MSE) and Root Mean Square Error penalizes the large prediction errors vi-a-vis Mean Absolute Error (MAE). However, RMSE is widely used than MSE to evaluate the performance of the regression model with other random models as it has the same units as the dependent variable (Y-axis).</li>
<li>  MSE is a differentiable function that makes it easy to perform mathematical operations in comparison to a non-differentiable function like MAE. Therefore, in many models, RMSE is used as a default metric for calculating Loss Function despite being harder to interpret than MAE.</li>
<li>  MAE is more robust to data with outliers.</li>
<li>  The lower value of MAE, MSE, and RMSE implies higher accuracy of a regression model. However, a higher value of R square is considered desirable.</li>
<li>  R Squared &amp; Adjusted R Squared are used for explaining how well the independent variables in the linear regression model explains the variability in the dependent variable. R Squared value always increases with the addition of the independent variables which might lead to the addition of the redundant variables in our model. However, the adjusted R-squared solves this problem.</li>
<li>  Adjusted R squared takes into account the number of predictor variables, and it is used to determine the number of independent variables in our model. The value of Adjusted R squared decreases if the increase in the R square by the additional variable isn’t significant enough.</li>
<li>  For comparing the accuracy among different linear regression models, RMSE is a better choice than R Squared.</li>
</ul>
<p><strong>Conclusion</strong><br>Therefore, if comparing the prediction accuracy among different linear regression (LR)models then RMSE is a better option as it is simple to calculate and differentiable. However, if your dataset has outliers then choose MAE over RMSE.</p>
<p>Besides, the number of predictor variables in a linear regression model is determined by adjusted R squared, and choose RMSE over adjusted R squared if you care about evaluating prediction accuracy among different LR models.</p>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br><span class="line">13</span><br><span class="line">14</span><br><span class="line">15</span><br><span class="line">16</span><br><span class="line">17</span><br><span class="line">18</span><br><span class="line">19</span><br><span class="line">20</span><br><span class="line">21</span><br><span class="line">22</span><br><span class="line">23</span><br><span class="line">24</span><br><span class="line">25</span><br><span class="line">26</span><br><span class="line">27</span><br><span class="line">28</span><br><span class="line">29</span><br><span class="line">30</span><br><span class="line">31</span><br><span class="line">32</span><br><span class="line">33</span><br><span class="line">34</span><br><span class="line">35</span><br><span class="line">36</span><br><span class="line">37</span><br><span class="line">38</span><br><span class="line">39</span><br><span class="line">40</span><br><span class="line">41</span><br><span class="line">42</span><br><span class="line">43</span><br><span class="line">44</span><br><span class="line">45</span><br><span class="line">46</span><br><span class="line">47</span><br><span class="line">48</span><br><span class="line">49</span><br><span class="line">50</span><br><span class="line">51</span><br><span class="line">52</span><br><span class="line">53</span><br><span class="line">54</span><br><span class="line">55</span><br><span class="line">56</span><br><span class="line">57</span><br><span class="line">58</span><br><span class="line">59</span><br><span class="line">60</span><br></pre></td><td class="code"><pre><span class="line">original &#x3D; c( -2,  1, -3, 2, 3, 5, 4, 6, 5, 6, 7)</span><br><span class="line">predicted &#x3D; c(-1, -1, -2, 2, 3, 4, 4, 5, 5, 7, 7)</span><br><span class="line">#</span><br><span class="line">x&#x3D;1:length(original)</span><br><span class="line">plot(x, original,pch&#x3D;19, col&#x3D;&quot;blue&quot;)</span><br><span class="line">lines(x, predicted, col&#x3D;&quot;red&quot;)</span><br><span class="line">legend(&quot;topleft&quot;, legend &#x3D; c(&quot;y-original&quot;, &quot;y-predicted&quot;),</span><br><span class="line">       col &#x3D; c(&quot;blue&quot;, &quot;red&quot;), pch &#x3D; c(19,NA), lty &#x3D; c(NA,1),  cex &#x3D; 0.7)</span><br><span class="line">#</span><br><span class="line">d &#x3D; original-predicted</span><br><span class="line">mse &#x3D; mean((d)^2)</span><br><span class="line">mae &#x3D; mean(abs(d))</span><br><span class="line">rmse &#x3D; sqrt(mse)</span><br><span class="line">R2 &#x3D; 1-(sum((d)^2)&#x2F;sum((original-mean(original))^2))</span><br><span class="line"></span><br><span class="line">cat(&quot; MAE:&quot;, mae, &quot;\n&quot;, &quot;MSE:&quot;, mse, &quot;\n&quot;, </span><br><span class="line">    &quot;RMSE:&quot;, rmse, &quot;\n&quot;, &quot;R-squared:&quot;, R2)</span><br><span class="line">#</span><br><span class="line">#######################################################</span><br><span class="line">#</span><br><span class="line">library(caret)</span><br><span class="line"># R native funcitons</span><br><span class="line">MAE(predicted, original)</span><br><span class="line">MSE(predicted, original) </span><br><span class="line"># caret package functions </span><br><span class="line">RMSE(predicted, original)</span><br><span class="line">R2(predicted, original, form &#x3D; &quot;traditional&quot;)</span><br><span class="line">#</span><br><span class="line">#######################################################</span><br><span class="line">#</span><br><span class="line">library(caret)</span><br><span class="line">#</span><br><span class="line">original &#x3D; c( -2,  1, -3, 2, 3, 5, 4, 6, 5, 6, 7)</span><br><span class="line">predicted &#x3D; c(-1, -1, -2, 2, 3, 4, 4, 5, 5, 7, 7)</span><br><span class="line">x &#x3D; 1:length(original)</span><br><span class="line">#</span><br><span class="line">plot(x, original,pch&#x3D;19, col&#x3D;&quot;blue&quot;)</span><br><span class="line">lines(x, predicted,col&#x3D;&quot;red&quot;)</span><br><span class="line">legend(&quot;topleft&quot;, legend &#x3D; c(&quot;y-original&quot;, &quot;y-predicted&quot;),</span><br><span class="line">       col&#x3D;c(&quot;blue&quot;, &quot;red&quot;), pch&#x3D;c(19,NA), lty&#x3D;c(NA,1), cex &#x3D; 0.7)</span><br><span class="line">d &#x3D; original-predicted</span><br><span class="line">mse &#x3D; mean((d)^2)</span><br><span class="line">mae &#x3D; mean(abs(d))</span><br><span class="line">rmse &#x3D; sqrt(mse)</span><br><span class="line">R2 &#x3D; 1-(sum((d)^2)&#x2F;sum((original-mean(original))^2)</span><br><span class="line"></span><br><span class="line">cat(&quot; MAE:&quot;, mae, &quot;\n&quot;, &quot;MSE:&quot;, mse, &quot;\n&quot;, </span><br><span class="line">    &quot;RMSE:&quot;, rmse, &quot;\n&quot;, &quot;R-squared:&quot;, R2)</span><br><span class="line"> </span><br><span class="line"></span><br><span class="line"># R native funcitons</span><br><span class="line">fmse &#x3D; MSE(predicted, original)</span><br><span class="line">fmae &#x3D; MAE(predicted, original)</span><br><span class="line"> </span><br><span class="line"># caret package functions </span><br><span class="line">frmse &#x3D; RMSE(predicted, original)</span><br><span class="line">fr2 &#x3D; R2(predicted, original, form &#x3D; &quot;traditional&quot;)</span><br><span class="line"> </span><br><span class="line">cat(&quot; MAE:&quot;, fmae, &quot;\n&quot;, &quot;MSE:&quot;, fmse, &quot;\n&quot;, </span><br><span class="line">     &quot;RMSE:&quot;, frmse, &quot;\n&quot;, &quot;R-squared:&quot;, fr2)</span><br></pre></td></tr></table></figure>
      

      
    </div>
    <div class="article-info article-info-index">
      
      
      
	<div class="article-category tagcloud">
		<i class="icon-book icon"></i>
		<ul class="article-tag-list">
			 
        		<li class="article-tag-list-item">
        			<a href="/categories/Log//" class="article-tag-list-link color4">Log</a>
        		</li>
      		
		</ul>
	</div>


      
        <p class="article-more-link">
          <a class="article-more-a" href="/2021/04/08/20210408/">Expand all items >></a>
        </p>
      

      
      <div class="clearfix"></div>
    </div>
  </div>
</article>

<aside class="wrap-side-operation">
    <div class="mod-side-operation">
        
        <div class="jump-container" id="js-jump-container" style="display:none;">
            <a href="javascript:void(0)" class="mod-side-operation__jump-to-top">
                <i class="icon-font icon-back"></i>
            </a>
            <div id="js-jump-plan-container" class="jump-plan-container" style="top: -11px;">
                <i class="icon-font icon-plane jump-plane"></i>
            </div>
        </div>
        
        
    </div>
</aside>




  
    <article id="post-20210403" class="article article-type-post  article-index" itemscope itemprop="blogPost">
  <div class="article-inner">
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2021/04/03/20210403/">20210403</a>
    </h1>
  

        
        <a href="/2021/04/03/20210403/" class="archive-article-date">
  	<time datetime="2021-04-02T16:00:00.000Z" itemprop="datePublished"><i class="icon-calendar icon"></i>20210403</time>
</a>
        
      </header>
    
    <div class="article-entry" itemprop="articleBody">
    

      
<div id="toc" class="toc-article">
  <strong class="toc-title">Contents</strong>
  <ol class="toc"><li class="toc-item toc-level-1"><a class="toc-link" href="#Working-directories"><span class="toc-number">1.</span> <span class="toc-text">Working directories</span></a></li></ol>
</div>

        <h1 id="Working-directories"><a href="#Working-directories" class="headerlink" title="Working directories"></a>Working directories</h1><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Stored_Softs&#x2F;Cross_Platform_Softs&#x2F;MS_Project_Alternative&#x2F;Planner&#39;; cd $&#123;i&#125;</span><br><span class="line">#</span><br><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Work&#x2F;Projects&#x2F;ht.planner&#39;</span><br><span class="line">setsid planner $&#123;i&#125;</span><br></pre></td></tr></table></figure>

      

      
    </div>
    <div class="article-info article-info-index">
      
      
      
	<div class="article-category tagcloud">
		<i class="icon-book icon"></i>
		<ul class="article-tag-list">
			 
        		<li class="article-tag-list-item">
        			<a href="/categories/Log//" class="article-tag-list-link color4">Log</a>
        		</li>
      		
		</ul>
	</div>


      
        <p class="article-more-link">
          <a class="article-more-a" href="/2021/04/03/20210403/">Expand all items >></a>
        </p>
      

      
      <div class="clearfix"></div>
    </div>
  </div>
</article>

<aside class="wrap-side-operation">
    <div class="mod-side-operation">
        
        <div class="jump-container" id="js-jump-container" style="display:none;">
            <a href="javascript:void(0)" class="mod-side-operation__jump-to-top">
                <i class="icon-font icon-back"></i>
            </a>
            <div id="js-jump-plan-container" class="jump-plan-container" style="top: -11px;">
                <i class="icon-font icon-plane jump-plane"></i>
            </div>
        </div>
        
        
    </div>
</aside>




  
    <article id="post-20210402" class="article article-type-post  article-index" itemscope itemprop="blogPost">
  <div class="article-inner">
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2021/04/02/20210402/">20210402</a>
    </h1>
  

        
        <a href="/2021/04/02/20210402/" class="archive-article-date">
  	<time datetime="2021-04-01T16:00:00.000Z" itemprop="datePublished"><i class="icon-calendar icon"></i>20210402</time>
</a>
        
      </header>
    
    <div class="article-entry" itemprop="articleBody">
    

      
<div id="toc" class="toc-article">
  <strong class="toc-title">Contents</strong>
  <ol class="toc"><li class="toc-item toc-level-1"><a class="toc-link" href="#Working-directories"><span class="toc-number">1.</span> <span class="toc-text">Working directories</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#Microsoft-Project-Alternative"><span class="toc-number">1.1.</span> <span class="toc-text">Microsoft Project Alternative</span></a></li></ol></li></ol>
</div>

        <h1 id="Working-directories"><a href="#Working-directories" class="headerlink" title="Working directories"></a>Working directories</h1><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">ftp:&#x2F;&#x2F;ht@192.168.1.8:2121&#x2F;Android&#x2F;data&#x2F;com.mendhak.gpslogger</span><br><span class="line">#</span><br><span class="line">cd &#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Management&#x2F;GPS&#x2F;20210401</span><br></pre></td></tr></table></figure>
<h2 id="Microsoft-Project-Alternative"><a href="#Microsoft-Project-Alternative" class="headerlink" title="Microsoft Project Alternative"></a>Microsoft Project Alternative</h2><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">i&#x3D;&#39;&#x2F;media&#x2F;ht&#x2F;ht_5T&#x2F;Stored_Softs&#x2F;Cross_Platform_Softs&#x2F;MS_Project_Alternative&#39;; cd $&#123;i&#125;</span><br></pre></td></tr></table></figure>
<ul>
<li>A good review in <a target="_blank" rel="noopener" href="https://www.linuxlinks.com/projectmanagement/">12 Best Free Linux Project Management Software (Updated 2019) - LinuxLinks</a></li>
<li>Another review in <a target="_blank" rel="noopener" href="https://www.goodfirms.co/blog/best-free-open-source-gantt-chart-software-solutions">The Best 8 Free and Open Source Gantt Chart Software Solutions</a></li>
</ul>

      

      
    </div>
    <div class="article-info article-info-index">
      
      
      
	<div class="article-category tagcloud">
		<i class="icon-book icon"></i>
		<ul class="article-tag-list">
			 
        		<li class="article-tag-list-item">
        			<a href="/categories/Log//" class="article-tag-list-link color4">Log</a>
        		</li>
      		
		</ul>
	</div>


      
        <p class="article-more-link">
          <a class="article-more-a" href="/2021/04/02/20210402/">Expand all items >></a>
        </p>
      

      
      <div class="clearfix"></div>
    </div>
  </div>
</article>

<aside class="wrap-side-operation">
    <div class="mod-side-operation">
        
        <div class="jump-container" id="js-jump-container" style="display:none;">
            <a href="javascript:void(0)" class="mod-side-operation__jump-to-top">
                <i class="icon-font icon-back"></i>
            </a>
            <div id="js-jump-plan-container" class="jump-plan-container" style="top: -11px;">
                <i class="icon-font icon-plane jump-plane"></i>
            </div>
        </div>
        
        
    </div>
</aside>




  
    <article id="post-20210330" class="article article-type-post  article-index" itemscope itemprop="blogPost">
  <div class="article-inner">
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2021/03/30/20210330/">20210330</a>
    </h1>
  

        
        <a href="/2021/03/30/20210330/" class="archive-article-date">
  	<time datetime="2021-03-29T16:00:00.000Z" itemprop="datePublished"><i class="icon-calendar icon"></i>20210330</time>
</a>
        
      </header>
    
    <div class="article-entry" itemprop="articleBody">
    

      
<div id="toc" class="toc-article">
  <strong class="toc-title">Contents</strong>
  <ol class="toc"><li class="toc-item toc-level-1"><a class="toc-link" href="#Protein-Protein-Docking"><span class="toc-number">1.</span> <span class="toc-text">Protein-Protein Docking</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#Ref"><span class="toc-number">1.1.</span> <span class="toc-text">Ref</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#Protein-Protein-Docking-Using-Bioinformatics-Tools-PPDock-Module"><span class="toc-number">1.2.</span> <span class="toc-text">Protein-Protein Docking Using Bioinformatics Tools (PPDock) Module</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#I-Introduction"><span class="toc-number">1.2.1.</span> <span class="toc-text">I. Introduction</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#II-Step-1-Gathering-Protein-Data-Bank-PDB-Files"><span class="toc-number">1.2.2.</span> <span class="toc-text">II. Step 1: Gathering Protein Data Bank (PDB) Files</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#III-Step-2-Splitting-PDB-Files-Using-DECOMP"><span class="toc-number">1.2.3.</span> <span class="toc-text">III. Step 2: Splitting PDB Files Using DECOMP</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#IV-Step-3-Use-GRAMM-to-Predict-the-Interactions"><span class="toc-number">1.2.4.</span> <span class="toc-text">IV. Step 3: Use GRAMM to Predict the Interactions.</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#V-Step-4-Visualizing-Protein-Interactions"><span class="toc-number">1.2.5.</span> <span class="toc-text">V. Step 4: Visualizing Protein Interactions</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#VI-Step-5-Showing-Interfacial"><span class="toc-number">1.2.6.</span> <span class="toc-text">VI. Step 5: Showing Interfacial</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#Amino-Acids"><span class="toc-number">1.2.7.</span> <span class="toc-text">Amino Acids</span></a></li></ol></li><li class="toc-item toc-level-2"><a class="toc-link" href="#Protein-Protein-Docking-1"><span class="toc-number">1.3.</span> <span class="toc-text">Protein-Protein Docking</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#Summary"><span class="toc-number">1.3.1.</span> <span class="toc-text">Summary</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#Docking-in-Rosetta"><span class="toc-number">1.3.2.</span> <span class="toc-text">Docking in Rosetta</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#Navigating-to-the-Demos"><span class="toc-number">1.3.3.</span> <span class="toc-text">Navigating to the Demos</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#Preparing-Structures-for-Docking"><span class="toc-number">1.3.4.</span> <span class="toc-text">Preparing Structures for Docking</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#Local-Docking"><span class="toc-number">1.3.5.</span> <span class="toc-text">Local Docking</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#Local-Refinement-of-Docked-Structures"><span class="toc-number">1.3.6.</span> <span class="toc-text">Local Refinement of Docked Structures</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#Global-Docking"><span class="toc-number">1.3.7.</span> <span class="toc-text">Global Docking</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#Docking-Flexible-Proteins"><span class="toc-number">1.3.8.</span> <span class="toc-text">Docking Flexible Proteins</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#Symmetric-Docking"><span class="toc-number">1.3.9.</span> <span class="toc-text">Symmetric Docking</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#Analyzing-Docked-Structures"><span class="toc-number">1.3.10.</span> <span class="toc-text">Analyzing Docked Structures</span></a></li></ol></li><li class="toc-item toc-level-2"><a class="toc-link" href="#Protein-Protein-Docking-From-Interaction-to-Interactome"><span class="toc-number">1.4.</span> <span class="toc-text">Protein-Protein Docking: From Interaction to Interactome</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#Abstract"><span class="toc-number">1.4.1.</span> <span class="toc-text">Abstract</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#Introduction"><span class="toc-number">1.4.2.</span> <span class="toc-text">Introduction</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#Beginnings"><span class="toc-number">1.4.3.</span> <span class="toc-text">Beginnings</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#Docking-Foundations"><span class="toc-number">1.4.4.</span> <span class="toc-text">Docking Foundations</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#Bound-and-Unbound-Docking"><span class="toc-number">1.4.5.</span> <span class="toc-text">Bound and Unbound Docking</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#Docking-of-Models"><span class="toc-number">1.4.6.</span> <span class="toc-text">Docking of Models</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#Physics-Versus-Knowledge-Based-Docking"><span class="toc-number">1.4.7.</span> <span class="toc-text">Physics Versus Knowledge-Based Docking</span></a></li></ol></li></ol></li></ol>
</div>

        <h1 id="Protein-Protein-Docking"><a href="#Protein-Protein-Docking" class="headerlink" title="Protein-Protein Docking"></a>Protein-Protein Docking</h1><h2 id="Ref"><a href="#Ref" class="headerlink" title="Ref"></a>Ref</h2><ul>
<li><a target="_blank" rel="noopener" href="https://www.utep.edu/science/upbit/curriculum/ppd-dock.html">Protein-Protein Docking Using Bioinformatics Tools (PPDock) Module</a></li>
<li><a target="_blank" rel="noopener" href="https://www.dnastar.com/workflows/protein-docking/">Protein Docking Analysis | DNASTAR</a></li>
<li><a target="_blank" rel="noopener" href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005905">High-resolution global peptide-protein docking using fragments-based PIPER-FlexPepDock</a><ul>
<li>Peptide-protein interactions contribute a significant fraction of the protein-protein interactome. Accurate modeling of these interactions is challenging due to the vast conformational space associated with interactions of highly flexible peptides with large receptor surfaces. To address this challenge we developed a fragment based high-resolution peptide-protein docking protocol. By streamlining the Rosetta fragment picker for accurate peptide fragment ensemble generation, the PIPER docking algorithm for exhaustive fragment-receptor rigid-body docking and Rosetta FlexPepDock for flexible full-atom refinement of PIPER docked models, we successfully addressed the challenge of accurate and efficient global peptide-protein docking at high-resolution with remarkable accuracy, as validated on a small but representative set of peptide-protein complex structures well resolved by X-ray crystallography. Our approach opens up the way to high-resolution modeling of many more peptide-protein interactions and to the detailed study of peptide-protein association in general. PIPER-FlexPepDock is freely available to the academic community as a server at <a target="_blank" rel="noopener" href="http://piperfpd.furmanlab.cs.huji.ac.il/">http://piperfpd.furmanlab.cs.huji.ac.il</a>.</li>
</ul>
</li>
</ul>
<ul>
<li><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking">Protein-Protein Docking</a></li>
<li><a target="_blank" rel="noopener" href="https://www.sciencedirect.com/topics/medicine-and-dentistry/docking-protein#:~:text=Protein%2Ddocking%20is%20a%20molecular,chain%20or%20a%20smaller%20molecule.">Docking Protein - an overview | ScienceDirect Topics</a></li>
<li><a target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Docking_(molecular)">Docking (molecular) - Wikipedia</a></li>
<li><a target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Protein%E2%80%93ligand_docking">Protein–ligand docking - Wikipedia</a></li>
<li><a target="_blank" rel="noopener" href="https://www.nature.com/articles/s41596-020-0312-x">The HDOCK server for integrated protein–protein docking | Nature Protocols</a><ul>
<li><a target="_blank" rel="noopener" href="http://hdock.phys.hust.edu.cn/">HDOCK Server</a></li>
<li>The HDOCK server (<a target="_blank" rel="noopener" href="http://hdock.phys.hust.edu.cn/">http://hdock.phys.hust.edu.cn/</a>) is a highly integrated suite of homology search, template-based modeling, structure prediction, macromolecular docking, biological information incorporation and job management for robust and fast protein–protein docking. With input information for receptor and ligand molecules (either amino acid sequences or Protein Data Bank structures), the server automatically predicts their interaction through a hybrid algorithm of template-based and template-free docking. The HDOCK server distinguishes itself from similar docking servers in its ability to support amino acid sequences as input and a hybrid docking strategy in which experimental information about the protein–protein binding site and small-angle X-ray scattering can be incorporated during the docking and post-docking processes. Moreover, HDOCK also supports protein–RNA/DNA docking with an intrinsic scoring function. The server delivers both template- and docking-based binding models of two molecules and allows for download and interactive visualization. The HDOCK server is user friendly and has processed &gt;30,000 docking jobs since its official release in 2017. The server can normally complete a docking job within 30 min.</li>
</ul>
</li>
<li><a target="_blank" rel="noopener" href="https://www.nature.com/articles/nprot.2016.169">The ClusPro web server for protein–protein docking | Nature Protocols</a><ul>
<li><a target="_blank" rel="noopener" href="https://cluspro.org/login.php">ClusPro 2.0: protein-protein docking</a></li>
<li>The ClusPro server (<a target="_blank" rel="noopener" href="https://cluspro.org/">https://cluspro.org</a>) is a widely used tool for protein–protein docking. The server provides a simple home page for basic use, requiring only two files in Protein Data Bank (PDB) format. However, ClusPro also offers a number of advanced options to modify the search; these include the removal of unstructured protein regions, application of attraction or repulsion, accounting for pairwise distance restraints, construction of homo-multimers, consideration of small-angle X-ray scattering (SAXS) data, and location of heparin-binding sites. Six different energy functions can be used, depending on the type of protein. Docking with each energy parameter set results in ten models defined by centers of highly populated clusters of low-energy docked structures. This protocol describes the use of the various options, the construction of auxiliary restraints files, the selection of the energy parameters, and the analysis of the results. Although the server is heavily used, runs are generally completed in &lt;4 h.</li>
</ul>
</li>
<li><a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/">Protein-Protein Docking: From Interaction to Interactome</a></li>
<li><a target="_blank" rel="noopener" href="https://pubmed.ncbi.nlm.nih.gov/29594778/">Protein-Protein Docking in Drug Design and Discovery - PubMed</a></li>
<li><a target="_blank" rel="noopener" href="https://link.springer.com/protocol/10.1007%2F978-1-4939-7756-7_15">Protein–Protein Docking in Drug Design and Discovery | SpringerLink</a><ul>
<li>Protein–protein interactions (PPIs) are responsible for a number of key physiological processes in the living cells and underlie the pathomechanism of many diseases. Nowadays, along with the concept of so-called “hot spots” in protein–protein interactions, which are well-defined interface regions responsible for most of the binding energy, these interfaces can be targeted with modulators. In order to apply structure-based design techniques to design PPIs modulators, a three-dimensional structure of protein complex has to be available. In this context in silico approaches, in particular protein–protein docking, are a valuable complement to experimental methods for elucidating 3D structure of protein complexes. Protein–protein docking is easy to use and does not require significant computer resources and time (in contrast to molecular dynamics) and it results in 3D structure of a protein complex (in contrast to sequence-based methods of predicting binding interfaces). However, protein–protein docking cannot address all the aspects of protein dynamics, in particular the global conformational changes during protein complex formation. In spite of this fact, protein–protein docking is widely used to model complexes of water-soluble proteins and less commonly to predict structures of transmembrane protein assemblies, including dimers and oligomers of G protein-coupled receptors (GPCRs). In this chapter we review the principles of protein–protein docking, available algorithms and software and discuss the recent examples, benefits, and drawbacks of protein–protein docking application to water-soluble proteins, membrane anchoring and transmembrane proteins, including GPCRs.</li>
</ul>
</li>
</ul>
<h2 id="Protein-Protein-Docking-Using-Bioinformatics-Tools-PPDock-Module"><a href="#Protein-Protein-Docking-Using-Bioinformatics-Tools-PPDock-Module" class="headerlink" title="Protein-Protein Docking Using Bioinformatics Tools (PPDock) Module"></a>Protein-Protein Docking Using Bioinformatics Tools (PPDock) Module</h2><ul>
<li><a target="_blank" rel="noopener" href="https://www.utep.edu/science/upbit/curriculum/ppd-dock.html">Protein-Protein Docking Using Bioinformatics Tools (PPDock) Module</a></li>
</ul>
<h3 id="I-Introduction"><a href="#I-Introduction" class="headerlink" title="I. Introduction"></a>I. Introduction</h3><p>As cancerous cells and normal cells exhibit a few biochemical differences, many anticancer drugs affect normal rapidly growing cells in the intestine and bone marrow areas and hence are toxic. Capabilities to determine drug-target binding affinities to achieve high levels of selective drug actions on cancer cells would be very useful for designing anti-cancer therapeutics. Since the over expression of the Janus Kinase 3 (JAK-3) has been implicated in cancerous disorders like adult T-cell lymphoma/leukemia (ATLL), JAK-3 inhibition is expected to play a vital role in treatment of cancer. The objective of this project is to use docking studies to identify potential JAK-3 inhibitors from a number of putative substrates, namely anaplastic lymphoma kinase (ALK), Gene transcription factor II-I (TFII-I) etc. Let us start the process by taking ALK as a potential inhibitor. Follow the following steps to get the process done.</p>
<h3 id="II-Step-1-Gathering-Protein-Data-Bank-PDB-Files"><a href="#II-Step-1-Gathering-Protein-Data-Bank-PDB-Files" class="headerlink" title="II. Step 1: Gathering Protein Data Bank (PDB) Files"></a>II. Step 1: Gathering Protein Data Bank (PDB) Files</h3><p>PDB files are the collection of experimentally determined three-dimensional   structures of macromolecules, which are generally used by researchers and students. The collection includes the atomic coordinates, crystallographic structure factors and NMR experimental data. It also includes name of     molecules, primary and secondary structure information, ligand and biological assembly information, details about data collection and bibliographic citations.</p>
<p>PDB files can be found at the following website: <a target="_blank" rel="noopener" href="http://www.rcsb.org/pdb/home/home.do">http://www.rcsb.org/pdb/home/home.do</a></p>
<p>Enter the PDB ID in the search bar as shown below </p>
<p>Gather the crystal   structures of Jak3 and ALK (anaplastic lymphoma receptor tyrosine kinase) from   RCSB Protein Data Bank. The crystal structures are available in PDB format. The pdb file of Jak3 is 1YVJ. The pdb file of ALK is 4DCE.</p>
<h3 id="III-Step-2-Splitting-PDB-Files-Using-DECOMP"><a href="#III-Step-2-Splitting-PDB-Files-Using-DECOMP" class="headerlink" title="III. Step 2: Splitting PDB Files Using DECOMP"></a>III. Step 2: Splitting PDB Files Using DECOMP</h3><p>DECOMP is a web-based decomposition tool for splitting PDB files. Protein information technology group of Eotvos University located in Hungary developed it. With this program, protein-ligand complexes can be identified reliably and the ligands are deposited in separate files. Missing residues and atoms in chains are handled properly and are inserted into chains for missing residues/atoms. DECOMP server can be found at the following website: <a target="_blank" rel="noopener" href="http://decomp.pitgroup.org/%C2%A0">http://decomp.pitgroup.org/</a><a target="_blank" rel="noopener" href="http://decomp.pitgroup.org/"></a></p>
<p>Enter the PDB ID in the empty box shown in the Figure 2. The pdb files of the above two proteins are available along with their ligands. So we have to remove the ligand. 1YVJ is the Jak3 kinase domain in complex with a straurosporine analogue. So we have to separate the straurosporine from the Jak3 kinase domain. In the same way 4DCE is anaplastic lymphoma kinase in complex with a piperidine - carboxamide inhibitor. So we have to remove the inhibitor.</p>
<p>Working with DECOMP:</p>
<p>We have to submit the pdb files of the proteins into the server. We are provided with various options to export ligands, ions, and water molecules or to insert missing atoms or residues. Choose the option to export ligand and submitted the files. The requests are in the form of a queue i.e., first in first out and the time of output depends on the traffic in the server. The output will be in the form of a tar.gz files i.e. the compressed version. So after we extract the files from the tar.gz files we have one directory for each of the pdb’s listed. Each of these directories contains an error log with “.Error” extension the decomposed pdb file with “. pdb ” extension and separate files of ligands or ions are present if the option export ligand or ion was chosen.</p>
<p>From this directory take the decomposed pdb file i.e., a file with “. pdb ” extension.</p>
<h3 id="IV-Step-3-Use-GRAMM-to-Predict-the-Interactions"><a href="#IV-Step-3-Use-GRAMM-to-Predict-the-Interactions" class="headerlink" title="IV. Step 3: Use GRAMM to Predict the Interactions."></a>IV. Step 3: Use GRAMM to Predict the Interactions.</h3><p>GRAMM (Global Range Molecular Matching) is a program for protein docking. GRAMM is open source software and can be installed on the personal computer. It is developed by the Vakser’s lab (Center for Bioinformatics) belonging to university of Kansas. It can be installed on MAC, Windows and Linux operating systems. The working instructions given in this guide pertain to the windows version. It can be downloaded from the following website. Its installation instruction was also given on the same page.</p>
<p><a target="_blank" rel="noopener" href="http://vakser.bioinformatics.ku.edu/main/resources_gramm1.03.php">http://vakser.bioinformatics.ku.edu/main/resources_gramm1.03.php</a></p>
<p>Working with GRAMM:</p>
<p>GRAMM has 3 parameter files rpar.gr, rmol.gr and wlist.gr files. All these files are text files. I prefer using notepad to edit these files. </p>
<ol>
<li>Parameters to be considered for rpar.gr file:</li>
</ol>
<p>The parameters for this file should be considered based on the type of molecules we work with. The options available are high resolution generic docking and low resolution generic docking. Let’s try low-resolution generic docking in this case as we are not sure about the structures of the given molecules and the parameters given for the low resolution generic docking are: </p>
<ol start="2">
<li>The other file is rmol.gr file. The following is the format of the file with the values I have used: </li>
</ol>
<p>The two molecules to which we need to see the predictions should be given here. Under the file name we should give the name of one of the files with atomic coordinates (PDB format). It should be converted in to . ent form before giving it here (1YVJ.ent). To change the file format go to command prompt in windows. To open command prompt press windows symbol on keyboard and type “ cmd ” in the search bar and then press enter. A black screen appears on your desktop, which is command prompt in windows. Now type the following commands in the command prompt to rename the files.</p>
<p>C:\ Users\Name&gt; cd C:\ (This command takes you to C drive).</p>
<p>C: \&gt; cd Gramm (This command takes you to Gramm directory in C drive).</p>
<p>C:\Gramm&gt; rename 1YVJ.pdb 1YVJ.ent (This command renames the file) </p>
<p>Under the column of the fragment you have to mention the range of atoms for which the interactions are to be found or simply you can give it as ’*’ so that the entire molecule can be considered. Under the column of the ID you can give some string of characters without spaces between them to identify your molecules. These ID’S will be used by GRAMM to name the output files.</p>
<p>Run GRAMM with parameter scan ( gramm scan) from the terminal in the GRAMM directory i.e., type “ gramm scan” command in the command prompt and press enter. It creates a .log file and .res file. To be aware which grid has been chosen, see the output .log file. Grid is the potential docking area in which the docking protein searches so as to release maximum amount of energy up on reacting with the docked protein.</p>
<ol start="3">
<li>The last step is giving parameters in wlist.gr file: The following is the format of the file with the values I have used:</li>
</ol>
<p>.res is the file that is obtained in the step b. </p>
<p>First match and last match here refer to the retrieval of the top 10 hits from thousands of complexes generated by Gramm. “Separate” here results in 10 separate pdb files instead of all the ten conformations in the same file. If you want all the 10 conformations in the same pdb file use “joint” Now run GRAMM with the parameter coord ( gramm coord ) from the commands prompt in the GRAMM directory i.e. type the command gramm coord in the command prompt and press enter. Place the GRAMM directory in C drive while you install.</p>
<p>C:\ Users\Name&gt; cd C:\ (This command takes you to C drive).</p>
<p>C: \&gt; cd Gramm (This command takes you to Gramm directory in C drive).</p>
<p>C:\ Gramm&gt; gramm scan (Creates .res and .log file).</p>
<p>C:\Gramm&gt;gramm coord (Creates the pdb files) </p>
<p><em>After this you will finally get a file in . pdb format if you use</em> <em>“__joint__”</em> <em>option in wlist.gr file or 10 separate pdb files for this example if you use</em> <em>“</em> <em>seperate</em> <em>”</em> <em>option, which shows us the various ways that the given two proteins interact.</em></p>
<h3 id="V-Step-4-Visualizing-Protein-Interactions"><a href="#V-Step-4-Visualizing-Protein-Interactions" class="headerlink" title="V. Step 4: Visualizing Protein Interactions"></a>V. Step 4: Visualizing Protein Interactions</h3><ol start="2">
<li>Python Molecular viewer (PMV): PMV is a powerful molecular viewer that has a number of customizable features. It is distributed as a part of MGLTools , we need to download and install MGLTools to get PMV. It is developed by Molecular Graphics Laboratory of The Scripps Research Institute located in La Jolla, California. It is freely available for download at <a target="_blank" rel="noopener" href="http://mgltools.scripps.edu/">http://mgltools.scripps.edu/</a> . It can be used on windows, linux and Mac operating systems. </li>
</ol>
<p>Three tools that can be used to visualize the final PDF files obtained from GRAMM are:</p>
<ol>
<li><p>UCSF Chimera: It is a program for interactive visualization and analysis of molecular structures. High quality images and animations can be generated. It is developed by Resource for Biocomputing, Visualization and Informatics (RBVI) Department belonging to University of California, San Francisco. Download at <a target="_blank" rel="noopener" href="http://www.cgl.ucsf.edu/chimera/download.html">http://www.cgl.ucsf.edu/chimera/download.html</a> . It can be used on windows, linux and Mac operating   systems. </p>
</li>
<li><p>Swiss- Pdb viewer: Swiss Pdb viewer can load and display several molecules simultaneously. Each molecule is loaded into its own layer. It was developed by The SIB Swiss Institute of Bioinformatics located in Switzerland. It is freely available for download at <a target="_blank" rel="noopener" href="http://spdbv.vital-it.ch/disclaim.html">http://spdbv.vital-it.ch/disclaim.html</a> . It can be used on windows, linux and Mac operating systems. </p>
</li>
</ol>
<h3 id="VI-Step-5-Showing-Interfacial"><a href="#VI-Step-5-Showing-Interfacial" class="headerlink" title="VI. Step 5: Showing Interfacial"></a>VI. Step 5: Showing Interfacial</h3><h3 id="Amino-Acids"><a href="#Amino-Acids" class="headerlink" title="Amino Acids"></a>Amino Acids</h3><p>Using servers such as SPPIDER and PISA server to get to know about the interfacial amino acids, number of hydrogen bonds, interfacial area and the amount of energy released. PISA server can be accessed at <a target="_blank" rel="noopener" href="http://www.ebi.ac.uk/msd-srv/prot_int/">http://www.ebi.ac.uk/msd-srv/prot_int/</a> . SPPIDER can be accessed at <a target="_blank" rel="noopener" href="http://sppider.cchmc.org/">http://sppider.cchmc.org/</a> . Some sample outputs are shown below</p>
<h2 id="Protein-Protein-Docking-1"><a href="#Protein-Protein-Docking-1" class="headerlink" title="Protein-Protein Docking"></a>Protein-Protein Docking</h2><ul>
<li><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking">Protein-Protein Docking</a></li>
</ul>
<ul>
<li><p><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking##" title="Search this wiki">Search</a></p>
</li>
<li><p><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/">Home</a></p>
</li>
<li><p><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/feedback?ref=tutorials/Protein-Protein-Docking/Protein-Protein-Docking.md">Feedback</a></p>
</li>
</ul>
<p>KEYWORDS: DOCKING GENERAL STRUCTURE_PREDICTION<br>Written by by Sebastian Rämisch (<a href="mailto:raemisch@scripps.edu">raemisch@scripps.edu</a>). Edited by Shourya S. Roy Burman (<a href="mailto:ssrb@jhu.edu">ssrb@jhu.edu</a>)<br>Edited Jun 24 2016</p>
<p>Table of Contents</p>
<ul>
<li><ul>
<li><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#summary">Summary</a><ul>
<li><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#docking-in-rosetta">Docking in Rosetta</a></li>
<li><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#navigating-to-the-demos">Navigating to the Demos</a></li>
<li><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#preparing-structures-for-docking">Preparing Structures for Docking</a></li>
<li><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#local-docking">Local Docking</a></li>
<li><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#local-refinement-of-docked-structures">Local Refinement of Docked Structures</a></li>
<li><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#global-docking">Global Docking</a></li>
<li><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#docking-flexible-proteins">Docking Flexible Proteins</a></li>
<li><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#symmetric-docking">Symmetric Docking</a></li>
<li><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#analyzing-docked-structures">Analyzing Docked Structures</a></li>
</ul>
</li>
</ul>
</li>
</ul>
<h3 id="Summary"><a href="#Summary" class="headerlink" title="Summary"></a><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#summary"></a>Summary</h3><p>Rosetta can be used to predict the bound structure of two proteins starting from unbound structures. By the end of this tutorial, you should be able to understand:</p>
<ul>
<li>How to prepare structures for docking</li>
<li>How to locally dock two proteins</li>
<li>Hoe to refine an already docked structure</li>
<li>How to dock two proteins whose interface region is unknown</li>
<li>How to dock flexible proteins</li>
<li>How to dock a flexible peptide to a protein</li>
<li>How to dock symmetric proteins</li>
<li>How to analyse the best docked model</li>
</ul>
<h3 id="Docking-in-Rosetta"><a href="#Docking-in-Rosetta" class="headerlink" title="Docking in Rosetta"></a><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#docking-in-rosetta"></a>Docking in Rosetta</h3><p>Docking in Rosetta in a two stage protocol - the first stage, where aggresive sampling is done, is done in the<a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/full_atom_vs_centroid/fullatom_centroid">centroid mode</a>and the second stage where smaller movements take place is done in the full atom mode. It will internally connect the centers of the two chains with a so-called jump (see<a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/fold_tree/fold_tree">Fold Tree</a>). Along this jump the chains are being pulled together (slide into contact). The<a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Core_Concepts/Core_Concepts#monte-carlo-sampling">Monte Carlo</a>moves, which are selected randomly, are:</p>
<ul>
<li>Translations (in x,y or z direction)</li>
<li>Rotations (around x,y, z axis)</li>
</ul>
<p>By default, the docking protocol assumes a fixed backbone and only does, translation, rotation and sidechain packing. A detailed algorithm can be found<a target="_blank" rel="noopener" href="https://www.rosettacommons.org/docs/latest/application_documentation/docking/docking-protocol">here</a>.</p>
<h3 id="Navigating-to-the-Demos"><a href="#Navigating-to-the-Demos" class="headerlink" title="Navigating to the Demos"></a><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#navigating-to-the-demos"></a>Navigating to the Demos</h3><p>The demos are available at<code>$ROSETTA3/demos/tutorials/Protein-Protein-Docking</code>. All demo commands listed in this tutorial should be executed when in this directory. All the demos here use the<code>linuxgccrelease</code>binary. You may be required to change it to whatever is appropriate given your operating system and compiler.</p>
<p>Compare your output files to the ones present in<code>output_files/expected_output</code>.</p>
<h3 id="Preparing-Structures-for-Docking"><a href="#Preparing-Structures-for-Docking" class="headerlink" title="Preparing Structures for Docking"></a><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#preparing-structures-for-docking"></a>Preparing Structures for Docking</h3><p>This tutorial will introduce you the main steps required for predicting the bound structure of two interacting proteins starting from the unbound structures. For this example, we will dock Colicin-D with its inhibitor, IMM. You are provided with the two refined input files<code>COL_D.pdb</code>and<code>IMM_D.pdb</code>, and a native file<code>1v74.pdb</code>in the folder<code>input_files</code>.</p>
<p>To prepare structures for docking, be sure to refine them as described in<a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/input_and_output/input_and_output#controlling-input_preparing-a-structure-by-refinement">preparing inputs tutorial</a>.</p>
<h3 id="Local-Docking"><a href="#Local-Docking" class="headerlink" title="Local Docking"></a><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#local-docking"></a>Local Docking</h3><p>Rosetta is most accurate when docking locally.**In local docking, we assume that we have some information about the binding pockets of the two proteins.**First, we must manually place the two proteins (within ~10 Å) with the binding pockets roughly facing each other as shown in this figure:<img src="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/images/COL_IMM_unbound.png" alt="unbound"></p>
<p>We will pass the following options to indicate that i) chain B is being docked to chain A, and ii) we want to randomly perturb the ligand of the input strucure (chain B) by 3 Å translation and 8° rotation before the start of every individual simulation.</p>
<pre><code>-partners A_B
-dock_pert 3 8</code></pre>
<p>If you are docking two multi-chain proteins (say one with chains A &amp; B, and the other with chains L &amp; H), the option becomes<code>-partners LH_AB</code>. Make sure that the input PDB has all the chains of each protein listed together.</p>
<p>We will also compare the input with the bound structure 1v74.pdb by passing it as native.</p>
<blockquote>
<p>Ensure that the native has the same number of residues, same chain ordering and same chain IDs as the input structure. You may be required to use<code>-ignore_unrecognized_res</code>option if the native contains unusual ligands.</p>
</blockquote>
<p>Now to start docking, run:</p>
<pre><code>$&gt;$ROSETTA3/main/source/bin/docking_protocol.linuxgccrelease @flag_local_docking</code></pre>
<p>This should take ~1 minute to run and produce a structure file and a score file in<code>output_files</code>. Make sure you use<code>-nstruct 500</code>or more in production runs.</p>
<h3 id="Local-Refinement-of-Docked-Structures"><a href="#Local-Refinement-of-Docked-Structures" class="headerlink" title="Local Refinement of Docked Structures"></a><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#local-refinement-of-docked-structures"></a>Local Refinement of Docked Structures</h3><p>Sometimes docked proteins score very high in Rosetta owing to small clashes. Before refining with<a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Relax_Tutorial/Relax">`relax`</a>, we need to fix the interface. Since the protein is already docked, we want to avoid large movements of the protein, so we skip the first, centroid mode stage completely and only run the high-resolution full atom mode which does not move the backbone of the docked proteins. We also want to add the rotamers of the input sidechains to our library of rotamers. We replace the flags used for<a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#local-docking">local docking</a>by:</p>
<pre><code>-docking_local_refine
-use_input_sc</code></pre>
<p>We will use the new set of flags to refine the bound structure<code>1v74.pdb</code>by running:</p>
<pre><code>$&gt;$ROSETTA3/main/source/bin/docking_protocol.linuxgccrelease @flag_local_refine</code></pre>
<p>This should take ~45 seconds to run and produce a structure file and a score file in<code>output_files</code>. Verify that the input structure and the output structure have the same backbone. Now you can refine further with<code>relax</code>as described in<a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/input_and_output/input_and_output#controlling-input_preparing-a-structure-by-refinement">preparing inputs tutorial</a>.</p>
<h3 id="Global-Docking"><a href="#Global-Docking" class="headerlink" title="Global Docking"></a><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#global-docking"></a>Global Docking</h3><p>Only run global docking if absolutely no information is available about the binding sites in the protein. Global docking assumes a spherical general structure of the proteins and rotates the smaller protein (ligand) around the larger protein (receptor). It also randomizes the starting position of the unbound proteins in every run, so their position in the input structure does not matter as much.</p>
<blockquote>
<p>Global docking works best for small complexes (&lt;450 residues).</p>
</blockquote>
<p>To do global docking, we add the following three options to the options already present in global docking.</p>
<pre><code>-spin
-randomize1
-randomize2</code></pre>
<p>We will use the same input PDB as in<a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#local-docking">local docking</a>to demostrate the differences in the output. Run:</p>
<pre><code>$&gt;$ROSETTA3/main/source/bin/docking_protocol.linuxgccrelease @flag_global_docking</code></pre>
<p>This should take ~1 minute to run and produce a structure file and a score file in<code>output_files</code>. You will notice that unlike the local docking case, global docking produces a docked structure, which is significantly different from the native. Compare this to the local docking instance, which should be much similar to the native.</p>
<p>Due to the large space sampled, global docking requires a large number of runs to converge on a structure, typically 10,000-100,000.</p>
<blockquote>
<p>An alternative to running global docking is to find the likely binding pockets by using an FFT-based global docking programs like<a target="_blank" rel="noopener" href="https://cluspro.bu.edu/login.php">ClusPro</a>and then run multiple local docking runs.</p>
</blockquote>
<h3 id="Docking-Flexible-Proteins"><a href="#Docking-Flexible-Proteins" class="headerlink" title="Docking Flexible Proteins"></a><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#docking-flexible-proteins"></a>Docking Flexible Proteins</h3><p>As mentioned in the introduction, the docking protocol in Rosetta assumes a fixed backbone. If the backbone changes a lot between the unbound and the bound conformations, we dock conformational ensembles of the proteins. Instead of docking one ligand conformation to one receptor conformation, we constantly switch between the conformations in the respective ensembles while sampling in the first stage (in centroid mode). This enables the sampling of multiple backbones with Rosetta’s fixed backbone architecture.</p>
<p>These ensembles can be generated using unconstrained<a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Relax_Tutorial/Relax">`relax`</a>. For this tutorial, we will use small ensembles - 3 COL_D and 3 IMM_D conformations. Make sure that the number of residues and the chain naming (and order) in every conformation in the ensemble is the same as that of the corresponding partner in the input file. To pass the locations of the conformational ensemble of COL_D, we will pass an ensemble list which looks like:</p>
<pre><code>input_files/COL_D_ensemble/COL_D_0001.pdb
input_files/COL_D_ensemble/COL_D_0002.pdb
input_files/COL_D_ensemble/COL_D_0003.pdb</code></pre>
<p>A similar file is made for IMM_D ensembles. These files are then passed on to docking by adding the following options to the<a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#local-docking">local docking</a>:</p>
<pre><code>-ensemble1 COL_D_ensemblelist
-ensemble2 IMM_D_ensemblelist</code></pre>
<p><code>-ensemble1</code>must contain the list of conformations to the protein listed first in the input PDB. Even while ensemble docking one conformation against many conformations, you must make ensemble lists for both partners.</p>
<p>However, before we can dock the ensembles, we need to perform an additional step calling_prepacking_. Prepacking prepares the conformers in the ensemble for docking and scores them to inform the docking protocol to remove any energy contribution arising out of the inherantly different energies of the conformers. It uses the same flags except<code>-dock_pert</code>as the ensemble docking protocol. Run:</p>
<pre><code>$&gt;$ROSETTA3/main/source/bin/docking_prepack_protocol.linuxgccrelease @flag_ensemble_prepack</code></pre>
<p>This should run in ~30 seconds and produce prepacked PDBs in the same location as the original ensemble. It will also modify the ensemble lists with the location of the prepacked PDBs, normalized centroid scores and full atom scores.</p>
<pre><code>input_files/COL_D_ensemble/COL_D_0001.pdb.ppk
input_files/COL_D_ensemble/COL_D_0002.pdb.ppk
input_files/COL_D_ensemble/COL_D_0003.pdb.ppk
0.77058
0
1.00377
-93.3588
-94.2715
-93.9065</code></pre>
<p><strong>Remember that<code>COL_D_ensemblelist</code>and<code>IMM_D_ensemblelist</code>need to be regenerated if you run docking prepack again, otherwise you will get uninformative error meaasges.</strong></p>
<p>The docking protocol expects these prepacked conformations. This will also produce a score file and an output, but you can ignore both. Now run the docking protocol.</p>
<pre><code>$&gt;$ROSETTA3/main/source/bin/docking_protocol.linuxgccrelease @flag_ensemble_docking</code></pre>
<p>While it is running, you will see the following in your log file:</p>
<pre><code>...
protocols.docking.DockingLowRes: /// Ensemble 1: on                                                           ///
protocols.docking.DockingLowRes: /// Ensemble 2: on                                                           ///
...
protocols.docking.ConformerSwitchMover: Switching partner with conformer: 3
...</code></pre>
<p>This indicates that your ensembles have been loaded and your backbones are being switched.</p>
<p>This should take about 3 minutes to run and produce a score file and an output.</p>
<p>You will typically use larger ensembles (25 to 100) formations. This does significantly slow the protocol down. Also, due to a dilution of the sampling per backbone, you will have to increase the number of structures.<code>-nstruc 5000</code>or more is suggested.</p>
<h3 id="Symmetric-Docking"><a href="#Symmetric-Docking" class="headerlink" title="Symmetric Docking"></a><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#symmetric-docking"></a>Symmetric Docking</h3><p>Homomeric protein complexes are often symmetric. Rosetta provides the a means to use symmetry information during docking (and many other applications). Check out the<a target="_blank" rel="noopener" href="https://www.rosettacommons.org/docs/latest/rosetta_basics/structural_concepts/symmetry">symmetry documentation</a>and<a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Symmetry/Symmetry">tutorials</a>to know more.</p>
<h3 id="Analyzing-Docked-Structures"><a href="#Analyzing-Docked-Structures" class="headerlink" title="Analyzing Docked Structures"></a><a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking#analyzing-docked-structures"></a>Analyzing Docked Structures</h3><p>Analyzing docked structures can be tricky. Let’s look at a score file generated from ensemble docking:</p>
<pre><code>SEQUENCE: 
SCORE: total_score         rms        Fnat        I_sc        Irms cen_dock_ens_conf           cen_rms conf_num1 conf_num2 conf_score dock_ens_conf1 dock_ens_conf2 dslf_ca_dih dslf_cs_ang dslf_ss_dih dslf_ss_dst      fa_atr      fa_dun     fa_elec     fa_pair      fa_rep      fa_sol hbond_bb_sc hbond_lr_bb    hbond_sc hbond_sr_bb interchain_contact interchain_env interchain_pair interchain_vdw        st_rmsd description 
SCORE:    -202.141      13.143       0.061      -3.086       5.015            -1.004            14.583     1.000     1.000    -25.732        -93.359        -83.050       0.000       0.000       0.000       0.000    -326.487      12.899      -3.158      -8.467       8.982     139.078      -3.968      -4.390      -2.692     -13.937            -20.000        -23.791          -2.002          0.357          7.729 col_complex_ensemble_dock_0001</code></pre>
<p>The<code>total_score</code>may not indicate the best docked model as the primary contribution comes from the folding energies of the monomers. You should rely more on the interface score<code>I_sc</code>which represents the energy of the interactions across the interface.<code>I_sc</code>scores are typically much smaller than total scores (typically in the range of -2 to -10 REU). Of course they depend on the size and geometry of the interface; so they are difficult to compare across different proteins.</p>
<p>If you provide a native structure, you get comparative data like the fraction of native contacts (<code>Fnat</code>), the CαRMSD of the ligand (<code>rms</code>) and the RMSD of the interface atoms (<code>Irms</code>). If you run ensemble docking, you will also get to know which of the conformers was finally selected for protein 1(<code>conf1</code>) and protein 2(<code>conf2</code>).</p>
<p>You can repeat the run with_-nstruct 20000_and outputting silent files. Then you could extract the best model and create a score vs. rmsd plot.</p>
<p>Look at<a target="_blank" rel="noopener" href="https://www.rosettacommons.org/demos/latest/tutorials/analysis/Analysis">Analysis</a>for further information on how to analyze your results.</p>
<h2 id="Protein-Protein-Docking-From-Interaction-to-Interactome"><a href="#Protein-Protein-Docking-From-Interaction-to-Interactome" class="headerlink" title="Protein-Protein Docking: From Interaction to Interactome"></a>Protein-Protein Docking: From Interaction to Interactome</h2><ul>
<li><a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/">Protein-Protein Docking: From Interaction to Interactome</a></li>
</ul>
<p><a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Vakser%20I%5BAuthor%5D&cauthor=true&cauthor_uid=25418159">Ilya A. Vakser</a>1,∗</p>
<p><a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#">Author information</a><a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#">Article notes</a><a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#">Copyright and License information</a><a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/about/disclaimer/">Disclaimer</a></p>
<p>This article has been<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/citedby/">cited by</a>other articles in PMC.</p>
<h3 id="Abstract"><a href="#Abstract" class="headerlink" title="Abstract"></a>Abstract</h3><p>The protein-protein docking problem is one of the focal points of activity in computational biophysics and structural biology. The three-dimensional structure of a protein-protein complex, generally, is more difficult to determine experimentally than the structure of an individual protein. Adequate computational techniques to model protein interactions are important because of the growing number of known protein structures, particularly in the context of structural genomics. Docking offers tools for fundamental studies of protein interactions and provides a structural basis for drug design. Protein-protein docking is the prediction of the structure of the complex, given the structures of the individual proteins. In the heart of the docking methodology is the notion of steric and physicochemical complementarity at the protein-protein interface. Originally, mostly high-resolution, experimentally determined (primarily by x-ray crystallography) protein structures were considered for docking. However, more recently, the focus has been shifting toward lower-resolution modeled structures. Docking approaches have to deal with the conformational changes between unbound and bound structures, as well as the inaccuracies of the interacting modeled structures, often in a high-throughput mode needed for modeling of large networks of protein interactions. The growing number of docking developers is engaged in the community-wide assessments of predictive methodologies. The development of more powerful and adequate docking approaches is facilitated by rapidly expanding information and data resources, growing computational capabilities, and a deeper understanding of the fundamental principles of protein interactions.</p>
<h3 id="Introduction"><a href="#Introduction" class="headerlink" title="Introduction"></a>Introduction</h3><p>Proteins recognize each other, typically in a crowded environment, and bind in a highly specific fashion. This process involves diffusion through a densely populated milieu of different proteins and other biomolecular structures, and binding (docking) to their designated protein partner in a structurally unique and precise way. Given the large size of these macromolecules, the great structural diversity, and the high density of the biomolecular environment, this constantly reoccurring process is truly remarkable.</p>
<p>Protein docking—prediction of the structure of a protein-protein complex from the structures of the individual proteins—has evolved significantly since its early days, by incorporating more adequate energy functions and powerful techniques to sample the energy landscapes, and by taking advantage of the rapidly growing body of knowledge on protein structures and interactions. Our current knowledge of protein interaction principles is far greater than before, helping design better docking approaches. The spectacular progress in computing hardware has obviously played a major role as well, opening new ways of thinking about modeling of protein interactions, and often allowing implementation of old but unfeasible at the time ideas. Still, some basic docking principles remain surprisingly unchanged, due to their true nature. Steric and physicochemical complementarity is still the foundation of most docking approaches, as it was in the beginning of the docking field.</p>
<h3 id="Beginnings"><a href="#Beginnings" class="headerlink" title="Beginnings"></a>Beginnings</h3><p>The origins of the protein docking field can be traced to the earlier days of molecular modeling. Back than in the 70s, the force fields were simpler, the minds not clouded by the power of computers, and the goals clearer (e.g., to fold proteins from the sequence based on the physical forces alone).</p>
<p>The first docking approaches dealt not with protein-protein complexes per se, but rather with protein interactions with other ligands at predetermined binding sites (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib1">1–4</a>). Despite the early times in molecular modeling, the approaches were remarkably sophisticated, implementing flexible docking, taking into account the internal coordinates of not only the ligand, but in some cases also the receptor—a challenging task largely avoided even in today’s community, with all its computing power and the history of methodology development. Protein-protein docking approaches followed shortly, implementing the global search for the docking pose in rigid-body approximation (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib5">5,6</a>).</p>
<p>A significant uptake in the development of protein docking techniques (that continues to this day) occurred in the early 90s. Among most influential and consequential approaches put forward at that time were those based on efficient sampling techniques borrowed from computer science (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib7">7,8</a>). The docking approach based on correlation by fast Fourier transform, commonly known as FFT docking (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib7">7</a>), developed back then by an interdisciplinary group of biologists, chemists, physicists, and computer scientists, has become arguably the most popular protein docking algorithm, implemented over the years in many groups (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib9">9</a>). The reason for its popularity is that, as opposed to employing a particular search strategy that may or may not lead to the global minimum (native complex), it allows computationally feasible exhaustive search of the full six-dimensional docking space. Although the space, for the purpose of the exhaustive sampling, has to be discretized, the atomic-size grid steps still provide the “comprehensive” solution to the rigid-body docking problem.</p>
<h3 id="Docking-Foundations"><a href="#Docking-Foundations" class="headerlink" title="Docking Foundations"></a>Docking Foundations</h3><p>The protein-protein docking problem can be formulated as the prediction of the structure of the complex, given the structures of the individual proteins. In the general case, no information other than the structure of these individual proteins is available.<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/figure/fig1/">Fig. 1</a>shows major steps in the proper development of a docking approach, involving scan (<em>global search</em>) of the docking space using simplified/coarse-grained protein representations, followed by a refinement to a higher resolution (<em>local search</em>), and systematic evaluation on comprehensive benchmark sets and blind community-wide assessments.</p>
<p><a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/figure/fig1/"><img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/bin/gr1.jpg" alt="An external file that holds a picture, illustration, etc.&lt;br&gt;Object name is gr1.jpg" title="An external file that holds a picture, illustration, etc.&lt;br&gt;Object name is gr1.jpg"></a></p>
<p><a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/figure/fig1/">Figure 1</a></p>
<p>The general scheme of protein docking methodology development. The scan (global search for complementarity) is performed on a simplified/coarse-grained representation of proteins (e.g., digitized on a grid, or discretized/approximated in other ways). The scan can be explicit (<em>free</em>) or based on similarity to known cocrystallized complexes (<em>comparative</em>). The refinement is supposed to bring back all or some structural resolution lost in the coarse-graining (e.g., by gradual transition from s smoothed intermolecular energy landscape to the one based on a physical force field, while tracking the position of the global minimum). The validity of the approach is determined by systematic benchmarking on representative sets of structures. To see this figure in color, go online.</p>
<p>In the heart of the docking methodology is the notion of steric complementarity at the protein-protein interface. These interfaces are indeed tightly packed, as observed in cocrystallized complexes in the Protein Data Bank (PDB). The steric complementarity has been the major driving force in the development of docking approaches, often with the addition of physicochemical complementarity—hydrophobicity, electrostatics, etc. (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib10">10,11</a>), and statistics-based propensities (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib12">12,13</a>). The structural complementarity has been observed at different resolutions, from the atomic to ultralow (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib14">14–18</a>).</p>
<p>The conformations of the protein within the complex (bound structure) and the one outside the complex (unbound structure) are different. In some cases, this difference can be neglected or approximated (rigid body docking), or taken into account through conformational search (flexible docking). The rigid body docking involves six degrees of freedom of the two rigid bodies system (e.g., three translations and three rotations in the Cartesian coordinates). The flexible docking involves a much greater number of coordinates, given the conformational search in the internal coordinates of the proteins. However, this search typically does not involve solving the elusive ‘protein folding problem’, but rather can be restricted to a much more tractable unbound-to-bound conformational transition.</p>
<p>Originally, mostly the high-resolution, experimentally determined (primarily by x-ray crystallography) structures were considered. However, more recently, the focus has been shifting toward lower resolution modeled structures. The correct prediction of the complex does not mean the exact native (cocrystallized) complex per se, which is mathematically/computationally impossible, but rather a near-native approximation.</p>
<p>The general question is: what is the necessary level of structural accuracy for predicted protein complexes? In protein-protein interactions, many experimental and theoretical studies require simple knowledge of the residues at the interfaces (e.g., for further experimental analysis) and have no use for atomic resolution structural details of the complex (specific atom-atom, or even residue-residue contacts across the interface). For the interface (binding site) prediction, the high-resolution protein structures, generally, are not needed. That has been extensively shown by systematic studies over a number of years (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib15">15</a>). Still, a high-resolution structure of the complex is required for a number of studies (e.g., for estimation of the binding affinity, certain approaches to inhibition of protein interactions, and such).</p>
<h3 id="Bound-and-Unbound-Docking"><a href="#Bound-and-Unbound-Docking" class="headerlink" title="Bound and Unbound Docking"></a>Bound and Unbound Docking</h3><p>The bound docking problem, where the proteins within a cocrystallized complex are separated and redocked by a computational procedure, is a useful tool for the development of new docking approaches, but obviously has no practical value for biology. Docking becomes useful when it is able to predict complexes from the separately determined protein structures (unbound docking), thus becoming a tool for generating new knowledge.</p>
<p>Bound docking is the easiest docking case, because by definition it does not involve conformational change. Thus, the structures match ideally at the interface and the rigid body approach is the only tool required to deliver the correct solution. The bound docking problem has been considered solved for a number of years, in the sense that the existing docking approaches reliably and routinely deliver the near-native structures of the complex among the top predictions.</p>
<p>The approaches to the unbound docking problem have to deal with the conformational difference between the unbound and the bound structures. The change from the unbound to the bound conformation is the basis of the protein’s function in its interactions with other proteins. The intermolecular energy landscapes are characterized by conformational properties of the interacting proteins (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib19">19–21</a>). One basic direction in the docking methodology involves coarse-graining (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib22">22,23</a>). At lower levels of structural resolution, the difference between unbound and bound conformations is less significant (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib24">24,25</a>), and ultimately disappears at ultralow (but still structurally meaningful) resolution (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib15">15,24,26</a>). Such approaches allow prediction of the gross features of the complex, due to the large structural recognition factors, and the related funnel in the intermolecular energy landscape (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib27">27–29</a>). However, prediction of the higher resolution structural details of interface requires modeling of the structural flexibility, at least at the interface regions.</p>
<p>Still, the majority of protein complexes in the nonredundant benchmark sets have small C_α_root mean-square deviation (RMSD) between bound and unbound structures. Indeed, 71% of thedockgroundset (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib30">30,31</a>) has RMSD between superimposed unbound and bound proteins &lt;2 Å for 71% of the complexes (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib31">31</a>). The benchmark set from Weng’s group (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib32">32</a>) has unbound/bound interface C_α_RMSD (between C_α_atoms of the interface residues only) &lt;2.2 Å for 86% of complexes. In a number of cases, when the RMSD is large, the conformational change upon binding is a domain shift. The domains themselves do not undergo a significant conformational change. Thus, this docking still can be addressed by a rigid body approach (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib33">33</a>).</p>
<p>Because most docking cases can be resolved by accounting for the flexibility of the surface side chains, the statistics of side-chain conformational changes is important. The results of a systematic large-scale study indicate that short and long side chains have different propensities for the conformational changes (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib34">34</a>). Long side chains with three or more dihedral angles are often subject to large conformational transition. Shorter residues with one or two dihedral angles typically undergo local conformational changes not leading to a conformational transition. Most side chains undergo larger changes in the dihedral angle most distant from the backbone. The binding increases both polar and nonpolar interface areas. However, the increase of the nonpolar area is larger, suggesting that the protein association perturbs the unbound interfaces to increase the hydrophobic contribution to the binding free energy (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib34">34</a>). Analysis of ensembles of bound and unbound conformations points to conformational selection as the binding mechanism for proteins. The bound and the unbound spectra of conformers also significantly overlap (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib35">35</a>). An elastic network model, accounting for the mass distribution, was used to compare the binding site residues fluctuations with other surface residues, showing that, on average, the interface is more rigid (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib36">36</a>).</p>
<p>Discretization of the conformational space into rotameric states is useful for the sampling of the conformational space in flexible docking (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib37">37,38</a>). Such rotameric libraries for the surface side chains in bound and unbound proteins were generated and used to calculate the probabilities of the rotamer transitions upon binding (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib38">38</a>). The stability of amino acids was quantified based on the transition maps. Most side chains changed conformation within the same rotamer or moved to an adjacent rotamer. The highest percentage of the transitions was observed primarily between the two most occupied rotamers (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib38">38</a>).</p>
<h3 id="Docking-of-Models"><a href="#Docking-of-Models" class="headerlink" title="Docking of Models"></a>Docking of Models</h3><p>The docking problem is further complicated if the interacting proteins are models rather than the experimentally determined structures. The errors in such ‘double modeling’ (first of the individual proteins, then of the complex) accumulate, which presents a greater challenge, especially in higher resolution docking (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/figure/fig2/">Fig. 2</a>). Thus, the use of approaches to dock these structures should be assessed by thorough benchmarking, specifically designed for protein models (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib39">39</a>). To be credible, such benchmarking has to be based on carefully curated sets of structures with levels of distortion typical for modeled proteins. A suite of models was generated for the benchmark set of the x-ray structures from thedockgroundresource (<a target="_blank" rel="noopener" href="http://dockground.bioinformatics.ku.edu/">http://dockground.bioinformatics.ku.edu</a>) by a combination of homology modeling and the nudged elastic band method (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib40">40</a>). For each monomer, six models were generated with predefined C_α_RMSD from the native structure (1, 2, …, 6 Å). The sets and the accompanying data provide a comprehensive resource for the development of docking methodology for modeled proteins (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib41">41</a>). A new approach, implementing only actual modeling of new protein structures as in the real case scenario, was used by the same group of authors to generate a larger set of models (165 complexes, with full arrays of models for each).</p>
<p><a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/figure/fig2/"><img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/bin/gr2.jpg" alt="An external file that holds a picture, illustration, etc.&lt;br&gt;Object name is gr2.jpg" title="An external file that holds a picture, illustration, etc.&lt;br&gt;Object name is gr2.jpg"></a></p>
<p><a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/figure/fig2/">Figure 2</a></p>
<p>Structures with the increasing level of inaccuracy. The model structures (<em>cyan</em>) are overlapped with the x-ray structure (<em>light brown</em>). To see this figure in color, go online.</p>
<h3 id="Physics-Versus-Knowledge-Based-Docking"><a href="#Physics-Versus-Knowledge-Based-Docking" class="headerlink" title="Physics Versus Knowledge-Based Docking"></a>Physics Versus Knowledge-Based Docking</h3><p>Solving the equations of motion for two proteins in arbitrary relative orientation, using atomic resolution force fields, does not dock them in the correct configuration. The reason is the extreme complexity of the energy landscape of the system—its span in multidimensional space of its coordinates and the multiplicity of the energy minima (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib24">24,42</a>)—all compounded by the approximate nature of the landscape, with error bars that are often larger than the relative depth of the energy basins.</p>
<p>Still, if one is interested just in the location of the global minimum of the free energy, corresponding to the native structure of the complex, with no regard to the binding pathways, there are nonphysical sampling protocols that efficiently search the landscape and deliver the solution. These protocols treat the problem as global optimization, and find the global minimum through various nonlinear programming techniques, including the most trivial (and effective) one—systematics search (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib7">7</a>). The main reason for their success, given the complexity of the landscape, is that the global minimum is significantly different from the local minima. It is not just deeper, as it is supposed to be by definition, but deeper by a significant margin (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib43">43</a>), and has a number of other distinguishing characteristics, such as size and ruggedness (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib28">28,42</a>). Thus, the unavoidable common approximations of the energy landscapes, although distorting the local minima hierarchy, are not approximate enough to eliminate the difference between the global and the local minima (or at least to remove the actual global minimum from the top candidates). Although such minimization protocols are nonphysical, because the landscape represents physics-based energy (even in its simplest form of steric complementarity, which is none other than the minimum of van der Waals energy) such approaches still pass as physical (often among nonphysicists, and physicists who know biology). Still, in many such approaches, the only physical concept is the trivial steric complementarity, and the rest are techniques borrowed from computer science and other engineering disciplines (pattern recognition, optimization, machine learning, etc.). Such approaches have been dominant in the protein docking field since its inception.</p>
<p>The whole notion of physics though goes out the window altogether with the recent docking approaches based solely on similarity to the existing experimentally determined complexes/templates. If two similar pairs of proteins generally bind in a similar way, and one of them is cocrystallized, for the other pair there may be no need to sample the extremely complex intermolecular landscape—one can simply get straight to the presumed global minimum (the correct structure of the complex) by assuming similarity to the experimentally determined complex.</p>
<p>Structural modeling by similarity (comparative modeling) of individual proteins has been around for a long time, since the establishment of the correlation between sequence and structure similarity in the 80s (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib44">44</a>). Such similarity suggested that if the sequence of protein A, the structure of which is to be modeled, is similar to the sequence of protein A′, the structure of which is known, one can put protein A in the same fold as protein A′. That provided a dramatic improvement in terms of prediction reliability over the proverbial ‘protein folding problem’ where the protein structure is supposed to be modeled based on the amino acid sequence alone. The atomic resolution prediction of the protein structure was reduced to the repacking of the side chains, and tweaking of the backbone (often involving flexible loops)—a difficult, but quite tractable task, incomparable with the ultimate complexity of the structure prediction from the sequence alone. The critical aspect of such approach is the availability of the experimentally determined (largely by x-ray crystallography) templates. One can date the emergence of the comparative modeling to the expansion of the PDB that at the time had become large enough to provide a meaningful pool of templates. Currently, with the rapid growth of PDB, the template-based modeling of individual proteins is a dominant approach to the prediction of protein structures (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib45">45</a>). The modeling of the folding-related physical processes is still a big challenge, and may well remain so in the future. However, the on-going expansion of PDB will arguably keep further simplifying the nonphysical prediction shortcut to the equilibrium structure, given the limited structural scope of the protein universe (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib46">46</a>), which causes the reduction of the pool of proteins with the new fold (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib45">45</a>).</p>
<p>In protein-protein docking, the similarity between proteins in complexes can be assessed through comparison/alignment of sequences (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib47">47–49</a>), sequences and structures (threading) (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib50">50–52</a>), or just the structures (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib52">52–58</a>) because the structures of the protein to be docked are assumed to be known by the very definition of docking. However, the protein docking field, as opposed to the prediction of individual proteins, largely has not been taking advantage of the template-based modeling. One reason is that protein-protein docking is younger and thus less advanced (the protein docking community is also significantly smaller than the one in modeling of individual proteins, based on the number of participants in the community-wide prediction assessments (∼200 in CASP vs. ∼40 in CAPRI) and prediction targets (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4213718/#bib45">45,59</a>).</p>
<p>Another reason has been the relative success of the traditional template free (ab initio) docking, as opposed to the ab initio modeling of the individual proteins. The rigid-body docking (six degrees of freedom) is a meaningful, working approximation for many complexes, whereas any practical approximation in protein folding involves the conformational search space of far greater dimensionality.</p>
<p>Still, the main reason for the almost comple</p>

      

      
    </div>
    <div class="article-info article-info-index">
      
      
	<div class="article-tag tagcloud">
		<i class="icon-price-tags icon"></i>
		<ul class="article-tag-list">
			 
        		<li class="article-tag-list-item">
        			<a href="javascript:void(0)" class="js-tag article-tag-list-link color4">Protein-Protein Docking</a>
        		</li>
      		 
        		<li class="article-tag-list-item">
        			<a href="javascript:void(0)" class="js-tag article-tag-list-link color4">PPI</a>
        		</li>
      		
		</ul>
	</div>

      
	<div class="article-category tagcloud">
		<i class="icon-book icon"></i>
		<ul class="article-tag-list">
			 
        		<li class="article-tag-list-item">
        			<a href="/categories/Log//" class="article-tag-list-link color4">Log</a>
        		</li>
      		
		</ul>
	</div>


      
        <p class="article-more-link">
          <a class="article-more-a" href="/2021/03/30/20210330/">Expand all items >></a>
        </p>
      

      
      <div class="clearfix"></div>
    </div>
  </div>
</article>

<aside class="wrap-side-operation">
    <div class="mod-side-operation">
        
        <div class="jump-container" id="js-jump-container" style="display:none;">
            <a href="javascript:void(0)" class="mod-side-operation__jump-to-top">
                <i class="icon-font icon-back"></i>
            </a>
            <div id="js-jump-plan-container" class="jump-plan-container" style="top: -11px;">
                <i class="icon-font icon-plane jump-plane"></i>
            </div>
        </div>
        
        
    </div>
</aside>




  
  
    <nav id="page-nav">
      <a class="extend prev" rel="prev" href="/page/7/">&amp;laquo; Prev</a><a class="page-number" href="/">1</a><span class="space">&hellip;</span><a class="page-number" href="/page/6/">6</a><a class="page-number" href="/page/7/">7</a><span class="page-number current">8</span><a class="page-number" href="/page/9/">9</a><a class="page-number" href="/page/10/">10</a><span class="space">&hellip;</span><a class="page-number" href="/page/17/">17</a><a class="extend next" rel="next" href="/page/9/">Next &amp;raquo;</a>
    </nav>
  


          </div>
        </div>
      </div>
      <footer id="footer">
  <div class="outer">
    <div id="footer-info">
    	<div class="footer-left">
    		&copy; 2022 Hongtu Liu
    	</div>
      	<div class="footer-right">
      		<a href="http://hexo.io/" target="_blank">Hexo</a>  Theme <a href="https://github.com/litten/hexo-theme-yilia" target="_blank">Yilia</a> by Litten
      	</div>
    </div>
  </div>
</footer>



<script src='https://unpkg.com/mermaid@7.1.2/dist/mermaid.min.js'></script>

<script>
    if (window.mermaid) {
        mermaid.initialize({"startOnload":true});
    }
</script>


    </div>
    <script>
	var yiliaConfig = {
		mathjax: false,
		isHome: true,
		isPost: false,
		isArchive: false,
		isTag: false,
		isCategory: false,
		open_in_new: false,
		toc_hide_index: true,
		root: "/",
		innerArchive: true,
		showTags: false
	}
</script>

<script>!function(t){function n(e){if(r[e])return r[e].exports;var i=r[e]={exports:{},id:e,loaded:!1};return t[e].call(i.exports,i,i.exports,n),i.loaded=!0,i.exports}var r={};n.m=t,n.c=r,n.p="./",n(0)}([function(t,n,r){r(195),t.exports=r(191)},function(t,n,r){var e=r(3),i=r(52),o=r(27),u=r(28),c=r(53),f="prototype",a=function(t,n,r){var s,l,h,v,p=t&a.F,d=t&a.G,y=t&a.S,g=t&a.P,b=t&a.B,m=d?e:y?e[n]||(e[n]={}):(e[n]||{})[f],x=d?i:i[n]||(i[n]={}),w=x[f]||(x[f]={});d&&(r=n);for(s in r)l=!p&&m&&void 0!==m[s],h=(l?m:r)[s],v=b&&l?c(h,e):g&&"function"==typeof h?c(Function.call,h):h,m&&u(m,s,h,t&a.U),x[s]!=h&&o(x,s,v),g&&w[s]!=h&&(w[s]=h)};e.core=i,a.F=1,a.G=2,a.S=4,a.P=8,a.B=16,a.W=32,a.U=64,a.R=128,t.exports=a},function(t,n,r){var e=r(6);t.exports=function(t){if(!e(t))throw TypeError(t+" is not an object!");return t}},function(t,n){var r=t.exports="undefined"!=typeof window&&window.Math==Math?window:"undefined"!=typeof self&&self.Math==Math?self:Function("return this")();"number"==typeof __g&&(__g=r)},function(t,n){t.exports=function(t){try{return!!t()}catch(t){return!0}}},function(t,n){var r=t.exports="undefined"!=typeof window&&window.Math==Math?window:"undefined"!=typeof self&&self.Math==Math?self:Function("return this")();"number"==typeof __g&&(__g=r)},function(t,n){t.exports=function(t){return"object"==typeof t?null!==t:"function"==typeof t}},function(t,n,r){var e=r(126)("wks"),i=r(76),o=r(3).Symbol,u="function"==typeof o;(t.exports=function(t){return e[t]||(e[t]=u&&o[t]||(u?o:i)("Symbol."+t))}).store=e},function(t,n){var r={}.hasOwnProperty;t.exports=function(t,n){return r.call(t,n)}},function(t,n,r){var e=r(94),i=r(33);t.exports=function(t){return e(i(t))}},function(t,n,r){t.exports=!r(4)(function(){return 7!=Object.defineProperty({},"a",{get:function(){return 7}}).a})},function(t,n,r){var e=r(2),i=r(167),o=r(50),u=Object.defineProperty;n.f=r(10)?Object.defineProperty:function(t,n,r){if(e(t),n=o(n,!0),e(r),i)try{return u(t,n,r)}catch(t){}if("get"in r||"set"in r)throw TypeError("Accessors not supported!");return"value"in r&&(t[n]=r.value),t}},function(t,n,r){t.exports=!r(18)(function(){return 7!=Object.defineProperty({},"a",{get:function(){return 7}}).a})},function(t,n,r){var e=r(14),i=r(22);t.exports=r(12)?function(t,n,r){return e.f(t,n,i(1,r))}:function(t,n,r){return t[n]=r,t}},function(t,n,r){var e=r(20),i=r(58),o=r(42),u=Object.defineProperty;n.f=r(12)?Object.defineProperty:function(t,n,r){if(e(t),n=o(n,!0),e(r),i)try{return u(t,n,r)}catch(t){}if("get"in r||"set"in r)throw TypeError("Accessors not supported!");return"value"in r&&(t[n]=r.value),t}},function(t,n,r){var e=r(40)("wks"),i=r(23),o=r(5).Symbol,u="function"==typeof o;(t.exports=function(t){return e[t]||(e[t]=u&&o[t]||(u?o:i)("Symbol."+t))}).store=e},function(t,n,r){var e=r(67),i=Math.min;t.exports=function(t){return t>0?i(e(t),9007199254740991):0}},function(t,n,r){var e=r(46);t.exports=function(t){return Object(e(t))}},function(t,n){t.exports=function(t){try{return!!t()}catch(t){return!0}}},function(t,n,r){var e=r(63),i=r(34);t.exports=Object.keys||function(t){return e(t,i)}},function(t,n,r){var e=r(21);t.exports=function(t){if(!e(t))throw TypeError(t+" is not an object!");return t}},function(t,n){t.exports=function(t){return"object"==typeof t?null!==t:"function"==typeof t}},function(t,n){t.exports=function(t,n){return{enumerable:!(1&t),configurable:!(2&t),writable:!(4&t),value:n}}},function(t,n){var r=0,e=Math.random();t.exports=function(t){return"Symbol(".concat(void 0===t?"":t,")_",(++r+e).toString(36))}},function(t,n){var r={}.hasOwnProperty;t.exports=function(t,n){return r.call(t,n)}},function(t,n){var r=t.exports={version:"2.4.0"};"number"==typeof __e&&(__e=r)},function(t,n){t.exports=function(t){if("function"!=typeof t)throw TypeError(t+" is not a function!");return t}},function(t,n,r){var e=r(11),i=r(66);t.exports=r(10)?function(t,n,r){return e.f(t,n,i(1,r))}:function(t,n,r){return t[n]=r,t}},function(t,n,r){var e=r(3),i=r(27),o=r(24),u=r(76)("src"),c="toString",f=Function[c],a=(""+f).split(c);r(52).inspectSource=function(t){return f.call(t)},(t.exports=function(t,n,r,c){var f="function"==typeof r;f&&(o(r,"name")||i(r,"name",n)),t[n]!==r&&(f&&(o(r,u)||i(r,u,t[n]?""+t[n]:a.join(String(n)))),t===e?t[n]=r:c?t[n]?t[n]=r:i(t,n,r):(delete t[n],i(t,n,r)))})(Function.prototype,c,function(){return"function"==typeof this&&this[u]||f.call(this)})},function(t,n,r){var e=r(1),i=r(4),o=r(46),u=function(t,n,r,e){var i=String(o(t)),u="<"+n;return""!==r&&(u+=" "+r+'="'+String(e).replace(/"/g,"&quot;")+'"'),u+">"+i+"</"+n+">"};t.exports=function(t,n){var r={};r[t]=n(u),e(e.P+e.F*i(function(){var n=""[t]('"');return n!==n.toLowerCase()||n.split('"').length>3}),"String",r)}},function(t,n,r){var e=r(115),i=r(46);t.exports=function(t){return e(i(t))}},function(t,n,r){var e=r(116),i=r(66),o=r(30),u=r(50),c=r(24),f=r(167),a=Object.getOwnPropertyDescriptor;n.f=r(10)?a:function(t,n){if(t=o(t),n=u(n,!0),f)try{return a(t,n)}catch(t){}if(c(t,n))return i(!e.f.call(t,n),t[n])}},function(t,n,r){var e=r(24),i=r(17),o=r(145)("IE_PROTO"),u=Object.prototype;t.exports=Object.getPrototypeOf||function(t){return t=i(t),e(t,o)?t[o]:"function"==typeof t.constructor&&t instanceof t.constructor?t.constructor.prototype:t instanceof Object?u:null}},function(t,n){t.exports=function(t){if(void 0==t)throw TypeError("Can't call method on  "+t);return t}},function(t,n){t.exports="constructor,hasOwnProperty,isPrototypeOf,propertyIsEnumerable,toLocaleString,toString,valueOf".split(",")},function(t,n){t.exports={}},function(t,n){t.exports=!0},function(t,n){n.f={}.propertyIsEnumerable},function(t,n,r){var e=r(14).f,i=r(8),o=r(15)("toStringTag");t.exports=function(t,n,r){t&&!i(t=r?t:t.prototype,o)&&e(t,o,{configurable:!0,value:n})}},function(t,n,r){var e=r(40)("keys"),i=r(23);t.exports=function(t){return e[t]||(e[t]=i(t))}},function(t,n,r){var e=r(5),i="__core-js_shared__",o=e[i]||(e[i]={});t.exports=function(t){return o[t]||(o[t]={})}},function(t,n){var r=Math.ceil,e=Math.floor;t.exports=function(t){return isNaN(t=+t)?0:(t>0?e:r)(t)}},function(t,n,r){var e=r(21);t.exports=function(t,n){if(!e(t))return t;var r,i;if(n&&"function"==typeof(r=t.toString)&&!e(i=r.call(t)))return i;if("function"==typeof(r=t.valueOf)&&!e(i=r.call(t)))return i;if(!n&&"function"==typeof(r=t.toString)&&!e(i=r.call(t)))return i;throw TypeError("Can't convert object to primitive value")}},function(t,n,r){var e=r(5),i=r(25),o=r(36),u=r(44),c=r(14).f;t.exports=function(t){var n=i.Symbol||(i.Symbol=o?{}:e.Symbol||{});"_"==t.charAt(0)||t in n||c(n,t,{value:u.f(t)})}},function(t,n,r){n.f=r(15)},function(t,n){var r={}.toString;t.exports=function(t){return r.call(t).slice(8,-1)}},function(t,n){t.exports=function(t){if(void 0==t)throw TypeError("Can't call method on  "+t);return t}},function(t,n,r){var e=r(4);t.exports=function(t,n){return!!t&&e(function(){n?t.call(null,function(){},1):t.call(null)})}},function(t,n,r){var e=r(53),i=r(115),o=r(17),u=r(16),c=r(203);t.exports=function(t,n){var r=1==t,f=2==t,a=3==t,s=4==t,l=6==t,h=5==t||l,v=n||c;return function(n,c,p){for(var d,y,g=o(n),b=i(g),m=e(c,p,3),x=u(b.length),w=0,S=r?v(n,x):f?v(n,0):void 0;x>w;w++)if((h||w in b)&&(d=b[w],y=m(d,w,g),t))if(r)S[w]=y;else if(y)switch(t){case 3:return!0;case 5:return d;case 6:return w;case 2:S.push(d)}else if(s)return!1;return l?-1:a||s?s:S}}},function(t,n,r){var e=r(1),i=r(52),o=r(4);t.exports=function(t,n){var r=(i.Object||{})[t]||Object[t],u={};u[t]=n(r),e(e.S+e.F*o(function(){r(1)}),"Object",u)}},function(t,n,r){var e=r(6);t.exports=function(t,n){if(!e(t))return t;var r,i;if(n&&"function"==typeof(r=t.toString)&&!e(i=r.call(t)))return i;if("function"==typeof(r=t.valueOf)&&!e(i=r.call(t)))return i;if(!n&&"function"==typeof(r=t.toString)&&!e(i=r.call(t)))return i;throw TypeError("Can't convert object to primitive value")}},function(t,n,r){var e=r(5),i=r(25),o=r(91),u=r(13),c="prototype",f=function(t,n,r){var a,s,l,h=t&f.F,v=t&f.G,p=t&f.S,d=t&f.P,y=t&f.B,g=t&f.W,b=v?i:i[n]||(i[n]={}),m=b[c],x=v?e:p?e[n]:(e[n]||{})[c];v&&(r=n);for(a in r)(s=!h&&x&&void 0!==x[a])&&a in b||(l=s?x[a]:r[a],b[a]=v&&"function"!=typeof x[a]?r[a]:y&&s?o(l,e):g&&x[a]==l?function(t){var n=function(n,r,e){if(this instanceof t){switch(arguments.length){case 0:return new t;case 1:return new t(n);case 2:return new t(n,r)}return new t(n,r,e)}return t.apply(this,arguments)};return n[c]=t[c],n}(l):d&&"function"==typeof l?o(Function.call,l):l,d&&((b.virtual||(b.virtual={}))[a]=l,t&f.R&&m&&!m[a]&&u(m,a,l)))};f.F=1,f.G=2,f.S=4,f.P=8,f.B=16,f.W=32,f.U=64,f.R=128,t.exports=f},function(t,n){var r=t.exports={version:"2.4.0"};"number"==typeof __e&&(__e=r)},function(t,n,r){var e=r(26);t.exports=function(t,n,r){if(e(t),void 0===n)return t;switch(r){case 1:return function(r){return t.call(n,r)};case 2:return function(r,e){return t.call(n,r,e)};case 3:return function(r,e,i){return t.call(n,r,e,i)}}return function(){return t.apply(n,arguments)}}},function(t,n,r){var e=r(183),i=r(1),o=r(126)("metadata"),u=o.store||(o.store=new(r(186))),c=function(t,n,r){var i=u.get(t);if(!i){if(!r)return;u.set(t,i=new e)}var o=i.get(n);if(!o){if(!r)return;i.set(n,o=new e)}return o},f=function(t,n,r){var e=c(n,r,!1);return void 0!==e&&e.has(t)},a=function(t,n,r){var e=c(n,r,!1);return void 0===e?void 0:e.get(t)},s=function(t,n,r,e){c(r,e,!0).set(t,n)},l=function(t,n){var r=c(t,n,!1),e=[];return r&&r.forEach(function(t,n){e.push(n)}),e},h=function(t){return void 0===t||"symbol"==typeof t?t:String(t)},v=function(t){i(i.S,"Reflect",t)};t.exports={store:u,map:c,has:f,get:a,set:s,keys:l,key:h,exp:v}},function(t,n,r){"use strict";if(r(10)){var e=r(69),i=r(3),o=r(4),u=r(1),c=r(127),f=r(152),a=r(53),s=r(68),l=r(66),h=r(27),v=r(73),p=r(67),d=r(16),y=r(75),g=r(50),b=r(24),m=r(180),x=r(114),w=r(6),S=r(17),_=r(137),O=r(70),E=r(32),P=r(71).f,j=r(154),F=r(76),M=r(7),A=r(48),N=r(117),T=r(146),I=r(155),k=r(80),L=r(123),R=r(74),C=r(130),D=r(160),U=r(11),W=r(31),G=U.f,B=W.f,V=i.RangeError,z=i.TypeError,q=i.Uint8Array,K="ArrayBuffer",J="Shared"+K,Y="BYTES_PER_ELEMENT",H="prototype",$=Array[H],X=f.ArrayBuffer,Q=f.DataView,Z=A(0),tt=A(2),nt=A(3),rt=A(4),et=A(5),it=A(6),ot=N(!0),ut=N(!1),ct=I.values,ft=I.keys,at=I.entries,st=$.lastIndexOf,lt=$.reduce,ht=$.reduceRight,vt=$.join,pt=$.sort,dt=$.slice,yt=$.toString,gt=$.toLocaleString,bt=M("iterator"),mt=M("toStringTag"),xt=F("typed_constructor"),wt=F("def_constructor"),St=c.CONSTR,_t=c.TYPED,Ot=c.VIEW,Et="Wrong length!",Pt=A(1,function(t,n){return Tt(T(t,t[wt]),n)}),jt=o(function(){return 1===new q(new Uint16Array([1]).buffer)[0]}),Ft=!!q&&!!q[H].set&&o(function(){new q(1).set({})}),Mt=function(t,n){if(void 0===t)throw z(Et);var r=+t,e=d(t);if(n&&!m(r,e))throw V(Et);return e},At=function(t,n){var r=p(t);if(r<0||r%n)throw V("Wrong offset!");return r},Nt=function(t){if(w(t)&&_t in t)return t;throw z(t+" is not a typed array!")},Tt=function(t,n){if(!(w(t)&&xt in t))throw z("It is not a typed array constructor!");return new t(n)},It=function(t,n){return kt(T(t,t[wt]),n)},kt=function(t,n){for(var r=0,e=n.length,i=Tt(t,e);e>r;)i[r]=n[r++];return i},Lt=function(t,n,r){G(t,n,{get:function(){return this._d[r]}})},Rt=function(t){var n,r,e,i,o,u,c=S(t),f=arguments.length,s=f>1?arguments[1]:void 0,l=void 0!==s,h=j(c);if(void 0!=h&&!_(h)){for(u=h.call(c),e=[],n=0;!(o=u.next()).done;n++)e.push(o.value);c=e}for(l&&f>2&&(s=a(s,arguments[2],2)),n=0,r=d(c.length),i=Tt(this,r);r>n;n++)i[n]=l?s(c[n],n):c[n];return i},Ct=function(){for(var t=0,n=arguments.length,r=Tt(this,n);n>t;)r[t]=arguments[t++];return r},Dt=!!q&&o(function(){gt.call(new q(1))}),Ut=function(){return gt.apply(Dt?dt.call(Nt(this)):Nt(this),arguments)},Wt={copyWithin:function(t,n){return D.call(Nt(this),t,n,arguments.length>2?arguments[2]:void 0)},every:function(t){return rt(Nt(this),t,arguments.length>1?arguments[1]:void 0)},fill:function(t){return C.apply(Nt(this),arguments)},filter:function(t){return It(this,tt(Nt(this),t,arguments.length>1?arguments[1]:void 0))},find:function(t){return et(Nt(this),t,arguments.length>1?arguments[1]:void 0)},findIndex:function(t){return it(Nt(this),t,arguments.length>1?arguments[1]:void 0)},forEach:function(t){Z(Nt(this),t,arguments.length>1?arguments[1]:void 0)},indexOf:function(t){return ut(Nt(this),t,arguments.length>1?arguments[1]:void 0)},includes:function(t){return ot(Nt(this),t,arguments.length>1?arguments[1]:void 0)},join:function(t){return vt.apply(Nt(this),arguments)},lastIndexOf:function(t){return st.apply(Nt(this),arguments)},map:function(t){return Pt(Nt(this),t,arguments.length>1?arguments[1]:void 0)},reduce:function(t){return lt.apply(Nt(this),arguments)},reduceRight:function(t){return ht.apply(Nt(this),arguments)},reverse:function(){for(var t,n=this,r=Nt(n).length,e=Math.floor(r/2),i=0;i<e;)t=n[i],n[i++]=n[--r],n[r]=t;return n},some:function(t){return nt(Nt(this),t,arguments.length>1?arguments[1]:void 0)},sort:function(t){return pt.call(Nt(this),t)},subarray:function(t,n){var r=Nt(this),e=r.length,i=y(t,e);return new(T(r,r[wt]))(r.buffer,r.byteOffset+i*r.BYTES_PER_ELEMENT,d((void 0===n?e:y(n,e))-i))}},Gt=function(t,n){return It(this,dt.call(Nt(this),t,n))},Bt=function(t){Nt(this);var n=At(arguments[1],1),r=this.length,e=S(t),i=d(e.length),o=0;if(i+n>r)throw V(Et);for(;o<i;)this[n+o]=e[o++]},Vt={entries:function(){return at.call(Nt(this))},keys:function(){return ft.call(Nt(this))},values:function(){return ct.call(Nt(this))}},zt=function(t,n){return w(t)&&t[_t]&&"symbol"!=typeof n&&n in t&&String(+n)==String(n)},qt=function(t,n){return zt(t,n=g(n,!0))?l(2,t[n]):B(t,n)},Kt=function(t,n,r){return!(zt(t,n=g(n,!0))&&w(r)&&b(r,"value"))||b(r,"get")||b(r,"set")||r.configurable||b(r,"writable")&&!r.writable||b(r,"enumerable")&&!r.enumerable?G(t,n,r):(t[n]=r.value,t)};St||(W.f=qt,U.f=Kt),u(u.S+u.F*!St,"Object",{getOwnPropertyDescriptor:qt,defineProperty:Kt}),o(function(){yt.call({})})&&(yt=gt=function(){return vt.call(this)});var Jt=v({},Wt);v(Jt,Vt),h(Jt,bt,Vt.values),v(Jt,{slice:Gt,set:Bt,constructor:function(){},toString:yt,toLocaleString:Ut}),Lt(Jt,"buffer","b"),Lt(Jt,"byteOffset","o"),Lt(Jt,"byteLength","l"),Lt(Jt,"length","e"),G(Jt,mt,{get:function(){return this[_t]}}),t.exports=function(t,n,r,f){f=!!f;var a=t+(f?"Clamped":"")+"Array",l="Uint8Array"!=a,v="get"+t,p="set"+t,y=i[a],g=y||{},b=y&&E(y),m=!y||!c.ABV,S={},_=y&&y[H],j=function(t,r){var e=t._d;return e.v[v](r*n+e.o,jt)},F=function(t,r,e){var i=t._d;f&&(e=(e=Math.round(e))<0?0:e>255?255:255&e),i.v[p](r*n+i.o,e,jt)},M=function(t,n){G(t,n,{get:function(){return j(this,n)},set:function(t){return F(this,n,t)},enumerable:!0})};m?(y=r(function(t,r,e,i){s(t,y,a,"_d");var o,u,c,f,l=0,v=0;if(w(r)){if(!(r instanceof X||(f=x(r))==K||f==J))return _t in r?kt(y,r):Rt.call(y,r);o=r,v=At(e,n);var p=r.byteLength;if(void 0===i){if(p%n)throw V(Et);if((u=p-v)<0)throw V(Et)}else if((u=d(i)*n)+v>p)throw V(Et);c=u/n}else c=Mt(r,!0),u=c*n,o=new X(u);for(h(t,"_d",{b:o,o:v,l:u,e:c,v:new Q(o)});l<c;)M(t,l++)}),_=y[H]=O(Jt),h(_,"constructor",y)):L(function(t){new y(null),new y(t)},!0)||(y=r(function(t,r,e,i){s(t,y,a);var o;return w(r)?r instanceof X||(o=x(r))==K||o==J?void 0!==i?new g(r,At(e,n),i):void 0!==e?new g(r,At(e,n)):new g(r):_t in r?kt(y,r):Rt.call(y,r):new g(Mt(r,l))}),Z(b!==Function.prototype?P(g).concat(P(b)):P(g),function(t){t in y||h(y,t,g[t])}),y[H]=_,e||(_.constructor=y));var A=_[bt],N=!!A&&("values"==A.name||void 0==A.name),T=Vt.values;h(y,xt,!0),h(_,_t,a),h(_,Ot,!0),h(_,wt,y),(f?new y(1)[mt]==a:mt in _)||G(_,mt,{get:function(){return a}}),S[a]=y,u(u.G+u.W+u.F*(y!=g),S),u(u.S,a,{BYTES_PER_ELEMENT:n,from:Rt,of:Ct}),Y in _||h(_,Y,n),u(u.P,a,Wt),R(a),u(u.P+u.F*Ft,a,{set:Bt}),u(u.P+u.F*!N,a,Vt),u(u.P+u.F*(_.toString!=yt),a,{toString:yt}),u(u.P+u.F*o(function(){new y(1).slice()}),a,{slice:Gt}),u(u.P+u.F*(o(function(){return[1,2].toLocaleString()!=new y([1,2]).toLocaleString()})||!o(function(){_.toLocaleString.call([1,2])})),a,{toLocaleString:Ut}),k[a]=N?A:T,e||N||h(_,bt,T)}}else t.exports=function(){}},function(t,n){var r={}.toString;t.exports=function(t){return r.call(t).slice(8,-1)}},function(t,n,r){var e=r(21),i=r(5).document,o=e(i)&&e(i.createElement);t.exports=function(t){return o?i.createElement(t):{}}},function(t,n,r){t.exports=!r(12)&&!r(18)(function(){return 7!=Object.defineProperty(r(57)("div"),"a",{get:function(){return 7}}).a})},function(t,n,r){"use strict";var e=r(36),i=r(51),o=r(64),u=r(13),c=r(8),f=r(35),a=r(96),s=r(38),l=r(103),h=r(15)("iterator"),v=!([].keys&&"next"in[].keys()),p="keys",d="values",y=function(){return this};t.exports=function(t,n,r,g,b,m,x){a(r,n,g);var w,S,_,O=function(t){if(!v&&t in F)return F[t];switch(t){case p:case d:return function(){return new r(this,t)}}return function(){return new r(this,t)}},E=n+" Iterator",P=b==d,j=!1,F=t.prototype,M=F[h]||F["@@iterator"]||b&&F[b],A=M||O(b),N=b?P?O("entries"):A:void 0,T="Array"==n?F.entries||M:M;if(T&&(_=l(T.call(new t)))!==Object.prototype&&(s(_,E,!0),e||c(_,h)||u(_,h,y)),P&&M&&M.name!==d&&(j=!0,A=function(){return M.call(this)}),e&&!x||!v&&!j&&F[h]||u(F,h,A),f[n]=A,f[E]=y,b)if(w={values:P?A:O(d),keys:m?A:O(p),entries:N},x)for(S in w)S in F||o(F,S,w[S]);else i(i.P+i.F*(v||j),n,w);return w}},function(t,n,r){var e=r(20),i=r(100),o=r(34),u=r(39)("IE_PROTO"),c=function(){},f="prototype",a=function(){var t,n=r(57)("iframe"),e=o.length;for(n.style.display="none",r(93).appendChild(n),n.src="javascript:",t=n.contentWindow.document,t.open(),t.write("<script>document.F=Object<\/script>"),t.close(),a=t.F;e--;)delete a[f][o[e]];return a()};t.exports=Object.create||function(t,n){var r;return null!==t?(c[f]=e(t),r=new c,c[f]=null,r[u]=t):r=a(),void 0===n?r:i(r,n)}},function(t,n,r){var e=r(63),i=r(34).concat("length","prototype");n.f=Object.getOwnPropertyNames||function(t){return e(t,i)}},function(t,n){n.f=Object.getOwnPropertySymbols},function(t,n,r){var e=r(8),i=r(9),o=r(90)(!1),u=r(39)("IE_PROTO");t.exports=function(t,n){var r,c=i(t),f=0,a=[];for(r in c)r!=u&&e(c,r)&&a.push(r);for(;n.length>f;)e(c,r=n[f++])&&(~o(a,r)||a.push(r));return a}},function(t,n,r){t.exports=r(13)},function(t,n,r){var e=r(76)("meta"),i=r(6),o=r(24),u=r(11).f,c=0,f=Object.isExtensible||function(){return!0},a=!r(4)(function(){return f(Object.preventExtensions({}))}),s=function(t){u(t,e,{value:{i:"O"+ ++c,w:{}}})},l=function(t,n){if(!i(t))return"symbol"==typeof t?t:("string"==typeof t?"S":"P")+t;if(!o(t,e)){if(!f(t))return"F";if(!n)return"E";s(t)}return t[e].i},h=function(t,n){if(!o(t,e)){if(!f(t))return!0;if(!n)return!1;s(t)}return t[e].w},v=function(t){return a&&p.NEED&&f(t)&&!o(t,e)&&s(t),t},p=t.exports={KEY:e,NEED:!1,fastKey:l,getWeak:h,onFreeze:v}},function(t,n){t.exports=function(t,n){return{enumerable:!(1&t),configurable:!(2&t),writable:!(4&t),value:n}}},function(t,n){var r=Math.ceil,e=Math.floor;t.exports=function(t){return isNaN(t=+t)?0:(t>0?e:r)(t)}},function(t,n){t.exports=function(t,n,r,e){if(!(t instanceof n)||void 0!==e&&e in t)throw TypeError(r+": incorrect invocation!");return t}},function(t,n){t.exports=!1},function(t,n,r){var e=r(2),i=r(173),o=r(133),u=r(145)("IE_PROTO"),c=function(){},f="prototype",a=function(){var t,n=r(132)("iframe"),e=o.length;for(n.style.display="none",r(135).appendChild(n),n.src="javascript:",t=n.contentWindow.document,t.open(),t.write("<script>document.F=Object<\/script>"),t.close(),a=t.F;e--;)delete a[f][o[e]];return a()};t.exports=Object.create||function(t,n){var r;return null!==t?(c[f]=e(t),r=new c,c[f]=null,r[u]=t):r=a(),void 0===n?r:i(r,n)}},function(t,n,r){var e=r(175),i=r(133).concat("length","prototype");n.f=Object.getOwnPropertyNames||function(t){return e(t,i)}},function(t,n,r){var e=r(175),i=r(133);t.exports=Object.keys||function(t){return e(t,i)}},function(t,n,r){var e=r(28);t.exports=function(t,n,r){for(var i in n)e(t,i,n[i],r);return t}},function(t,n,r){"use strict";var e=r(3),i=r(11),o=r(10),u=r(7)("species");t.exports=function(t){var n=e[t];o&&n&&!n[u]&&i.f(n,u,{configurable:!0,get:function(){return this}})}},function(t,n,r){var e=r(67),i=Math.max,o=Math.min;t.exports=function(t,n){return t=e(t),t<0?i(t+n,0):o(t,n)}},function(t,n){var r=0,e=Math.random();t.exports=function(t){return"Symbol(".concat(void 0===t?"":t,")_",(++r+e).toString(36))}},function(t,n,r){var e=r(33);t.exports=function(t){return Object(e(t))}},function(t,n,r){var e=r(7)("unscopables"),i=Array.prototype;void 0==i[e]&&r(27)(i,e,{}),t.exports=function(t){i[e][t]=!0}},function(t,n,r){var e=r(53),i=r(169),o=r(137),u=r(2),c=r(16),f=r(154),a={},s={},n=t.exports=function(t,n,r,l,h){var v,p,d,y,g=h?function(){return t}:f(t),b=e(r,l,n?2:1),m=0;if("function"!=typeof g)throw TypeError(t+" is not iterable!");if(o(g)){for(v=c(t.length);v>m;m++)if((y=n?b(u(p=t[m])[0],p[1]):b(t[m]))===a||y===s)return y}else for(d=g.call(t);!(p=d.next()).done;)if((y=i(d,b,p.value,n))===a||y===s)return y};n.BREAK=a,n.RETURN=s},function(t,n){t.exports={}},function(t,n,r){var e=r(11).f,i=r(24),o=r(7)("toStringTag");t.exports=function(t,n,r){t&&!i(t=r?t:t.prototype,o)&&e(t,o,{configurable:!0,value:n})}},function(t,n,r){var e=r(1),i=r(46),o=r(4),u=r(150),c="["+u+"]",f="​",a=RegExp("^"+c+c+"*"),s=RegExp(c+c+"*$"),l=function(t,n,r){var i={},c=o(function(){return!!u[t]()||f[t]()!=f}),a=i[t]=c?n(h):u[t];r&&(i[r]=a),e(e.P+e.F*c,"String",i)},h=l.trim=function(t,n){return t=String(i(t)),1&n&&(t=t.replace(a,"")),2&n&&(t=t.replace(s,"")),t};t.exports=l},function(t,n,r){t.exports={default:r(86),__esModule:!0}},function(t,n,r){t.exports={default:r(87),__esModule:!0}},function(t,n,r){"use strict";function e(t){return t&&t.__esModule?t:{default:t}}n.__esModule=!0;var i=r(84),o=e(i),u=r(83),c=e(u),f="function"==typeof c.default&&"symbol"==typeof o.default?function(t){return typeof t}:function(t){return t&&"function"==typeof c.default&&t.constructor===c.default&&t!==c.default.prototype?"symbol":typeof t};n.default="function"==typeof c.default&&"symbol"===f(o.default)?function(t){return void 0===t?"undefined":f(t)}:function(t){return t&&"function"==typeof c.default&&t.constructor===c.default&&t!==c.default.prototype?"symbol":void 0===t?"undefined":f(t)}},function(t,n,r){r(110),r(108),r(111),r(112),t.exports=r(25).Symbol},function(t,n,r){r(109),r(113),t.exports=r(44).f("iterator")},function(t,n){t.exports=function(t){if("function"!=typeof t)throw TypeError(t+" is not a function!");return t}},function(t,n){t.exports=function(){}},function(t,n,r){var e=r(9),i=r(106),o=r(105);t.exports=function(t){return function(n,r,u){var c,f=e(n),a=i(f.length),s=o(u,a);if(t&&r!=r){for(;a>s;)if((c=f[s++])!=c)return!0}else for(;a>s;s++)if((t||s in f)&&f[s]===r)return t||s||0;return!t&&-1}}},function(t,n,r){var e=r(88);t.exports=function(t,n,r){if(e(t),void 0===n)return t;switch(r){case 1:return function(r){return t.call(n,r)};case 2:return function(r,e){return t.call(n,r,e)};case 3:return function(r,e,i){return t.call(n,r,e,i)}}return function(){return t.apply(n,arguments)}}},function(t,n,r){var e=r(19),i=r(62),o=r(37);t.exports=function(t){var n=e(t),r=i.f;if(r)for(var u,c=r(t),f=o.f,a=0;c.length>a;)f.call(t,u=c[a++])&&n.push(u);return n}},function(t,n,r){t.exports=r(5).document&&document.documentElement},function(t,n,r){var e=r(56);t.exports=Object("z").propertyIsEnumerable(0)?Object:function(t){return"String"==e(t)?t.split(""):Object(t)}},function(t,n,r){var e=r(56);t.exports=Array.isArray||function(t){return"Array"==e(t)}},function(t,n,r){"use strict";var e=r(60),i=r(22),o=r(38),u={};r(13)(u,r(15)("iterator"),function(){return this}),t.exports=function(t,n,r){t.prototype=e(u,{next:i(1,r)}),o(t,n+" Iterator")}},function(t,n){t.exports=function(t,n){return{value:n,done:!!t}}},function(t,n,r){var e=r(19),i=r(9);t.exports=function(t,n){for(var r,o=i(t),u=e(o),c=u.length,f=0;c>f;)if(o[r=u[f++]]===n)return r}},function(t,n,r){var e=r(23)("meta"),i=r(21),o=r(8),u=r(14).f,c=0,f=Object.isExtensible||function(){return!0},a=!r(18)(function(){return f(Object.preventExtensions({}))}),s=function(t){u(t,e,{value:{i:"O"+ ++c,w:{}}})},l=function(t,n){if(!i(t))return"symbol"==typeof t?t:("string"==typeof t?"S":"P")+t;if(!o(t,e)){if(!f(t))return"F";if(!n)return"E";s(t)}return t[e].i},h=function(t,n){if(!o(t,e)){if(!f(t))return!0;if(!n)return!1;s(t)}return t[e].w},v=function(t){return a&&p.NEED&&f(t)&&!o(t,e)&&s(t),t},p=t.exports={KEY:e,NEED:!1,fastKey:l,getWeak:h,onFreeze:v}},function(t,n,r){var e=r(14),i=r(20),o=r(19);t.exports=r(12)?Object.defineProperties:function(t,n){i(t);for(var r,u=o(n),c=u.length,f=0;c>f;)e.f(t,r=u[f++],n[r]);return t}},function(t,n,r){var e=r(37),i=r(22),o=r(9),u=r(42),c=r(8),f=r(58),a=Object.getOwnPropertyDescriptor;n.f=r(12)?a:function(t,n){if(t=o(t),n=u(n,!0),f)try{return a(t,n)}catch(t){}if(c(t,n))return i(!e.f.call(t,n),t[n])}},function(t,n,r){var e=r(9),i=r(61).f,o={}.toString,u="object"==typeof window&&window&&Object.getOwnPropertyNames?Object.getOwnPropertyNames(window):[],c=function(t){try{return i(t)}catch(t){return u.slice()}};t.exports.f=function(t){return u&&"[object Window]"==o.call(t)?c(t):i(e(t))}},function(t,n,r){var e=r(8),i=r(77),o=r(39)("IE_PROTO"),u=Object.prototype;t.exports=Object.getPrototypeOf||function(t){return t=i(t),e(t,o)?t[o]:"function"==typeof t.constructor&&t instanceof t.constructor?t.constructor.prototype:t instanceof Object?u:null}},function(t,n,r){var e=r(41),i=r(33);t.exports=function(t){return function(n,r){var o,u,c=String(i(n)),f=e(r),a=c.length;return f<0||f>=a?t?"":void 0:(o=c.charCodeAt(f),o<55296||o>56319||f+1===a||(u=c.charCodeAt(f+1))<56320||u>57343?t?c.charAt(f):o:t?c.slice(f,f+2):u-56320+(o-55296<<10)+65536)}}},function(t,n,r){var e=r(41),i=Math.max,o=Math.min;t.exports=function(t,n){return t=e(t),t<0?i(t+n,0):o(t,n)}},function(t,n,r){var e=r(41),i=Math.min;t.exports=function(t){return t>0?i(e(t),9007199254740991):0}},function(t,n,r){"use strict";var e=r(89),i=r(97),o=r(35),u=r(9);t.exports=r(59)(Array,"Array",function(t,n){this._t=u(t),this._i=0,this._k=n},function(){var t=this._t,n=this._k,r=this._i++;return!t||r>=t.length?(this._t=void 0,i(1)):"keys"==n?i(0,r):"values"==n?i(0,t[r]):i(0,[r,t[r]])},"values"),o.Arguments=o.Array,e("keys"),e("values"),e("entries")},function(t,n){},function(t,n,r){"use strict";var e=r(104)(!0);r(59)(String,"String",function(t){this._t=String(t),this._i=0},function(){var t,n=this._t,r=this._i;return r>=n.length?{value:void 0,done:!0}:(t=e(n,r),this._i+=t.length,{value:t,done:!1})})},function(t,n,r){"use strict";var e=r(5),i=r(8),o=r(12),u=r(51),c=r(64),f=r(99).KEY,a=r(18),s=r(40),l=r(38),h=r(23),v=r(15),p=r(44),d=r(43),y=r(98),g=r(92),b=r(95),m=r(20),x=r(9),w=r(42),S=r(22),_=r(60),O=r(102),E=r(101),P=r(14),j=r(19),F=E.f,M=P.f,A=O.f,N=e.Symbol,T=e.JSON,I=T&&T.stringify,k="prototype",L=v("_hidden"),R=v("toPrimitive"),C={}.propertyIsEnumerable,D=s("symbol-registry"),U=s("symbols"),W=s("op-symbols"),G=Object[k],B="function"==typeof N,V=e.QObject,z=!V||!V[k]||!V[k].findChild,q=o&&a(function(){return 7!=_(M({},"a",{get:function(){return M(this,"a",{value:7}).a}})).a})?function(t,n,r){var e=F(G,n);e&&delete G[n],M(t,n,r),e&&t!==G&&M(G,n,e)}:M,K=function(t){var n=U[t]=_(N[k]);return n._k=t,n},J=B&&"symbol"==typeof N.iterator?function(t){return"symbol"==typeof t}:function(t){return t instanceof N},Y=function(t,n,r){return t===G&&Y(W,n,r),m(t),n=w(n,!0),m(r),i(U,n)?(r.enumerable?(i(t,L)&&t[L][n]&&(t[L][n]=!1),r=_(r,{enumerable:S(0,!1)})):(i(t,L)||M(t,L,S(1,{})),t[L][n]=!0),q(t,n,r)):M(t,n,r)},H=function(t,n){m(t);for(var r,e=g(n=x(n)),i=0,o=e.length;o>i;)Y(t,r=e[i++],n[r]);return t},$=function(t,n){return void 0===n?_(t):H(_(t),n)},X=function(t){var n=C.call(this,t=w(t,!0));return!(this===G&&i(U,t)&&!i(W,t))&&(!(n||!i(this,t)||!i(U,t)||i(this,L)&&this[L][t])||n)},Q=function(t,n){if(t=x(t),n=w(n,!0),t!==G||!i(U,n)||i(W,n)){var r=F(t,n);return!r||!i(U,n)||i(t,L)&&t[L][n]||(r.enumerable=!0),r}},Z=function(t){for(var n,r=A(x(t)),e=[],o=0;r.length>o;)i(U,n=r[o++])||n==L||n==f||e.push(n);return e},tt=function(t){for(var n,r=t===G,e=A(r?W:x(t)),o=[],u=0;e.length>u;)!i(U,n=e[u++])||r&&!i(G,n)||o.push(U[n]);return o};B||(N=function(){if(this instanceof N)throw TypeError("Symbol is not a constructor!");var t=h(arguments.length>0?arguments[0]:void 0),n=function(r){this===G&&n.call(W,r),i(this,L)&&i(this[L],t)&&(this[L][t]=!1),q(this,t,S(1,r))};return o&&z&&q(G,t,{configurable:!0,set:n}),K(t)},c(N[k],"toString",function(){return this._k}),E.f=Q,P.f=Y,r(61).f=O.f=Z,r(37).f=X,r(62).f=tt,o&&!r(36)&&c(G,"propertyIsEnumerable",X,!0),p.f=function(t){return K(v(t))}),u(u.G+u.W+u.F*!B,{Symbol:N});for(var nt="hasInstance,isConcatSpreadable,iterator,match,replace,search,species,split,toPrimitive,toStringTag,unscopables".split(","),rt=0;nt.length>rt;)v(nt[rt++]);for(var nt=j(v.store),rt=0;nt.length>rt;)d(nt[rt++]);u(u.S+u.F*!B,"Symbol",{for:function(t){return i(D,t+="")?D[t]:D[t]=N(t)},keyFor:function(t){if(J(t))return y(D,t);throw TypeError(t+" is not a symbol!")},useSetter:function(){z=!0},useSimple:function(){z=!1}}),u(u.S+u.F*!B,"Object",{create:$,defineProperty:Y,defineProperties:H,getOwnPropertyDescriptor:Q,getOwnPropertyNames:Z,getOwnPropertySymbols:tt}),T&&u(u.S+u.F*(!B||a(function(){var t=N();return"[null]"!=I([t])||"{}"!=I({a:t})||"{}"!=I(Object(t))})),"JSON",{stringify:function(t){if(void 0!==t&&!J(t)){for(var n,r,e=[t],i=1;arguments.length>i;)e.push(arguments[i++]);return n=e[1],"function"==typeof n&&(r=n),!r&&b(n)||(n=function(t,n){if(r&&(n=r.call(this,t,n)),!J(n))return n}),e[1]=n,I.apply(T,e)}}}),N[k][R]||r(13)(N[k],R,N[k].valueOf),l(N,"Symbol"),l(Math,"Math",!0),l(e.JSON,"JSON",!0)},function(t,n,r){r(43)("asyncIterator")},function(t,n,r){r(43)("observable")},function(t,n,r){r(107);for(var e=r(5),i=r(13),o=r(35),u=r(15)("toStringTag"),c=["NodeList","DOMTokenList","MediaList","StyleSheetList","CSSRuleList"],f=0;f<5;f++){var a=c[f],s=e[a],l=s&&s.prototype;l&&!l[u]&&i(l,u,a),o[a]=o.Array}},function(t,n,r){var e=r(45),i=r(7)("toStringTag"),o="Arguments"==e(function(){return arguments}()),u=function(t,n){try{return t[n]}catch(t){}};t.exports=function(t){var n,r,c;return void 0===t?"Undefined":null===t?"Null":"string"==typeof(r=u(n=Object(t),i))?r:o?e(n):"Object"==(c=e(n))&&"function"==typeof n.callee?"Arguments":c}},function(t,n,r){var e=r(45);t.exports=Object("z").propertyIsEnumerable(0)?Object:function(t){return"String"==e(t)?t.split(""):Object(t)}},function(t,n){n.f={}.propertyIsEnumerable},function(t,n,r){var e=r(30),i=r(16),o=r(75);t.exports=function(t){return function(n,r,u){var c,f=e(n),a=i(f.length),s=o(u,a);if(t&&r!=r){for(;a>s;)if((c=f[s++])!=c)return!0}else for(;a>s;s++)if((t||s in f)&&f[s]===r)return t||s||0;return!t&&-1}}},function(t,n,r){"use strict";var e=r(3),i=r(1),o=r(28),u=r(73),c=r(65),f=r(79),a=r(68),s=r(6),l=r(4),h=r(123),v=r(81),p=r(136);t.exports=function(t,n,r,d,y,g){var b=e[t],m=b,x=y?"set":"add",w=m&&m.prototype,S={},_=function(t){var n=w[t];o(w,t,"delete"==t?function(t){return!(g&&!s(t))&&n.call(this,0===t?0:t)}:"has"==t?function(t){return!(g&&!s(t))&&n.call(this,0===t?0:t)}:"get"==t?function(t){return g&&!s(t)?void 0:n.call(this,0===t?0:t)}:"add"==t?function(t){return n.call(this,0===t?0:t),this}:function(t,r){return n.call(this,0===t?0:t,r),this})};if("function"==typeof m&&(g||w.forEach&&!l(function(){(new m).entries().next()}))){var O=new m,E=O[x](g?{}:-0,1)!=O,P=l(function(){O.has(1)}),j=h(function(t){new m(t)}),F=!g&&l(function(){for(var t=new m,n=5;n--;)t[x](n,n);return!t.has(-0)});j||(m=n(function(n,r){a(n,m,t);var e=p(new b,n,m);return void 0!=r&&f(r,y,e[x],e),e}),m.prototype=w,w.constructor=m),(P||F)&&(_("delete"),_("has"),y&&_("get")),(F||E)&&_(x),g&&w.clear&&delete w.clear}else m=d.getConstructor(n,t,y,x),u(m.prototype,r),c.NEED=!0;return v(m,t),S[t]=m,i(i.G+i.W+i.F*(m!=b),S),g||d.setStrong(m,t,y),m}},function(t,n,r){"use strict";var e=r(27),i=r(28),o=r(4),u=r(46),c=r(7);t.exports=function(t,n,r){var f=c(t),a=r(u,f,""[t]),s=a[0],l=a[1];o(function(){var n={};return n[f]=function(){return 7},7!=""[t](n)})&&(i(String.prototype,t,s),e(RegExp.prototype,f,2==n?function(t,n){return l.call(t,this,n)}:function(t){return l.call(t,this)}))}
},function(t,n,r){"use strict";var e=r(2);t.exports=function(){var t=e(this),n="";return t.global&&(n+="g"),t.ignoreCase&&(n+="i"),t.multiline&&(n+="m"),t.unicode&&(n+="u"),t.sticky&&(n+="y"),n}},function(t,n){t.exports=function(t,n,r){var e=void 0===r;switch(n.length){case 0:return e?t():t.call(r);case 1:return e?t(n[0]):t.call(r,n[0]);case 2:return e?t(n[0],n[1]):t.call(r,n[0],n[1]);case 3:return e?t(n[0],n[1],n[2]):t.call(r,n[0],n[1],n[2]);case 4:return e?t(n[0],n[1],n[2],n[3]):t.call(r,n[0],n[1],n[2],n[3])}return t.apply(r,n)}},function(t,n,r){var e=r(6),i=r(45),o=r(7)("match");t.exports=function(t){var n;return e(t)&&(void 0!==(n=t[o])?!!n:"RegExp"==i(t))}},function(t,n,r){var e=r(7)("iterator"),i=!1;try{var o=[7][e]();o.return=function(){i=!0},Array.from(o,function(){throw 2})}catch(t){}t.exports=function(t,n){if(!n&&!i)return!1;var r=!1;try{var o=[7],u=o[e]();u.next=function(){return{done:r=!0}},o[e]=function(){return u},t(o)}catch(t){}return r}},function(t,n,r){t.exports=r(69)||!r(4)(function(){var t=Math.random();__defineSetter__.call(null,t,function(){}),delete r(3)[t]})},function(t,n){n.f=Object.getOwnPropertySymbols},function(t,n,r){var e=r(3),i="__core-js_shared__",o=e[i]||(e[i]={});t.exports=function(t){return o[t]||(o[t]={})}},function(t,n,r){for(var e,i=r(3),o=r(27),u=r(76),c=u("typed_array"),f=u("view"),a=!(!i.ArrayBuffer||!i.DataView),s=a,l=0,h="Int8Array,Uint8Array,Uint8ClampedArray,Int16Array,Uint16Array,Int32Array,Uint32Array,Float32Array,Float64Array".split(",");l<9;)(e=i[h[l++]])?(o(e.prototype,c,!0),o(e.prototype,f,!0)):s=!1;t.exports={ABV:a,CONSTR:s,TYPED:c,VIEW:f}},function(t,n){"use strict";var r={versions:function(){var t=window.navigator.userAgent;return{trident:t.indexOf("Trident")>-1,presto:t.indexOf("Presto")>-1,webKit:t.indexOf("AppleWebKit")>-1,gecko:t.indexOf("Gecko")>-1&&-1==t.indexOf("KHTML"),mobile:!!t.match(/AppleWebKit.*Mobile.*/),ios:!!t.match(/\(i[^;]+;( U;)? CPU.+Mac OS X/),android:t.indexOf("Android")>-1||t.indexOf("Linux")>-1,iPhone:t.indexOf("iPhone")>-1||t.indexOf("Mac")>-1,iPad:t.indexOf("iPad")>-1,webApp:-1==t.indexOf("Safari"),weixin:-1==t.indexOf("MicroMessenger")}}()};t.exports=r},function(t,n,r){"use strict";var e=r(85),i=function(t){return t&&t.__esModule?t:{default:t}}(e),o=function(){function t(t,n,e){return n||e?String.fromCharCode(n||e):r[t]||t}function n(t){return e[t]}var r={"&quot;":'"',"&lt;":"<","&gt;":">","&amp;":"&","&nbsp;":" "},e={};for(var u in r)e[r[u]]=u;return r["&apos;"]="'",e["'"]="&#39;",{encode:function(t){return t?(""+t).replace(/['<> "&]/g,n).replace(/\r?\n/g,"<br/>").replace(/\s/g,"&nbsp;"):""},decode:function(n){return n?(""+n).replace(/<br\s*\/?>/gi,"\n").replace(/&quot;|&lt;|&gt;|&amp;|&nbsp;|&apos;|&#(\d+);|&#(\d+)/g,t).replace(/\u00a0/g," "):""},encodeBase16:function(t){if(!t)return t;t+="";for(var n=[],r=0,e=t.length;e>r;r++)n.push(t.charCodeAt(r).toString(16).toUpperCase());return n.join("")},encodeBase16forJSON:function(t){if(!t)return t;t=t.replace(/[\u4E00-\u9FBF]/gi,function(t){return escape(t).replace("%u","\\u")});for(var n=[],r=0,e=t.length;e>r;r++)n.push(t.charCodeAt(r).toString(16).toUpperCase());return n.join("")},decodeBase16:function(t){if(!t)return t;t+="";for(var n=[],r=0,e=t.length;e>r;r+=2)n.push(String.fromCharCode("0x"+t.slice(r,r+2)));return n.join("")},encodeObject:function(t){if(t instanceof Array)for(var n=0,r=t.length;r>n;n++)t[n]=o.encodeObject(t[n]);else if("object"==(void 0===t?"undefined":(0,i.default)(t)))for(var e in t)t[e]=o.encodeObject(t[e]);else if("string"==typeof t)return o.encode(t);return t},loadScript:function(t){var n=document.createElement("script");document.getElementsByTagName("body")[0].appendChild(n),n.setAttribute("src",t)},addLoadEvent:function(t){var n=window.onload;"function"!=typeof window.onload?window.onload=t:window.onload=function(){n(),t()}}}}();t.exports=o},function(t,n,r){"use strict";var e=r(17),i=r(75),o=r(16);t.exports=function(t){for(var n=e(this),r=o(n.length),u=arguments.length,c=i(u>1?arguments[1]:void 0,r),f=u>2?arguments[2]:void 0,a=void 0===f?r:i(f,r);a>c;)n[c++]=t;return n}},function(t,n,r){"use strict";var e=r(11),i=r(66);t.exports=function(t,n,r){n in t?e.f(t,n,i(0,r)):t[n]=r}},function(t,n,r){var e=r(6),i=r(3).document,o=e(i)&&e(i.createElement);t.exports=function(t){return o?i.createElement(t):{}}},function(t,n){t.exports="constructor,hasOwnProperty,isPrototypeOf,propertyIsEnumerable,toLocaleString,toString,valueOf".split(",")},function(t,n,r){var e=r(7)("match");t.exports=function(t){var n=/./;try{"/./"[t](n)}catch(r){try{return n[e]=!1,!"/./"[t](n)}catch(t){}}return!0}},function(t,n,r){t.exports=r(3).document&&document.documentElement},function(t,n,r){var e=r(6),i=r(144).set;t.exports=function(t,n,r){var o,u=n.constructor;return u!==r&&"function"==typeof u&&(o=u.prototype)!==r.prototype&&e(o)&&i&&i(t,o),t}},function(t,n,r){var e=r(80),i=r(7)("iterator"),o=Array.prototype;t.exports=function(t){return void 0!==t&&(e.Array===t||o[i]===t)}},function(t,n,r){var e=r(45);t.exports=Array.isArray||function(t){return"Array"==e(t)}},function(t,n,r){"use strict";var e=r(70),i=r(66),o=r(81),u={};r(27)(u,r(7)("iterator"),function(){return this}),t.exports=function(t,n,r){t.prototype=e(u,{next:i(1,r)}),o(t,n+" Iterator")}},function(t,n,r){"use strict";var e=r(69),i=r(1),o=r(28),u=r(27),c=r(24),f=r(80),a=r(139),s=r(81),l=r(32),h=r(7)("iterator"),v=!([].keys&&"next"in[].keys()),p="keys",d="values",y=function(){return this};t.exports=function(t,n,r,g,b,m,x){a(r,n,g);var w,S,_,O=function(t){if(!v&&t in F)return F[t];switch(t){case p:case d:return function(){return new r(this,t)}}return function(){return new r(this,t)}},E=n+" Iterator",P=b==d,j=!1,F=t.prototype,M=F[h]||F["@@iterator"]||b&&F[b],A=M||O(b),N=b?P?O("entries"):A:void 0,T="Array"==n?F.entries||M:M;if(T&&(_=l(T.call(new t)))!==Object.prototype&&(s(_,E,!0),e||c(_,h)||u(_,h,y)),P&&M&&M.name!==d&&(j=!0,A=function(){return M.call(this)}),e&&!x||!v&&!j&&F[h]||u(F,h,A),f[n]=A,f[E]=y,b)if(w={values:P?A:O(d),keys:m?A:O(p),entries:N},x)for(S in w)S in F||o(F,S,w[S]);else i(i.P+i.F*(v||j),n,w);return w}},function(t,n){var r=Math.expm1;t.exports=!r||r(10)>22025.465794806718||r(10)<22025.465794806718||-2e-17!=r(-2e-17)?function(t){return 0==(t=+t)?t:t>-1e-6&&t<1e-6?t+t*t/2:Math.exp(t)-1}:r},function(t,n){t.exports=Math.sign||function(t){return 0==(t=+t)||t!=t?t:t<0?-1:1}},function(t,n,r){var e=r(3),i=r(151).set,o=e.MutationObserver||e.WebKitMutationObserver,u=e.process,c=e.Promise,f="process"==r(45)(u);t.exports=function(){var t,n,r,a=function(){var e,i;for(f&&(e=u.domain)&&e.exit();t;){i=t.fn,t=t.next;try{i()}catch(e){throw t?r():n=void 0,e}}n=void 0,e&&e.enter()};if(f)r=function(){u.nextTick(a)};else if(o){var s=!0,l=document.createTextNode("");new o(a).observe(l,{characterData:!0}),r=function(){l.data=s=!s}}else if(c&&c.resolve){var h=c.resolve();r=function(){h.then(a)}}else r=function(){i.call(e,a)};return function(e){var i={fn:e,next:void 0};n&&(n.next=i),t||(t=i,r()),n=i}}},function(t,n,r){var e=r(6),i=r(2),o=function(t,n){if(i(t),!e(n)&&null!==n)throw TypeError(n+": can't set as prototype!")};t.exports={set:Object.setPrototypeOf||("__proto__"in{}?function(t,n,e){try{e=r(53)(Function.call,r(31).f(Object.prototype,"__proto__").set,2),e(t,[]),n=!(t instanceof Array)}catch(t){n=!0}return function(t,r){return o(t,r),n?t.__proto__=r:e(t,r),t}}({},!1):void 0),check:o}},function(t,n,r){var e=r(126)("keys"),i=r(76);t.exports=function(t){return e[t]||(e[t]=i(t))}},function(t,n,r){var e=r(2),i=r(26),o=r(7)("species");t.exports=function(t,n){var r,u=e(t).constructor;return void 0===u||void 0==(r=e(u)[o])?n:i(r)}},function(t,n,r){var e=r(67),i=r(46);t.exports=function(t){return function(n,r){var o,u,c=String(i(n)),f=e(r),a=c.length;return f<0||f>=a?t?"":void 0:(o=c.charCodeAt(f),o<55296||o>56319||f+1===a||(u=c.charCodeAt(f+1))<56320||u>57343?t?c.charAt(f):o:t?c.slice(f,f+2):u-56320+(o-55296<<10)+65536)}}},function(t,n,r){var e=r(122),i=r(46);t.exports=function(t,n,r){if(e(n))throw TypeError("String#"+r+" doesn't accept regex!");return String(i(t))}},function(t,n,r){"use strict";var e=r(67),i=r(46);t.exports=function(t){var n=String(i(this)),r="",o=e(t);if(o<0||o==1/0)throw RangeError("Count can't be negative");for(;o>0;(o>>>=1)&&(n+=n))1&o&&(r+=n);return r}},function(t,n){t.exports="\t\n\v\f\r   ᠎             　\u2028\u2029\ufeff"},function(t,n,r){var e,i,o,u=r(53),c=r(121),f=r(135),a=r(132),s=r(3),l=s.process,h=s.setImmediate,v=s.clearImmediate,p=s.MessageChannel,d=0,y={},g="onreadystatechange",b=function(){var t=+this;if(y.hasOwnProperty(t)){var n=y[t];delete y[t],n()}},m=function(t){b.call(t.data)};h&&v||(h=function(t){for(var n=[],r=1;arguments.length>r;)n.push(arguments[r++]);return y[++d]=function(){c("function"==typeof t?t:Function(t),n)},e(d),d},v=function(t){delete y[t]},"process"==r(45)(l)?e=function(t){l.nextTick(u(b,t,1))}:p?(i=new p,o=i.port2,i.port1.onmessage=m,e=u(o.postMessage,o,1)):s.addEventListener&&"function"==typeof postMessage&&!s.importScripts?(e=function(t){s.postMessage(t+"","*")},s.addEventListener("message",m,!1)):e=g in a("script")?function(t){f.appendChild(a("script"))[g]=function(){f.removeChild(this),b.call(t)}}:function(t){setTimeout(u(b,t,1),0)}),t.exports={set:h,clear:v}},function(t,n,r){"use strict";var e=r(3),i=r(10),o=r(69),u=r(127),c=r(27),f=r(73),a=r(4),s=r(68),l=r(67),h=r(16),v=r(71).f,p=r(11).f,d=r(130),y=r(81),g="ArrayBuffer",b="DataView",m="prototype",x="Wrong length!",w="Wrong index!",S=e[g],_=e[b],O=e.Math,E=e.RangeError,P=e.Infinity,j=S,F=O.abs,M=O.pow,A=O.floor,N=O.log,T=O.LN2,I="buffer",k="byteLength",L="byteOffset",R=i?"_b":I,C=i?"_l":k,D=i?"_o":L,U=function(t,n,r){var e,i,o,u=Array(r),c=8*r-n-1,f=(1<<c)-1,a=f>>1,s=23===n?M(2,-24)-M(2,-77):0,l=0,h=t<0||0===t&&1/t<0?1:0;for(t=F(t),t!=t||t===P?(i=t!=t?1:0,e=f):(e=A(N(t)/T),t*(o=M(2,-e))<1&&(e--,o*=2),t+=e+a>=1?s/o:s*M(2,1-a),t*o>=2&&(e++,o/=2),e+a>=f?(i=0,e=f):e+a>=1?(i=(t*o-1)*M(2,n),e+=a):(i=t*M(2,a-1)*M(2,n),e=0));n>=8;u[l++]=255&i,i/=256,n-=8);for(e=e<<n|i,c+=n;c>0;u[l++]=255&e,e/=256,c-=8);return u[--l]|=128*h,u},W=function(t,n,r){var e,i=8*r-n-1,o=(1<<i)-1,u=o>>1,c=i-7,f=r-1,a=t[f--],s=127&a;for(a>>=7;c>0;s=256*s+t[f],f--,c-=8);for(e=s&(1<<-c)-1,s>>=-c,c+=n;c>0;e=256*e+t[f],f--,c-=8);if(0===s)s=1-u;else{if(s===o)return e?NaN:a?-P:P;e+=M(2,n),s-=u}return(a?-1:1)*e*M(2,s-n)},G=function(t){return t[3]<<24|t[2]<<16|t[1]<<8|t[0]},B=function(t){return[255&t]},V=function(t){return[255&t,t>>8&255]},z=function(t){return[255&t,t>>8&255,t>>16&255,t>>24&255]},q=function(t){return U(t,52,8)},K=function(t){return U(t,23,4)},J=function(t,n,r){p(t[m],n,{get:function(){return this[r]}})},Y=function(t,n,r,e){var i=+r,o=l(i);if(i!=o||o<0||o+n>t[C])throw E(w);var u=t[R]._b,c=o+t[D],f=u.slice(c,c+n);return e?f:f.reverse()},H=function(t,n,r,e,i,o){var u=+r,c=l(u);if(u!=c||c<0||c+n>t[C])throw E(w);for(var f=t[R]._b,a=c+t[D],s=e(+i),h=0;h<n;h++)f[a+h]=s[o?h:n-h-1]},$=function(t,n){s(t,S,g);var r=+n,e=h(r);if(r!=e)throw E(x);return e};if(u.ABV){if(!a(function(){new S})||!a(function(){new S(.5)})){S=function(t){return new j($(this,t))};for(var X,Q=S[m]=j[m],Z=v(j),tt=0;Z.length>tt;)(X=Z[tt++])in S||c(S,X,j[X]);o||(Q.constructor=S)}var nt=new _(new S(2)),rt=_[m].setInt8;nt.setInt8(0,2147483648),nt.setInt8(1,2147483649),!nt.getInt8(0)&&nt.getInt8(1)||f(_[m],{setInt8:function(t,n){rt.call(this,t,n<<24>>24)},setUint8:function(t,n){rt.call(this,t,n<<24>>24)}},!0)}else S=function(t){var n=$(this,t);this._b=d.call(Array(n),0),this[C]=n},_=function(t,n,r){s(this,_,b),s(t,S,b);var e=t[C],i=l(n);if(i<0||i>e)throw E("Wrong offset!");if(r=void 0===r?e-i:h(r),i+r>e)throw E(x);this[R]=t,this[D]=i,this[C]=r},i&&(J(S,k,"_l"),J(_,I,"_b"),J(_,k,"_l"),J(_,L,"_o")),f(_[m],{getInt8:function(t){return Y(this,1,t)[0]<<24>>24},getUint8:function(t){return Y(this,1,t)[0]},getInt16:function(t){var n=Y(this,2,t,arguments[1]);return(n[1]<<8|n[0])<<16>>16},getUint16:function(t){var n=Y(this,2,t,arguments[1]);return n[1]<<8|n[0]},getInt32:function(t){return G(Y(this,4,t,arguments[1]))},getUint32:function(t){return G(Y(this,4,t,arguments[1]))>>>0},getFloat32:function(t){return W(Y(this,4,t,arguments[1]),23,4)},getFloat64:function(t){return W(Y(this,8,t,arguments[1]),52,8)},setInt8:function(t,n){H(this,1,t,B,n)},setUint8:function(t,n){H(this,1,t,B,n)},setInt16:function(t,n){H(this,2,t,V,n,arguments[2])},setUint16:function(t,n){H(this,2,t,V,n,arguments[2])},setInt32:function(t,n){H(this,4,t,z,n,arguments[2])},setUint32:function(t,n){H(this,4,t,z,n,arguments[2])},setFloat32:function(t,n){H(this,4,t,K,n,arguments[2])},setFloat64:function(t,n){H(this,8,t,q,n,arguments[2])}});y(S,g),y(_,b),c(_[m],u.VIEW,!0),n[g]=S,n[b]=_},function(t,n,r){var e=r(3),i=r(52),o=r(69),u=r(182),c=r(11).f;t.exports=function(t){var n=i.Symbol||(i.Symbol=o?{}:e.Symbol||{});"_"==t.charAt(0)||t in n||c(n,t,{value:u.f(t)})}},function(t,n,r){var e=r(114),i=r(7)("iterator"),o=r(80);t.exports=r(52).getIteratorMethod=function(t){if(void 0!=t)return t[i]||t["@@iterator"]||o[e(t)]}},function(t,n,r){"use strict";var e=r(78),i=r(170),o=r(80),u=r(30);t.exports=r(140)(Array,"Array",function(t,n){this._t=u(t),this._i=0,this._k=n},function(){var t=this._t,n=this._k,r=this._i++;return!t||r>=t.length?(this._t=void 0,i(1)):"keys"==n?i(0,r):"values"==n?i(0,t[r]):i(0,[r,t[r]])},"values"),o.Arguments=o.Array,e("keys"),e("values"),e("entries")},function(t,n){function r(t,n){t.classList?t.classList.add(n):t.className+=" "+n}t.exports=r},function(t,n){function r(t,n){if(t.classList)t.classList.remove(n);else{var r=new RegExp("(^|\\b)"+n.split(" ").join("|")+"(\\b|$)","gi");t.className=t.className.replace(r," ")}}t.exports=r},function(t,n){function r(){throw new Error("setTimeout has not been defined")}function e(){throw new Error("clearTimeout has not been defined")}function i(t){if(s===setTimeout)return setTimeout(t,0);if((s===r||!s)&&setTimeout)return s=setTimeout,setTimeout(t,0);try{return s(t,0)}catch(n){try{return s.call(null,t,0)}catch(n){return s.call(this,t,0)}}}function o(t){if(l===clearTimeout)return clearTimeout(t);if((l===e||!l)&&clearTimeout)return l=clearTimeout,clearTimeout(t);try{return l(t)}catch(n){try{return l.call(null,t)}catch(n){return l.call(this,t)}}}function u(){d&&v&&(d=!1,v.length?p=v.concat(p):y=-1,p.length&&c())}function c(){if(!d){var t=i(u);d=!0;for(var n=p.length;n;){for(v=p,p=[];++y<n;)v&&v[y].run();y=-1,n=p.length}v=null,d=!1,o(t)}}function f(t,n){this.fun=t,this.array=n}function a(){}var s,l,h=t.exports={};!function(){try{s="function"==typeof setTimeout?setTimeout:r}catch(t){s=r}try{l="function"==typeof clearTimeout?clearTimeout:e}catch(t){l=e}}();var v,p=[],d=!1,y=-1;h.nextTick=function(t){var n=new Array(arguments.length-1);if(arguments.length>1)for(var r=1;r<arguments.length;r++)n[r-1]=arguments[r];p.push(new f(t,n)),1!==p.length||d||i(c)},f.prototype.run=function(){this.fun.apply(null,this.array)},h.title="browser",h.browser=!0,h.env={},h.argv=[],h.version="",h.versions={},h.on=a,h.addListener=a,h.once=a,h.off=a,h.removeListener=a,h.removeAllListeners=a,h.emit=a,h.prependListener=a,h.prependOnceListener=a,h.listeners=function(t){return[]},h.binding=function(t){throw new Error("process.binding is not supported")},h.cwd=function(){return"/"},h.chdir=function(t){throw new Error("process.chdir is not supported")},h.umask=function(){return 0}},function(t,n,r){var e=r(45);t.exports=function(t,n){if("number"!=typeof t&&"Number"!=e(t))throw TypeError(n);return+t}},function(t,n,r){"use strict";var e=r(17),i=r(75),o=r(16);t.exports=[].copyWithin||function(t,n){var r=e(this),u=o(r.length),c=i(t,u),f=i(n,u),a=arguments.length>2?arguments[2]:void 0,s=Math.min((void 0===a?u:i(a,u))-f,u-c),l=1;for(f<c&&c<f+s&&(l=-1,f+=s-1,c+=s-1);s-- >0;)f in r?r[c]=r[f]:delete r[c],c+=l,f+=l;return r}},function(t,n,r){var e=r(79);t.exports=function(t,n){var r=[];return e(t,!1,r.push,r,n),r}},function(t,n,r){var e=r(26),i=r(17),o=r(115),u=r(16);t.exports=function(t,n,r,c,f){e(n);var a=i(t),s=o(a),l=u(a.length),h=f?l-1:0,v=f?-1:1;if(r<2)for(;;){if(h in s){c=s[h],h+=v;break}if(h+=v,f?h<0:l<=h)throw TypeError("Reduce of empty array with no initial value")}for(;f?h>=0:l>h;h+=v)h in s&&(c=n(c,s[h],h,a));return c}},function(t,n,r){"use strict";var e=r(26),i=r(6),o=r(121),u=[].slice,c={},f=function(t,n,r){if(!(n in c)){for(var e=[],i=0;i<n;i++)e[i]="a["+i+"]";c[n]=Function("F,a","return new F("+e.join(",")+")")}return c[n](t,r)};t.exports=Function.bind||function(t){var n=e(this),r=u.call(arguments,1),c=function(){var e=r.concat(u.call(arguments));return this instanceof c?f(n,e.length,e):o(n,e,t)};return i(n.prototype)&&(c.prototype=n.prototype),c}},function(t,n,r){"use strict";var e=r(11).f,i=r(70),o=r(73),u=r(53),c=r(68),f=r(46),a=r(79),s=r(140),l=r(170),h=r(74),v=r(10),p=r(65).fastKey,d=v?"_s":"size",y=function(t,n){var r,e=p(n);if("F"!==e)return t._i[e];for(r=t._f;r;r=r.n)if(r.k==n)return r};t.exports={getConstructor:function(t,n,r,s){var l=t(function(t,e){c(t,l,n,"_i"),t._i=i(null),t._f=void 0,t._l=void 0,t[d]=0,void 0!=e&&a(e,r,t[s],t)});return o(l.prototype,{clear:function(){for(var t=this,n=t._i,r=t._f;r;r=r.n)r.r=!0,r.p&&(r.p=r.p.n=void 0),delete n[r.i];t._f=t._l=void 0,t[d]=0},delete:function(t){var n=this,r=y(n,t);if(r){var e=r.n,i=r.p;delete n._i[r.i],r.r=!0,i&&(i.n=e),e&&(e.p=i),n._f==r&&(n._f=e),n._l==r&&(n._l=i),n[d]--}return!!r},forEach:function(t){c(this,l,"forEach");for(var n,r=u(t,arguments.length>1?arguments[1]:void 0,3);n=n?n.n:this._f;)for(r(n.v,n.k,this);n&&n.r;)n=n.p},has:function(t){return!!y(this,t)}}),v&&e(l.prototype,"size",{get:function(){return f(this[d])}}),l},def:function(t,n,r){var e,i,o=y(t,n);return o?o.v=r:(t._l=o={i:i=p(n,!0),k:n,v:r,p:e=t._l,n:void 0,r:!1},t._f||(t._f=o),e&&(e.n=o),t[d]++,"F"!==i&&(t._i[i]=o)),t},getEntry:y,setStrong:function(t,n,r){s(t,n,function(t,n){this._t=t,this._k=n,this._l=void 0},function(){for(var t=this,n=t._k,r=t._l;r&&r.r;)r=r.p;return t._t&&(t._l=r=r?r.n:t._t._f)?"keys"==n?l(0,r.k):"values"==n?l(0,r.v):l(0,[r.k,r.v]):(t._t=void 0,l(1))},r?"entries":"values",!r,!0),h(n)}}},function(t,n,r){var e=r(114),i=r(161);t.exports=function(t){return function(){if(e(this)!=t)throw TypeError(t+"#toJSON isn't generic");return i(this)}}},function(t,n,r){"use strict";var e=r(73),i=r(65).getWeak,o=r(2),u=r(6),c=r(68),f=r(79),a=r(48),s=r(24),l=a(5),h=a(6),v=0,p=function(t){return t._l||(t._l=new d)},d=function(){this.a=[]},y=function(t,n){return l(t.a,function(t){return t[0]===n})};d.prototype={get:function(t){var n=y(this,t);if(n)return n[1]},has:function(t){return!!y(this,t)},set:function(t,n){var r=y(this,t);r?r[1]=n:this.a.push([t,n])},delete:function(t){var n=h(this.a,function(n){return n[0]===t});return~n&&this.a.splice(n,1),!!~n}},t.exports={getConstructor:function(t,n,r,o){var a=t(function(t,e){c(t,a,n,"_i"),t._i=v++,t._l=void 0,void 0!=e&&f(e,r,t[o],t)});return e(a.prototype,{delete:function(t){if(!u(t))return!1;var n=i(t);return!0===n?p(this).delete(t):n&&s(n,this._i)&&delete n[this._i]},has:function(t){if(!u(t))return!1;var n=i(t);return!0===n?p(this).has(t):n&&s(n,this._i)}}),a},def:function(t,n,r){var e=i(o(n),!0);return!0===e?p(t).set(n,r):e[t._i]=r,t},ufstore:p}},function(t,n,r){t.exports=!r(10)&&!r(4)(function(){return 7!=Object.defineProperty(r(132)("div"),"a",{get:function(){return 7}}).a})},function(t,n,r){var e=r(6),i=Math.floor;t.exports=function(t){return!e(t)&&isFinite(t)&&i(t)===t}},function(t,n,r){var e=r(2);t.exports=function(t,n,r,i){try{return i?n(e(r)[0],r[1]):n(r)}catch(n){var o=t.return;throw void 0!==o&&e(o.call(t)),n}}},function(t,n){t.exports=function(t,n){return{value:n,done:!!t}}},function(t,n){t.exports=Math.log1p||function(t){return(t=+t)>-1e-8&&t<1e-8?t-t*t/2:Math.log(1+t)}},function(t,n,r){"use strict";var e=r(72),i=r(125),o=r(116),u=r(17),c=r(115),f=Object.assign;t.exports=!f||r(4)(function(){var t={},n={},r=Symbol(),e="abcdefghijklmnopqrst";return t[r]=7,e.split("").forEach(function(t){n[t]=t}),7!=f({},t)[r]||Object.keys(f({},n)).join("")!=e})?function(t,n){for(var r=u(t),f=arguments.length,a=1,s=i.f,l=o.f;f>a;)for(var h,v=c(arguments[a++]),p=s?e(v).concat(s(v)):e(v),d=p.length,y=0;d>y;)l.call(v,h=p[y++])&&(r[h]=v[h]);return r}:f},function(t,n,r){var e=r(11),i=r(2),o=r(72);t.exports=r(10)?Object.defineProperties:function(t,n){i(t);for(var r,u=o(n),c=u.length,f=0;c>f;)e.f(t,r=u[f++],n[r]);return t}},function(t,n,r){var e=r(30),i=r(71).f,o={}.toString,u="object"==typeof window&&window&&Object.getOwnPropertyNames?Object.getOwnPropertyNames(window):[],c=function(t){try{return i(t)}catch(t){return u.slice()}};t.exports.f=function(t){return u&&"[object Window]"==o.call(t)?c(t):i(e(t))}},function(t,n,r){var e=r(24),i=r(30),o=r(117)(!1),u=r(145)("IE_PROTO");t.exports=function(t,n){var r,c=i(t),f=0,a=[];for(r in c)r!=u&&e(c,r)&&a.push(r);for(;n.length>f;)e(c,r=n[f++])&&(~o(a,r)||a.push(r));return a}},function(t,n,r){var e=r(72),i=r(30),o=r(116).f;t.exports=function(t){return function(n){for(var r,u=i(n),c=e(u),f=c.length,a=0,s=[];f>a;)o.call(u,r=c[a++])&&s.push(t?[r,u[r]]:u[r]);return s}}},function(t,n,r){var e=r(71),i=r(125),o=r(2),u=r(3).Reflect;t.exports=u&&u.ownKeys||function(t){var n=e.f(o(t)),r=i.f;return r?n.concat(r(t)):n}},function(t,n,r){var e=r(3).parseFloat,i=r(82).trim;t.exports=1/e(r(150)+"-0")!=-1/0?function(t){var n=i(String(t),3),r=e(n);return 0===r&&"-"==n.charAt(0)?-0:r}:e},function(t,n,r){var e=r(3).parseInt,i=r(82).trim,o=r(150),u=/^[\-+]?0[xX]/;t.exports=8!==e(o+"08")||22!==e(o+"0x16")?function(t,n){var r=i(String(t),3);return e(r,n>>>0||(u.test(r)?16:10))}:e},function(t,n){t.exports=Object.is||function(t,n){return t===n?0!==t||1/t==1/n:t!=t&&n!=n}},function(t,n,r){var e=r(16),i=r(149),o=r(46);t.exports=function(t,n,r,u){var c=String(o(t)),f=c.length,a=void 0===r?" ":String(r),s=e(n);if(s<=f||""==a)return c;var l=s-f,h=i.call(a,Math.ceil(l/a.length));return h.length>l&&(h=h.slice(0,l)),u?h+c:c+h}},function(t,n,r){n.f=r(7)},function(t,n,r){"use strict";var e=r(164);t.exports=r(118)("Map",function(t){return function(){return t(this,arguments.length>0?arguments[0]:void 0)}},{get:function(t){var n=e.getEntry(this,t);return n&&n.v},set:function(t,n){return e.def(this,0===t?0:t,n)}},e,!0)},function(t,n,r){r(10)&&"g"!=/./g.flags&&r(11).f(RegExp.prototype,"flags",{configurable:!0,get:r(120)})},function(t,n,r){"use strict";var e=r(164);t.exports=r(118)("Set",function(t){return function(){return t(this,arguments.length>0?arguments[0]:void 0)}},{add:function(t){return e.def(this,t=0===t?0:t,t)}},e)},function(t,n,r){"use strict";var e,i=r(48)(0),o=r(28),u=r(65),c=r(172),f=r(166),a=r(6),s=u.getWeak,l=Object.isExtensible,h=f.ufstore,v={},p=function(t){return function(){return t(this,arguments.length>0?arguments[0]:void 0)}},d={get:function(t){if(a(t)){var n=s(t);return!0===n?h(this).get(t):n?n[this._i]:void 0}},set:function(t,n){return f.def(this,t,n)}},y=t.exports=r(118)("WeakMap",p,d,f,!0,!0);7!=(new y).set((Object.freeze||Object)(v),7).get(v)&&(e=f.getConstructor(p),c(e.prototype,d),u.NEED=!0,i(["delete","has","get","set"],function(t){var n=y.prototype,r=n[t];o(n,t,function(n,i){if(a(n)&&!l(n)){this._f||(this._f=new e);var o=this._f[t](n,i);return"set"==t?this:o}return r.call(this,n,i)})}))},,,,function(t,n){"use strict";function r(){var t=document.querySelector("#page-nav");if(t&&!document.querySelector("#page-nav .extend.prev")&&(t.innerHTML='<a class="extend prev disabled" rel="prev">&laquo; Prev</a>'+t.innerHTML),t&&!document.querySelector("#page-nav .extend.next")&&(t.innerHTML=t.innerHTML+'<a class="extend next disabled" rel="next">Next &raquo;</a>'),yiliaConfig&&yiliaConfig.open_in_new){document.querySelectorAll(".article-entry a:not(.article-more-a)").forEach(function(t){var n=t.getAttribute("target");n&&""!==n||t.setAttribute("target","_blank")})}if(yiliaConfig&&yiliaConfig.toc_hide_index){document.querySelectorAll(".toc-number").forEach(function(t){t.style.display="none"})}var n=document.querySelector("#js-aboutme");n&&0!==n.length&&(n.innerHTML=n.innerText)}t.exports={init:r}},function(t,n,r){"use strict";function e(t){return t&&t.__esModule?t:{default:t}}function i(t,n){var r=/\/|index.html/g;return t.replace(r,"")===n.replace(r,"")}function o(){for(var t=document.querySelectorAll(".js-header-menu li a"),n=window.location.pathname,r=0,e=t.length;r<e;r++){var o=t[r];i(n,o.getAttribute("href"))&&(0,h.default)(o,"active")}}function u(t){for(var n=t.offsetLeft,r=t.offsetParent;null!==r;)n+=r.offsetLeft,r=r.offsetParent;return n}function c(t){for(var n=t.offsetTop,r=t.offsetParent;null!==r;)n+=r.offsetTop,r=r.offsetParent;return n}function f(t,n,r,e,i){var o=u(t),f=c(t)-n;if(f-r<=i){var a=t.$newDom;a||(a=t.cloneNode(!0),(0,d.default)(t,a),t.$newDom=a,a.style.position="fixed",a.style.top=(r||f)+"px",a.style.left=o+"px",a.style.zIndex=e||2,a.style.width="100%",a.style.color="#fff"),a.style.visibility="visible",t.style.visibility="hidden"}else{t.style.visibility="visible";var s=t.$newDom;s&&(s.style.visibility="hidden")}}function a(){var t=document.querySelector(".js-overlay"),n=document.querySelector(".js-header-menu");f(t,document.body.scrollTop,-63,2,0),f(n,document.body.scrollTop,1,3,0)}function s(){document.querySelector("#container").addEventListener("scroll",function(t){a()}),window.addEventListener("scroll",function(t){a()}),a()}var l=r(156),h=e(l),v=r(157),p=(e(v),r(382)),d=e(p),y=r(128),g=e(y),b=r(190),m=e(b),x=r(129);(function(){g.default.versions.mobile&&window.screen.width<800&&(o(),s())})(),(0,x.addLoadEvent)(function(){m.default.init()}),t.exports={}},,,,function(t,n,r){(function(t){"use strict";function n(t,n,r){t[n]||Object[e](t,n,{writable:!0,configurable:!0,value:r})}if(r(381),r(391),r(198),t._babelPolyfill)throw new Error("only one instance of babel-polyfill is allowed");t._babelPolyfill=!0;var e="defineProperty";n(String.prototype,"padLeft","".padStart),n(String.prototype,"padRight","".padEnd),"pop,reverse,shift,keys,values,entries,indexOf,every,some,forEach,map,filter,find,findIndex,includes,join,slice,concat,push,splice,unshift,sort,lastIndexOf,reduce,reduceRight,copyWithin,fill".split(",").forEach(function(t){[][t]&&n(Array,t,Function.call.bind([][t]))})}).call(n,function(){return this}())},,,function(t,n,r){r(210),t.exports=r(52).RegExp.escape},,,,function(t,n,r){var e=r(6),i=r(138),o=r(7)("species");t.exports=function(t){var n;return i(t)&&(n=t.constructor,"function"!=typeof n||n!==Array&&!i(n.prototype)||(n=void 0),e(n)&&null===(n=n[o])&&(n=void 0)),void 0===n?Array:n}},function(t,n,r){var e=r(202);t.exports=function(t,n){return new(e(t))(n)}},function(t,n,r){"use strict";var e=r(2),i=r(50),o="number";t.exports=function(t){if("string"!==t&&t!==o&&"default"!==t)throw TypeError("Incorrect hint");return i(e(this),t!=o)}},function(t,n,r){var e=r(72),i=r(125),o=r(116);t.exports=function(t){var n=e(t),r=i.f;if(r)for(var u,c=r(t),f=o.f,a=0;c.length>a;)f.call(t,u=c[a++])&&n.push(u);return n}},function(t,n,r){var e=r(72),i=r(30);t.exports=function(t,n){for(var r,o=i(t),u=e(o),c=u.length,f=0;c>f;)if(o[r=u[f++]]===n)return r}},function(t,n,r){"use strict";var e=r(208),i=r(121),o=r(26);t.exports=function(){for(var t=o(this),n=arguments.length,r=Array(n),u=0,c=e._,f=!1;n>u;)(r[u]=arguments[u++])===c&&(f=!0);return function(){var e,o=this,u=arguments.length,a=0,s=0;if(!f&&!u)return i(t,r,o);if(e=r.slice(),f)for(;n>a;a++)e[a]===c&&(e[a]=arguments[s++]);for(;u>s;)e.push(arguments[s++]);return i(t,e,o)}}},function(t,n,r){t.exports=r(3)},function(t,n){t.exports=function(t,n){var r=n===Object(n)?function(t){return n[t]}:n;return function(n){return String(n).replace(t,r)}}},function(t,n,r){var e=r(1),i=r(209)(/[\\^$*+?.()|[\]{}]/g,"\\$&");e(e.S,"RegExp",{escape:function(t){return i(t)}})},function(t,n,r){var e=r(1);e(e.P,"Array",{copyWithin:r(160)}),r(78)("copyWithin")},function(t,n,r){"use strict";var e=r(1),i=r(48)(4);e(e.P+e.F*!r(47)([].every,!0),"Array",{every:function(t){return i(this,t,arguments[1])}})},function(t,n,r){var e=r(1);e(e.P,"Array",{fill:r(130)}),r(78)("fill")},function(t,n,r){"use strict";var e=r(1),i=r(48)(2);e(e.P+e.F*!r(47)([].filter,!0),"Array",{filter:function(t){return i(this,t,arguments[1])}})},function(t,n,r){"use strict";var e=r(1),i=r(48)(6),o="findIndex",u=!0;o in[]&&Array(1)[o](function(){u=!1}),e(e.P+e.F*u,"Array",{findIndex:function(t){return i(this,t,arguments.length>1?arguments[1]:void 0)}}),r(78)(o)},function(t,n,r){"use strict";var e=r(1),i=r(48)(5),o="find",u=!0;o in[]&&Array(1)[o](function(){u=!1}),e(e.P+e.F*u,"Array",{find:function(t){return i(this,t,arguments.length>1?arguments[1]:void 0)}}),r(78)(o)},function(t,n,r){"use strict";var e=r(1),i=r(48)(0),o=r(47)([].forEach,!0);e(e.P+e.F*!o,"Array",{forEach:function(t){return i(this,t,arguments[1])}})},function(t,n,r){"use strict";var e=r(53),i=r(1),o=r(17),u=r(169),c=r(137),f=r(16),a=r(131),s=r(154);i(i.S+i.F*!r(123)(function(t){Array.from(t)}),"Array",{from:function(t){var n,r,i,l,h=o(t),v="function"==typeof this?this:Array,p=arguments.length,d=p>1?arguments[1]:void 0,y=void 0!==d,g=0,b=s(h);if(y&&(d=e(d,p>2?arguments[2]:void 0,2)),void 0==b||v==Array&&c(b))for(n=f(h.length),r=new v(n);n>g;g++)a(r,g,y?d(h[g],g):h[g]);else for(l=b.call(h),r=new v;!(i=l.next()).done;g++)a(r,g,y?u(l,d,[i.value,g],!0):i.value);return r.length=g,r}})},function(t,n,r){"use strict";var e=r(1),i=r(117)(!1),o=[].indexOf,u=!!o&&1/[1].indexOf(1,-0)<0;e(e.P+e.F*(u||!r(47)(o)),"Array",{indexOf:function(t){return u?o.apply(this,arguments)||0:i(this,t,arguments[1])}})},function(t,n,r){var e=r(1);e(e.S,"Array",{isArray:r(138)})},function(t,n,r){"use strict";var e=r(1),i=r(30),o=[].join;e(e.P+e.F*(r(115)!=Object||!r(47)(o)),"Array",{join:function(t){return o.call(i(this),void 0===t?",":t)}})},function(t,n,r){"use strict";var e=r(1),i=r(30),o=r(67),u=r(16),c=[].lastIndexOf,f=!!c&&1/[1].lastIndexOf(1,-0)<0;e(e.P+e.F*(f||!r(47)(c)),"Array",{lastIndexOf:function(t){if(f)return c.apply(this,arguments)||0;var n=i(this),r=u(n.length),e=r-1;for(arguments.length>1&&(e=Math.min(e,o(arguments[1]))),e<0&&(e=r+e);e>=0;e--)if(e in n&&n[e]===t)return e||0;return-1}})},function(t,n,r){"use strict";var e=r(1),i=r(48)(1);e(e.P+e.F*!r(47)([].map,!0),"Array",{map:function(t){return i(this,t,arguments[1])}})},function(t,n,r){"use strict";var e=r(1),i=r(131);e(e.S+e.F*r(4)(function(){function t(){}return!(Array.of.call(t)instanceof t)}),"Array",{of:function(){for(var t=0,n=arguments.length,r=new("function"==typeof this?this:Array)(n);n>t;)i(r,t,arguments[t++]);return r.length=n,r}})},function(t,n,r){"use strict";var e=r(1),i=r(162);e(e.P+e.F*!r(47)([].reduceRight,!0),"Array",{reduceRight:function(t){return i(this,t,arguments.length,arguments[1],!0)}})},function(t,n,r){"use strict";var e=r(1),i=r(162);e(e.P+e.F*!r(47)([].reduce,!0),"Array",{reduce:function(t){return i(this,t,arguments.length,arguments[1],!1)}})},function(t,n,r){"use strict";var e=r(1),i=r(135),o=r(45),u=r(75),c=r(16),f=[].slice;e(e.P+e.F*r(4)(function(){i&&f.call(i)}),"Array",{slice:function(t,n){var r=c(this.length),e=o(this);if(n=void 0===n?r:n,"Array"==e)return f.call(this,t,n);for(var i=u(t,r),a=u(n,r),s=c(a-i),l=Array(s),h=0;h<s;h++)l[h]="String"==e?this.charAt(i+h):this[i+h];return l}})},function(t,n,r){"use strict";var e=r(1),i=r(48)(3);e(e.P+e.F*!r(47)([].some,!0),"Array",{some:function(t){return i(this,t,arguments[1])}})},function(t,n,r){"use strict";var e=r(1),i=r(26),o=r(17),u=r(4),c=[].sort,f=[1,2,3];e(e.P+e.F*(u(function(){f.sort(void 0)})||!u(function(){f.sort(null)})||!r(47)(c)),"Array",{sort:function(t){return void 0===t?c.call(o(this)):c.call(o(this),i(t))}})},function(t,n,r){r(74)("Array")},function(t,n,r){var e=r(1);e(e.S,"Date",{now:function(){return(new Date).getTime()}})},function(t,n,r){"use strict";var e=r(1),i=r(4),o=Date.prototype.getTime,u=function(t){return t>9?t:"0"+t};e(e.P+e.F*(i(function(){return"0385-07-25T07:06:39.999Z"!=new Date(-5e13-1).toISOString()})||!i(function(){new Date(NaN).toISOString()})),"Date",{toISOString:function(){
if(!isFinite(o.call(this)))throw RangeError("Invalid time value");var t=this,n=t.getUTCFullYear(),r=t.getUTCMilliseconds(),e=n<0?"-":n>9999?"+":"";return e+("00000"+Math.abs(n)).slice(e?-6:-4)+"-"+u(t.getUTCMonth()+1)+"-"+u(t.getUTCDate())+"T"+u(t.getUTCHours())+":"+u(t.getUTCMinutes())+":"+u(t.getUTCSeconds())+"."+(r>99?r:"0"+u(r))+"Z"}})},function(t,n,r){"use strict";var e=r(1),i=r(17),o=r(50);e(e.P+e.F*r(4)(function(){return null!==new Date(NaN).toJSON()||1!==Date.prototype.toJSON.call({toISOString:function(){return 1}})}),"Date",{toJSON:function(t){var n=i(this),r=o(n);return"number"!=typeof r||isFinite(r)?n.toISOString():null}})},function(t,n,r){var e=r(7)("toPrimitive"),i=Date.prototype;e in i||r(27)(i,e,r(204))},function(t,n,r){var e=Date.prototype,i="Invalid Date",o="toString",u=e[o],c=e.getTime;new Date(NaN)+""!=i&&r(28)(e,o,function(){var t=c.call(this);return t===t?u.call(this):i})},function(t,n,r){var e=r(1);e(e.P,"Function",{bind:r(163)})},function(t,n,r){"use strict";var e=r(6),i=r(32),o=r(7)("hasInstance"),u=Function.prototype;o in u||r(11).f(u,o,{value:function(t){if("function"!=typeof this||!e(t))return!1;if(!e(this.prototype))return t instanceof this;for(;t=i(t);)if(this.prototype===t)return!0;return!1}})},function(t,n,r){var e=r(11).f,i=r(66),o=r(24),u=Function.prototype,c="name",f=Object.isExtensible||function(){return!0};c in u||r(10)&&e(u,c,{configurable:!0,get:function(){try{var t=this,n=(""+t).match(/^\s*function ([^ (]*)/)[1];return o(t,c)||!f(t)||e(t,c,i(5,n)),n}catch(t){return""}}})},function(t,n,r){var e=r(1),i=r(171),o=Math.sqrt,u=Math.acosh;e(e.S+e.F*!(u&&710==Math.floor(u(Number.MAX_VALUE))&&u(1/0)==1/0),"Math",{acosh:function(t){return(t=+t)<1?NaN:t>94906265.62425156?Math.log(t)+Math.LN2:i(t-1+o(t-1)*o(t+1))}})},function(t,n,r){function e(t){return isFinite(t=+t)&&0!=t?t<0?-e(-t):Math.log(t+Math.sqrt(t*t+1)):t}var i=r(1),o=Math.asinh;i(i.S+i.F*!(o&&1/o(0)>0),"Math",{asinh:e})},function(t,n,r){var e=r(1),i=Math.atanh;e(e.S+e.F*!(i&&1/i(-0)<0),"Math",{atanh:function(t){return 0==(t=+t)?t:Math.log((1+t)/(1-t))/2}})},function(t,n,r){var e=r(1),i=r(142);e(e.S,"Math",{cbrt:function(t){return i(t=+t)*Math.pow(Math.abs(t),1/3)}})},function(t,n,r){var e=r(1);e(e.S,"Math",{clz32:function(t){return(t>>>=0)?31-Math.floor(Math.log(t+.5)*Math.LOG2E):32}})},function(t,n,r){var e=r(1),i=Math.exp;e(e.S,"Math",{cosh:function(t){return(i(t=+t)+i(-t))/2}})},function(t,n,r){var e=r(1),i=r(141);e(e.S+e.F*(i!=Math.expm1),"Math",{expm1:i})},function(t,n,r){var e=r(1),i=r(142),o=Math.pow,u=o(2,-52),c=o(2,-23),f=o(2,127)*(2-c),a=o(2,-126),s=function(t){return t+1/u-1/u};e(e.S,"Math",{fround:function(t){var n,r,e=Math.abs(t),o=i(t);return e<a?o*s(e/a/c)*a*c:(n=(1+c/u)*e,r=n-(n-e),r>f||r!=r?o*(1/0):o*r)}})},function(t,n,r){var e=r(1),i=Math.abs;e(e.S,"Math",{hypot:function(t,n){for(var r,e,o=0,u=0,c=arguments.length,f=0;u<c;)r=i(arguments[u++]),f<r?(e=f/r,o=o*e*e+1,f=r):r>0?(e=r/f,o+=e*e):o+=r;return f===1/0?1/0:f*Math.sqrt(o)}})},function(t,n,r){var e=r(1),i=Math.imul;e(e.S+e.F*r(4)(function(){return-5!=i(4294967295,5)||2!=i.length}),"Math",{imul:function(t,n){var r=65535,e=+t,i=+n,o=r&e,u=r&i;return 0|o*u+((r&e>>>16)*u+o*(r&i>>>16)<<16>>>0)}})},function(t,n,r){var e=r(1);e(e.S,"Math",{log10:function(t){return Math.log(t)/Math.LN10}})},function(t,n,r){var e=r(1);e(e.S,"Math",{log1p:r(171)})},function(t,n,r){var e=r(1);e(e.S,"Math",{log2:function(t){return Math.log(t)/Math.LN2}})},function(t,n,r){var e=r(1);e(e.S,"Math",{sign:r(142)})},function(t,n,r){var e=r(1),i=r(141),o=Math.exp;e(e.S+e.F*r(4)(function(){return-2e-17!=!Math.sinh(-2e-17)}),"Math",{sinh:function(t){return Math.abs(t=+t)<1?(i(t)-i(-t))/2:(o(t-1)-o(-t-1))*(Math.E/2)}})},function(t,n,r){var e=r(1),i=r(141),o=Math.exp;e(e.S,"Math",{tanh:function(t){var n=i(t=+t),r=i(-t);return n==1/0?1:r==1/0?-1:(n-r)/(o(t)+o(-t))}})},function(t,n,r){var e=r(1);e(e.S,"Math",{trunc:function(t){return(t>0?Math.floor:Math.ceil)(t)}})},function(t,n,r){"use strict";var e=r(3),i=r(24),o=r(45),u=r(136),c=r(50),f=r(4),a=r(71).f,s=r(31).f,l=r(11).f,h=r(82).trim,v="Number",p=e[v],d=p,y=p.prototype,g=o(r(70)(y))==v,b="trim"in String.prototype,m=function(t){var n=c(t,!1);if("string"==typeof n&&n.length>2){n=b?n.trim():h(n,3);var r,e,i,o=n.charCodeAt(0);if(43===o||45===o){if(88===(r=n.charCodeAt(2))||120===r)return NaN}else if(48===o){switch(n.charCodeAt(1)){case 66:case 98:e=2,i=49;break;case 79:case 111:e=8,i=55;break;default:return+n}for(var u,f=n.slice(2),a=0,s=f.length;a<s;a++)if((u=f.charCodeAt(a))<48||u>i)return NaN;return parseInt(f,e)}}return+n};if(!p(" 0o1")||!p("0b1")||p("+0x1")){p=function(t){var n=arguments.length<1?0:t,r=this;return r instanceof p&&(g?f(function(){y.valueOf.call(r)}):o(r)!=v)?u(new d(m(n)),r,p):m(n)};for(var x,w=r(10)?a(d):"MAX_VALUE,MIN_VALUE,NaN,NEGATIVE_INFINITY,POSITIVE_INFINITY,EPSILON,isFinite,isInteger,isNaN,isSafeInteger,MAX_SAFE_INTEGER,MIN_SAFE_INTEGER,parseFloat,parseInt,isInteger".split(","),S=0;w.length>S;S++)i(d,x=w[S])&&!i(p,x)&&l(p,x,s(d,x));p.prototype=y,y.constructor=p,r(28)(e,v,p)}},function(t,n,r){var e=r(1);e(e.S,"Number",{EPSILON:Math.pow(2,-52)})},function(t,n,r){var e=r(1),i=r(3).isFinite;e(e.S,"Number",{isFinite:function(t){return"number"==typeof t&&i(t)}})},function(t,n,r){var e=r(1);e(e.S,"Number",{isInteger:r(168)})},function(t,n,r){var e=r(1);e(e.S,"Number",{isNaN:function(t){return t!=t}})},function(t,n,r){var e=r(1),i=r(168),o=Math.abs;e(e.S,"Number",{isSafeInteger:function(t){return i(t)&&o(t)<=9007199254740991}})},function(t,n,r){var e=r(1);e(e.S,"Number",{MAX_SAFE_INTEGER:9007199254740991})},function(t,n,r){var e=r(1);e(e.S,"Number",{MIN_SAFE_INTEGER:-9007199254740991})},function(t,n,r){var e=r(1),i=r(178);e(e.S+e.F*(Number.parseFloat!=i),"Number",{parseFloat:i})},function(t,n,r){var e=r(1),i=r(179);e(e.S+e.F*(Number.parseInt!=i),"Number",{parseInt:i})},function(t,n,r){"use strict";var e=r(1),i=r(67),o=r(159),u=r(149),c=1..toFixed,f=Math.floor,a=[0,0,0,0,0,0],s="Number.toFixed: incorrect invocation!",l="0",h=function(t,n){for(var r=-1,e=n;++r<6;)e+=t*a[r],a[r]=e%1e7,e=f(e/1e7)},v=function(t){for(var n=6,r=0;--n>=0;)r+=a[n],a[n]=f(r/t),r=r%t*1e7},p=function(){for(var t=6,n="";--t>=0;)if(""!==n||0===t||0!==a[t]){var r=String(a[t]);n=""===n?r:n+u.call(l,7-r.length)+r}return n},d=function(t,n,r){return 0===n?r:n%2==1?d(t,n-1,r*t):d(t*t,n/2,r)},y=function(t){for(var n=0,r=t;r>=4096;)n+=12,r/=4096;for(;r>=2;)n+=1,r/=2;return n};e(e.P+e.F*(!!c&&("0.000"!==8e-5.toFixed(3)||"1"!==.9.toFixed(0)||"1.25"!==1.255.toFixed(2)||"1000000000000000128"!==(0xde0b6b3a7640080).toFixed(0))||!r(4)(function(){c.call({})})),"Number",{toFixed:function(t){var n,r,e,c,f=o(this,s),a=i(t),g="",b=l;if(a<0||a>20)throw RangeError(s);if(f!=f)return"NaN";if(f<=-1e21||f>=1e21)return String(f);if(f<0&&(g="-",f=-f),f>1e-21)if(n=y(f*d(2,69,1))-69,r=n<0?f*d(2,-n,1):f/d(2,n,1),r*=4503599627370496,(n=52-n)>0){for(h(0,r),e=a;e>=7;)h(1e7,0),e-=7;for(h(d(10,e,1),0),e=n-1;e>=23;)v(1<<23),e-=23;v(1<<e),h(1,1),v(2),b=p()}else h(0,r),h(1<<-n,0),b=p()+u.call(l,a);return a>0?(c=b.length,b=g+(c<=a?"0."+u.call(l,a-c)+b:b.slice(0,c-a)+"."+b.slice(c-a))):b=g+b,b}})},function(t,n,r){"use strict";var e=r(1),i=r(4),o=r(159),u=1..toPrecision;e(e.P+e.F*(i(function(){return"1"!==u.call(1,void 0)})||!i(function(){u.call({})})),"Number",{toPrecision:function(t){var n=o(this,"Number#toPrecision: incorrect invocation!");return void 0===t?u.call(n):u.call(n,t)}})},function(t,n,r){var e=r(1);e(e.S+e.F,"Object",{assign:r(172)})},function(t,n,r){var e=r(1);e(e.S,"Object",{create:r(70)})},function(t,n,r){var e=r(1);e(e.S+e.F*!r(10),"Object",{defineProperties:r(173)})},function(t,n,r){var e=r(1);e(e.S+e.F*!r(10),"Object",{defineProperty:r(11).f})},function(t,n,r){var e=r(6),i=r(65).onFreeze;r(49)("freeze",function(t){return function(n){return t&&e(n)?t(i(n)):n}})},function(t,n,r){var e=r(30),i=r(31).f;r(49)("getOwnPropertyDescriptor",function(){return function(t,n){return i(e(t),n)}})},function(t,n,r){r(49)("getOwnPropertyNames",function(){return r(174).f})},function(t,n,r){var e=r(17),i=r(32);r(49)("getPrototypeOf",function(){return function(t){return i(e(t))}})},function(t,n,r){var e=r(6);r(49)("isExtensible",function(t){return function(n){return!!e(n)&&(!t||t(n))}})},function(t,n,r){var e=r(6);r(49)("isFrozen",function(t){return function(n){return!e(n)||!!t&&t(n)}})},function(t,n,r){var e=r(6);r(49)("isSealed",function(t){return function(n){return!e(n)||!!t&&t(n)}})},function(t,n,r){var e=r(1);e(e.S,"Object",{is:r(180)})},function(t,n,r){var e=r(17),i=r(72);r(49)("keys",function(){return function(t){return i(e(t))}})},function(t,n,r){var e=r(6),i=r(65).onFreeze;r(49)("preventExtensions",function(t){return function(n){return t&&e(n)?t(i(n)):n}})},function(t,n,r){var e=r(6),i=r(65).onFreeze;r(49)("seal",function(t){return function(n){return t&&e(n)?t(i(n)):n}})},function(t,n,r){var e=r(1);e(e.S,"Object",{setPrototypeOf:r(144).set})},function(t,n,r){"use strict";var e=r(114),i={};i[r(7)("toStringTag")]="z",i+""!="[object z]"&&r(28)(Object.prototype,"toString",function(){return"[object "+e(this)+"]"},!0)},function(t,n,r){var e=r(1),i=r(178);e(e.G+e.F*(parseFloat!=i),{parseFloat:i})},function(t,n,r){var e=r(1),i=r(179);e(e.G+e.F*(parseInt!=i),{parseInt:i})},function(t,n,r){"use strict";var e,i,o,u=r(69),c=r(3),f=r(53),a=r(114),s=r(1),l=r(6),h=r(26),v=r(68),p=r(79),d=r(146),y=r(151).set,g=r(143)(),b="Promise",m=c.TypeError,x=c.process,w=c[b],x=c.process,S="process"==a(x),_=function(){},O=!!function(){try{var t=w.resolve(1),n=(t.constructor={})[r(7)("species")]=function(t){t(_,_)};return(S||"function"==typeof PromiseRejectionEvent)&&t.then(_)instanceof n}catch(t){}}(),E=function(t,n){return t===n||t===w&&n===o},P=function(t){var n;return!(!l(t)||"function"!=typeof(n=t.then))&&n},j=function(t){return E(w,t)?new F(t):new i(t)},F=i=function(t){var n,r;this.promise=new t(function(t,e){if(void 0!==n||void 0!==r)throw m("Bad Promise constructor");n=t,r=e}),this.resolve=h(n),this.reject=h(r)},M=function(t){try{t()}catch(t){return{error:t}}},A=function(t,n){if(!t._n){t._n=!0;var r=t._c;g(function(){for(var e=t._v,i=1==t._s,o=0;r.length>o;)!function(n){var r,o,u=i?n.ok:n.fail,c=n.resolve,f=n.reject,a=n.domain;try{u?(i||(2==t._h&&I(t),t._h=1),!0===u?r=e:(a&&a.enter(),r=u(e),a&&a.exit()),r===n.promise?f(m("Promise-chain cycle")):(o=P(r))?o.call(r,c,f):c(r)):f(e)}catch(t){f(t)}}(r[o++]);t._c=[],t._n=!1,n&&!t._h&&N(t)})}},N=function(t){y.call(c,function(){var n,r,e,i=t._v;if(T(t)&&(n=M(function(){S?x.emit("unhandledRejection",i,t):(r=c.onunhandledrejection)?r({promise:t,reason:i}):(e=c.console)&&e.error&&e.error("Unhandled promise rejection",i)}),t._h=S||T(t)?2:1),t._a=void 0,n)throw n.error})},T=function(t){if(1==t._h)return!1;for(var n,r=t._a||t._c,e=0;r.length>e;)if(n=r[e++],n.fail||!T(n.promise))return!1;return!0},I=function(t){y.call(c,function(){var n;S?x.emit("rejectionHandled",t):(n=c.onrejectionhandled)&&n({promise:t,reason:t._v})})},k=function(t){var n=this;n._d||(n._d=!0,n=n._w||n,n._v=t,n._s=2,n._a||(n._a=n._c.slice()),A(n,!0))},L=function(t){var n,r=this;if(!r._d){r._d=!0,r=r._w||r;try{if(r===t)throw m("Promise can't be resolved itself");(n=P(t))?g(function(){var e={_w:r,_d:!1};try{n.call(t,f(L,e,1),f(k,e,1))}catch(t){k.call(e,t)}}):(r._v=t,r._s=1,A(r,!1))}catch(t){k.call({_w:r,_d:!1},t)}}};O||(w=function(t){v(this,w,b,"_h"),h(t),e.call(this);try{t(f(L,this,1),f(k,this,1))}catch(t){k.call(this,t)}},e=function(t){this._c=[],this._a=void 0,this._s=0,this._d=!1,this._v=void 0,this._h=0,this._n=!1},e.prototype=r(73)(w.prototype,{then:function(t,n){var r=j(d(this,w));return r.ok="function"!=typeof t||t,r.fail="function"==typeof n&&n,r.domain=S?x.domain:void 0,this._c.push(r),this._a&&this._a.push(r),this._s&&A(this,!1),r.promise},catch:function(t){return this.then(void 0,t)}}),F=function(){var t=new e;this.promise=t,this.resolve=f(L,t,1),this.reject=f(k,t,1)}),s(s.G+s.W+s.F*!O,{Promise:w}),r(81)(w,b),r(74)(b),o=r(52)[b],s(s.S+s.F*!O,b,{reject:function(t){var n=j(this);return(0,n.reject)(t),n.promise}}),s(s.S+s.F*(u||!O),b,{resolve:function(t){if(t instanceof w&&E(t.constructor,this))return t;var n=j(this);return(0,n.resolve)(t),n.promise}}),s(s.S+s.F*!(O&&r(123)(function(t){w.all(t).catch(_)})),b,{all:function(t){var n=this,r=j(n),e=r.resolve,i=r.reject,o=M(function(){var r=[],o=0,u=1;p(t,!1,function(t){var c=o++,f=!1;r.push(void 0),u++,n.resolve(t).then(function(t){f||(f=!0,r[c]=t,--u||e(r))},i)}),--u||e(r)});return o&&i(o.error),r.promise},race:function(t){var n=this,r=j(n),e=r.reject,i=M(function(){p(t,!1,function(t){n.resolve(t).then(r.resolve,e)})});return i&&e(i.error),r.promise}})},function(t,n,r){var e=r(1),i=r(26),o=r(2),u=(r(3).Reflect||{}).apply,c=Function.apply;e(e.S+e.F*!r(4)(function(){u(function(){})}),"Reflect",{apply:function(t,n,r){var e=i(t),f=o(r);return u?u(e,n,f):c.call(e,n,f)}})},function(t,n,r){var e=r(1),i=r(70),o=r(26),u=r(2),c=r(6),f=r(4),a=r(163),s=(r(3).Reflect||{}).construct,l=f(function(){function t(){}return!(s(function(){},[],t)instanceof t)}),h=!f(function(){s(function(){})});e(e.S+e.F*(l||h),"Reflect",{construct:function(t,n){o(t),u(n);var r=arguments.length<3?t:o(arguments[2]);if(h&&!l)return s(t,n,r);if(t==r){switch(n.length){case 0:return new t;case 1:return new t(n[0]);case 2:return new t(n[0],n[1]);case 3:return new t(n[0],n[1],n[2]);case 4:return new t(n[0],n[1],n[2],n[3])}var e=[null];return e.push.apply(e,n),new(a.apply(t,e))}var f=r.prototype,v=i(c(f)?f:Object.prototype),p=Function.apply.call(t,v,n);return c(p)?p:v}})},function(t,n,r){var e=r(11),i=r(1),o=r(2),u=r(50);i(i.S+i.F*r(4)(function(){Reflect.defineProperty(e.f({},1,{value:1}),1,{value:2})}),"Reflect",{defineProperty:function(t,n,r){o(t),n=u(n,!0),o(r);try{return e.f(t,n,r),!0}catch(t){return!1}}})},function(t,n,r){var e=r(1),i=r(31).f,o=r(2);e(e.S,"Reflect",{deleteProperty:function(t,n){var r=i(o(t),n);return!(r&&!r.configurable)&&delete t[n]}})},function(t,n,r){"use strict";var e=r(1),i=r(2),o=function(t){this._t=i(t),this._i=0;var n,r=this._k=[];for(n in t)r.push(n)};r(139)(o,"Object",function(){var t,n=this,r=n._k;do{if(n._i>=r.length)return{value:void 0,done:!0}}while(!((t=r[n._i++])in n._t));return{value:t,done:!1}}),e(e.S,"Reflect",{enumerate:function(t){return new o(t)}})},function(t,n,r){var e=r(31),i=r(1),o=r(2);i(i.S,"Reflect",{getOwnPropertyDescriptor:function(t,n){return e.f(o(t),n)}})},function(t,n,r){var e=r(1),i=r(32),o=r(2);e(e.S,"Reflect",{getPrototypeOf:function(t){return i(o(t))}})},function(t,n,r){function e(t,n){var r,c,s=arguments.length<3?t:arguments[2];return a(t)===s?t[n]:(r=i.f(t,n))?u(r,"value")?r.value:void 0!==r.get?r.get.call(s):void 0:f(c=o(t))?e(c,n,s):void 0}var i=r(31),o=r(32),u=r(24),c=r(1),f=r(6),a=r(2);c(c.S,"Reflect",{get:e})},function(t,n,r){var e=r(1);e(e.S,"Reflect",{has:function(t,n){return n in t}})},function(t,n,r){var e=r(1),i=r(2),o=Object.isExtensible;e(e.S,"Reflect",{isExtensible:function(t){return i(t),!o||o(t)}})},function(t,n,r){var e=r(1);e(e.S,"Reflect",{ownKeys:r(177)})},function(t,n,r){var e=r(1),i=r(2),o=Object.preventExtensions;e(e.S,"Reflect",{preventExtensions:function(t){i(t);try{return o&&o(t),!0}catch(t){return!1}}})},function(t,n,r){var e=r(1),i=r(144);i&&e(e.S,"Reflect",{setPrototypeOf:function(t,n){i.check(t,n);try{return i.set(t,n),!0}catch(t){return!1}}})},function(t,n,r){function e(t,n,r){var f,h,v=arguments.length<4?t:arguments[3],p=o.f(s(t),n);if(!p){if(l(h=u(t)))return e(h,n,r,v);p=a(0)}return c(p,"value")?!(!1===p.writable||!l(v)||(f=o.f(v,n)||a(0),f.value=r,i.f(v,n,f),0)):void 0!==p.set&&(p.set.call(v,r),!0)}var i=r(11),o=r(31),u=r(32),c=r(24),f=r(1),a=r(66),s=r(2),l=r(6);f(f.S,"Reflect",{set:e})},function(t,n,r){var e=r(3),i=r(136),o=r(11).f,u=r(71).f,c=r(122),f=r(120),a=e.RegExp,s=a,l=a.prototype,h=/a/g,v=/a/g,p=new a(h)!==h;if(r(10)&&(!p||r(4)(function(){return v[r(7)("match")]=!1,a(h)!=h||a(v)==v||"/a/i"!=a(h,"i")}))){a=function(t,n){var r=this instanceof a,e=c(t),o=void 0===n;return!r&&e&&t.constructor===a&&o?t:i(p?new s(e&&!o?t.source:t,n):s((e=t instanceof a)?t.source:t,e&&o?f.call(t):n),r?this:l,a)};for(var d=u(s),y=0;d.length>y;)!function(t){t in a||o(a,t,{configurable:!0,get:function(){return s[t]},set:function(n){s[t]=n}})}(d[y++]);l.constructor=a,a.prototype=l,r(28)(e,"RegExp",a)}r(74)("RegExp")},function(t,n,r){r(119)("match",1,function(t,n,r){return[function(r){"use strict";var e=t(this),i=void 0==r?void 0:r[n];return void 0!==i?i.call(r,e):new RegExp(r)[n](String(e))},r]})},function(t,n,r){r(119)("replace",2,function(t,n,r){return[function(e,i){"use strict";var o=t(this),u=void 0==e?void 0:e[n];return void 0!==u?u.call(e,o,i):r.call(String(o),e,i)},r]})},function(t,n,r){r(119)("search",1,function(t,n,r){return[function(r){"use strict";var e=t(this),i=void 0==r?void 0:r[n];return void 0!==i?i.call(r,e):new RegExp(r)[n](String(e))},r]})},function(t,n,r){r(119)("split",2,function(t,n,e){"use strict";var i=r(122),o=e,u=[].push,c="split",f="length",a="lastIndex";if("c"=="abbc"[c](/(b)*/)[1]||4!="test"[c](/(?:)/,-1)[f]||2!="ab"[c](/(?:ab)*/)[f]||4!="."[c](/(.?)(.?)/)[f]||"."[c](/()()/)[f]>1||""[c](/.?/)[f]){var s=void 0===/()??/.exec("")[1];e=function(t,n){var r=String(this);if(void 0===t&&0===n)return[];if(!i(t))return o.call(r,t,n);var e,c,l,h,v,p=[],d=(t.ignoreCase?"i":"")+(t.multiline?"m":"")+(t.unicode?"u":"")+(t.sticky?"y":""),y=0,g=void 0===n?4294967295:n>>>0,b=new RegExp(t.source,d+"g");for(s||(e=new RegExp("^"+b.source+"$(?!\\s)",d));(c=b.exec(r))&&!((l=c.index+c[0][f])>y&&(p.push(r.slice(y,c.index)),!s&&c[f]>1&&c[0].replace(e,function(){for(v=1;v<arguments[f]-2;v++)void 0===arguments[v]&&(c[v]=void 0)}),c[f]>1&&c.index<r[f]&&u.apply(p,c.slice(1)),h=c[0][f],y=l,p[f]>=g));)b[a]===c.index&&b[a]++;return y===r[f]?!h&&b.test("")||p.push(""):p.push(r.slice(y)),p[f]>g?p.slice(0,g):p}}else"0"[c](void 0,0)[f]&&(e=function(t,n){return void 0===t&&0===n?[]:o.call(this,t,n)});return[function(r,i){var o=t(this),u=void 0==r?void 0:r[n];return void 0!==u?u.call(r,o,i):e.call(String(o),r,i)},e]})},function(t,n,r){"use strict";r(184);var e=r(2),i=r(120),o=r(10),u="toString",c=/./[u],f=function(t){r(28)(RegExp.prototype,u,t,!0)};r(4)(function(){return"/a/b"!=c.call({source:"a",flags:"b"})})?f(function(){var t=e(this);return"/".concat(t.source,"/","flags"in t?t.flags:!o&&t instanceof RegExp?i.call(t):void 0)}):c.name!=u&&f(function(){return c.call(this)})},function(t,n,r){"use strict";r(29)("anchor",function(t){return function(n){return t(this,"a","name",n)}})},function(t,n,r){"use strict";r(29)("big",function(t){return function(){return t(this,"big","","")}})},function(t,n,r){"use strict";r(29)("blink",function(t){return function(){return t(this,"blink","","")}})},function(t,n,r){"use strict";r(29)("bold",function(t){return function(){return t(this,"b","","")}})},function(t,n,r){"use strict";var e=r(1),i=r(147)(!1);e(e.P,"String",{codePointAt:function(t){return i(this,t)}})},function(t,n,r){"use strict";var e=r(1),i=r(16),o=r(148),u="endsWith",c=""[u];e(e.P+e.F*r(134)(u),"String",{endsWith:function(t){var n=o(this,t,u),r=arguments.length>1?arguments[1]:void 0,e=i(n.length),f=void 0===r?e:Math.min(i(r),e),a=String(t);return c?c.call(n,a,f):n.slice(f-a.length,f)===a}})},function(t,n,r){"use strict";r(29)("fixed",function(t){return function(){return t(this,"tt","","")}})},function(t,n,r){"use strict";r(29)("fontcolor",function(t){return function(n){return t(this,"font","color",n)}})},function(t,n,r){"use strict";r(29)("fontsize",function(t){return function(n){return t(this,"font","size",n)}})},function(t,n,r){var e=r(1),i=r(75),o=String.fromCharCode,u=String.fromCodePoint;e(e.S+e.F*(!!u&&1!=u.length),"String",{fromCodePoint:function(t){for(var n,r=[],e=arguments.length,u=0;e>u;){if(n=+arguments[u++],i(n,1114111)!==n)throw RangeError(n+" is not a valid code point");r.push(n<65536?o(n):o(55296+((n-=65536)>>10),n%1024+56320))}return r.join("")}})},function(t,n,r){"use strict";var e=r(1),i=r(148),o="includes";e(e.P+e.F*r(134)(o),"String",{includes:function(t){return!!~i(this,t,o).indexOf(t,arguments.length>1?arguments[1]:void 0)}})},function(t,n,r){"use strict";r(29)("italics",function(t){return function(){return t(this,"i","","")}})},function(t,n,r){"use strict";var e=r(147)(!0);r(140)(String,"String",function(t){this._t=String(t),this._i=0},function(){var t,n=this._t,r=this._i;return r>=n.length?{value:void 0,done:!0}:(t=e(n,r),this._i+=t.length,{value:t,done:!1})})},function(t,n,r){"use strict";r(29)("link",function(t){return function(n){return t(this,"a","href",n)}})},function(t,n,r){var e=r(1),i=r(30),o=r(16);e(e.S,"String",{raw:function(t){for(var n=i(t.raw),r=o(n.length),e=arguments.length,u=[],c=0;r>c;)u.push(String(n[c++])),c<e&&u.push(String(arguments[c]));return u.join("")}})},function(t,n,r){var e=r(1);e(e.P,"String",{repeat:r(149)})},function(t,n,r){"use strict";r(29)("small",function(t){return function(){return t(this,"small","","")}})},function(t,n,r){"use strict";var e=r(1),i=r(16),o=r(148),u="startsWith",c=""[u];e(e.P+e.F*r(134)(u),"String",{startsWith:function(t){var n=o(this,t,u),r=i(Math.min(arguments.length>1?arguments[1]:void 0,n.length)),e=String(t);return c?c.call(n,e,r):n.slice(r,r+e.length)===e}})},function(t,n,r){"use strict";r(29)("strike",function(t){return function(){return t(this,"strike","","")}})},function(t,n,r){"use strict";r(29)("sub",function(t){return function(){return t(this,"sub","","")}})},function(t,n,r){"use strict";r(29)("sup",function(t){return function(){return t(this,"sup","","")}})},function(t,n,r){"use strict";r(82)("trim",function(t){return function(){return t(this,3)}})},function(t,n,r){"use strict";var e=r(3),i=r(24),o=r(10),u=r(1),c=r(28),f=r(65).KEY,a=r(4),s=r(126),l=r(81),h=r(76),v=r(7),p=r(182),d=r(153),y=r(206),g=r(205),b=r(138),m=r(2),x=r(30),w=r(50),S=r(66),_=r(70),O=r(174),E=r(31),P=r(11),j=r(72),F=E.f,M=P.f,A=O.f,N=e.Symbol,T=e.JSON,I=T&&T.stringify,k="prototype",L=v("_hidden"),R=v("toPrimitive"),C={}.propertyIsEnumerable,D=s("symbol-registry"),U=s("symbols"),W=s("op-symbols"),G=Object[k],B="function"==typeof N,V=e.QObject,z=!V||!V[k]||!V[k].findChild,q=o&&a(function(){return 7!=_(M({},"a",{get:function(){return M(this,"a",{value:7}).a}})).a})?function(t,n,r){var e=F(G,n);e&&delete G[n],M(t,n,r),e&&t!==G&&M(G,n,e)}:M,K=function(t){var n=U[t]=_(N[k]);return n._k=t,n},J=B&&"symbol"==typeof N.iterator?function(t){return"symbol"==typeof t}:function(t){return t instanceof N},Y=function(t,n,r){return t===G&&Y(W,n,r),m(t),n=w(n,!0),m(r),i(U,n)?(r.enumerable?(i(t,L)&&t[L][n]&&(t[L][n]=!1),r=_(r,{enumerable:S(0,!1)})):(i(t,L)||M(t,L,S(1,{})),t[L][n]=!0),q(t,n,r)):M(t,n,r)},H=function(t,n){m(t);for(var r,e=g(n=x(n)),i=0,o=e.length;o>i;)Y(t,r=e[i++],n[r]);return t},$=function(t,n){return void 0===n?_(t):H(_(t),n)},X=function(t){var n=C.call(this,t=w(t,!0));return!(this===G&&i(U,t)&&!i(W,t))&&(!(n||!i(this,t)||!i(U,t)||i(this,L)&&this[L][t])||n)},Q=function(t,n){if(t=x(t),n=w(n,!0),t!==G||!i(U,n)||i(W,n)){var r=F(t,n);return!r||!i(U,n)||i(t,L)&&t[L][n]||(r.enumerable=!0),r}},Z=function(t){for(var n,r=A(x(t)),e=[],o=0;r.length>o;)i(U,n=r[o++])||n==L||n==f||e.push(n);return e},tt=function(t){for(var n,r=t===G,e=A(r?W:x(t)),o=[],u=0;e.length>u;)!i(U,n=e[u++])||r&&!i(G,n)||o.push(U[n]);return o};B||(N=function(){if(this instanceof N)throw TypeError("Symbol is not a constructor!");var t=h(arguments.length>0?arguments[0]:void 0),n=function(r){this===G&&n.call(W,r),i(this,L)&&i(this[L],t)&&(this[L][t]=!1),q(this,t,S(1,r))};return o&&z&&q(G,t,{configurable:!0,set:n}),K(t)},c(N[k],"toString",function(){return this._k}),E.f=Q,P.f=Y,r(71).f=O.f=Z,r(116).f=X,r(125).f=tt,o&&!r(69)&&c(G,"propertyIsEnumerable",X,!0),p.f=function(t){return K(v(t))}),u(u.G+u.W+u.F*!B,{Symbol:N});for(var nt="hasInstance,isConcatSpreadable,iterator,match,replace,search,species,split,toPrimitive,toStringTag,unscopables".split(","),rt=0;nt.length>rt;)v(nt[rt++]);for(var nt=j(v.store),rt=0;nt.length>rt;)d(nt[rt++]);u(u.S+u.F*!B,"Symbol",{for:function(t){return i(D,t+="")?D[t]:D[t]=N(t)},keyFor:function(t){if(J(t))return y(D,t);throw TypeError(t+" is not a symbol!")},useSetter:function(){z=!0},useSimple:function(){z=!1}}),u(u.S+u.F*!B,"Object",{create:$,defineProperty:Y,defineProperties:H,getOwnPropertyDescriptor:Q,getOwnPropertyNames:Z,getOwnPropertySymbols:tt}),T&&u(u.S+u.F*(!B||a(function(){var t=N();return"[null]"!=I([t])||"{}"!=I({a:t})||"{}"!=I(Object(t))})),"JSON",{stringify:function(t){if(void 0!==t&&!J(t)){for(var n,r,e=[t],i=1;arguments.length>i;)e.push(arguments[i++]);return n=e[1],"function"==typeof n&&(r=n),!r&&b(n)||(n=function(t,n){if(r&&(n=r.call(this,t,n)),!J(n))return n}),e[1]=n,I.apply(T,e)}}}),N[k][R]||r(27)(N[k],R,N[k].valueOf),l(N,"Symbol"),l(Math,"Math",!0),l(e.JSON,"JSON",!0)},function(t,n,r){"use strict";var e=r(1),i=r(127),o=r(152),u=r(2),c=r(75),f=r(16),a=r(6),s=r(3).ArrayBuffer,l=r(146),h=o.ArrayBuffer,v=o.DataView,p=i.ABV&&s.isView,d=h.prototype.slice,y=i.VIEW,g="ArrayBuffer";e(e.G+e.W+e.F*(s!==h),{ArrayBuffer:h}),e(e.S+e.F*!i.CONSTR,g,{isView:function(t){return p&&p(t)||a(t)&&y in t}}),e(e.P+e.U+e.F*r(4)(function(){return!new h(2).slice(1,void 0).byteLength}),g,{slice:function(t,n){if(void 0!==d&&void 0===n)return d.call(u(this),t);for(var r=u(this).byteLength,e=c(t,r),i=c(void 0===n?r:n,r),o=new(l(this,h))(f(i-e)),a=new v(this),s=new v(o),p=0;e<i;)s.setUint8(p++,a.getUint8(e++));return o}}),r(74)(g)},function(t,n,r){var e=r(1);e(e.G+e.W+e.F*!r(127).ABV,{DataView:r(152).DataView})},function(t,n,r){r(55)("Float32",4,function(t){return function(n,r,e){return t(this,n,r,e)}})},function(t,n,r){r(55)("Float64",8,function(t){return function(n,r,e){return t(this,n,r,e)}})},function(t,n,r){r(55)("Int16",2,function(t){return function(n,r,e){return t(this,n,r,e)}})},function(t,n,r){r(55)("Int32",4,function(t){return function(n,r,e){return t(this,n,r,e)}})},function(t,n,r){r(55)("Int8",1,function(t){return function(n,r,e){return t(this,n,r,e)}})},function(t,n,r){r(55)("Uint16",2,function(t){return function(n,r,e){return t(this,n,r,e)}})},function(t,n,r){r(55)("Uint32",4,function(t){return function(n,r,e){return t(this,n,r,e)}})},function(t,n,r){r(55)("Uint8",1,function(t){return function(n,r,e){return t(this,n,r,e)}})},function(t,n,r){r(55)("Uint8",1,function(t){return function(n,r,e){return t(this,n,r,e)}},!0)},function(t,n,r){"use strict";var e=r(166);r(118)("WeakSet",function(t){return function(){return t(this,arguments.length>0?arguments[0]:void 0)}},{add:function(t){return e.def(this,t,!0)}},e,!1,!0)},function(t,n,r){"use strict";var e=r(1),i=r(117)(!0);e(e.P,"Array",{includes:function(t){return i(this,t,arguments.length>1?arguments[1]:void 0)}}),r(78)("includes")},function(t,n,r){var e=r(1),i=r(143)(),o=r(3).process,u="process"==r(45)(o);e(e.G,{asap:function(t){var n=u&&o.domain;i(n?n.bind(t):t)}})},function(t,n,r){var e=r(1),i=r(45);e(e.S,"Error",{isError:function(t){return"Error"===i(t)}})},function(t,n,r){var e=r(1);e(e.P+e.R,"Map",{toJSON:r(165)("Map")})},function(t,n,r){var e=r(1);e(e.S,"Math",{iaddh:function(t,n,r,e){var i=t>>>0,o=n>>>0,u=r>>>0;return o+(e>>>0)+((i&u|(i|u)&~(i+u>>>0))>>>31)|0}})},function(t,n,r){var e=r(1);e(e.S,"Math",{imulh:function(t,n){var r=65535,e=+t,i=+n,o=e&r,u=i&r,c=e>>16,f=i>>16,a=(c*u>>>0)+(o*u>>>16);return c*f+(a>>16)+((o*f>>>0)+(a&r)>>16)}})},function(t,n,r){var e=r(1);e(e.S,"Math",{isubh:function(t,n,r,e){var i=t>>>0,o=n>>>0,u=r>>>0;return o-(e>>>0)-((~i&u|~(i^u)&i-u>>>0)>>>31)|0}})},function(t,n,r){var e=r(1);e(e.S,"Math",{umulh:function(t,n){var r=65535,e=+t,i=+n,o=e&r,u=i&r,c=e>>>16,f=i>>>16,a=(c*u>>>0)+(o*u>>>16);return c*f+(a>>>16)+((o*f>>>0)+(a&r)>>>16)}})},function(t,n,r){"use strict";var e=r(1),i=r(17),o=r(26),u=r(11);r(10)&&e(e.P+r(124),"Object",{__defineGetter__:function(t,n){u.f(i(this),t,{get:o(n),enumerable:!0,configurable:!0})}})},function(t,n,r){"use strict";var e=r(1),i=r(17),o=r(26),u=r(11);r(10)&&e(e.P+r(124),"Object",{__defineSetter__:function(t,n){u.f(i(this),t,{set:o(n),enumerable:!0,configurable:!0})}})},function(t,n,r){var e=r(1),i=r(176)(!0);e(e.S,"Object",{entries:function(t){return i(t)}})},function(t,n,r){var e=r(1),i=r(177),o=r(30),u=r(31),c=r(131);e(e.S,"Object",{getOwnPropertyDescriptors:function(t){for(var n,r=o(t),e=u.f,f=i(r),a={},s=0;f.length>s;)c(a,n=f[s++],e(r,n));return a}})},function(t,n,r){"use strict";var e=r(1),i=r(17),o=r(50),u=r(32),c=r(31).f;r(10)&&e(e.P+r(124),"Object",{__lookupGetter__:function(t){var n,r=i(this),e=o(t,!0);do{if(n=c(r,e))return n.get}while(r=u(r))}})},function(t,n,r){"use strict";var e=r(1),i=r(17),o=r(50),u=r(32),c=r(31).f;r(10)&&e(e.P+r(124),"Object",{__lookupSetter__:function(t){var n,r=i(this),e=o(t,!0);do{if(n=c(r,e))return n.set}while(r=u(r))}})},function(t,n,r){var e=r(1),i=r(176)(!1);e(e.S,"Object",{values:function(t){return i(t)}})},function(t,n,r){"use strict";var e=r(1),i=r(3),o=r(52),u=r(143)(),c=r(7)("observable"),f=r(26),a=r(2),s=r(68),l=r(73),h=r(27),v=r(79),p=v.RETURN,d=function(t){return null==t?void 0:f(t)},y=function(t){var n=t._c;n&&(t._c=void 0,n())},g=function(t){return void 0===t._o},b=function(t){g(t)||(t._o=void 0,y(t))},m=function(t,n){a(t),this._c=void 0,this._o=t,t=new x(this);try{var r=n(t),e=r;null!=r&&("function"==typeof r.unsubscribe?r=function(){e.unsubscribe()}:f(r),this._c=r)}catch(n){return void t.error(n)}g(this)&&y(this)};m.prototype=l({},{unsubscribe:function(){b(this)}});var x=function(t){this._s=t};x.prototype=l({},{next:function(t){var n=this._s;if(!g(n)){var r=n._o;try{var e=d(r.next);if(e)return e.call(r,t)}catch(t){try{b(n)}finally{throw t}}}},error:function(t){var n=this._s;if(g(n))throw t;var r=n._o;n._o=void 0;try{var e=d(r.error);if(!e)throw t;t=e.call(r,t)}catch(t){try{y(n)}finally{throw t}}return y(n),t},complete:function(t){var n=this._s;if(!g(n)){var r=n._o;n._o=void 0;try{var e=d(r.complete);t=e?e.call(r,t):void 0}catch(t){try{y(n)}finally{throw t}}return y(n),t}}});var w=function(t){s(this,w,"Observable","_f")._f=f(t)};l(w.prototype,{subscribe:function(t){return new m(t,this._f)},forEach:function(t){var n=this;return new(o.Promise||i.Promise)(function(r,e){f(t);var i=n.subscribe({next:function(n){try{return t(n)}catch(t){e(t),i.unsubscribe()}},error:e,complete:r})})}}),l(w,{from:function(t){var n="function"==typeof this?this:w,r=d(a(t)[c]);if(r){var e=a(r.call(t));return e.constructor===n?e:new n(function(t){return e.subscribe(t)})}return new n(function(n){var r=!1;return u(function(){if(!r){try{if(v(t,!1,function(t){if(n.next(t),r)return p})===p)return}catch(t){if(r)throw t;return void n.error(t)}n.complete()}}),function(){r=!0}})},of:function(){for(var t=0,n=arguments.length,r=Array(n);t<n;)r[t]=arguments[t++];return new("function"==typeof this?this:w)(function(t){var n=!1;return u(function(){if(!n){for(var e=0;e<r.length;++e)if(t.next(r[e]),n)return;t.complete()}}),function(){n=!0}})}}),h(w.prototype,c,function(){return this}),e(e.G,{Observable:w}),r(74)("Observable")},function(t,n,r){var e=r(54),i=r(2),o=e.key,u=e.set;e.exp({defineMetadata:function(t,n,r,e){u(t,n,i(r),o(e))}})},function(t,n,r){var e=r(54),i=r(2),o=e.key,u=e.map,c=e.store;e.exp({deleteMetadata:function(t,n){var r=arguments.length<3?void 0:o(arguments[2]),e=u(i(n),r,!1);if(void 0===e||!e.delete(t))return!1;if(e.size)return!0;var f=c.get(n);return f.delete(r),!!f.size||c.delete(n)}})},function(t,n,r){var e=r(185),i=r(161),o=r(54),u=r(2),c=r(32),f=o.keys,a=o.key,s=function(t,n){var r=f(t,n),o=c(t);if(null===o)return r;var u=s(o,n);return u.length?r.length?i(new e(r.concat(u))):u:r};o.exp({getMetadataKeys:function(t){return s(u(t),arguments.length<2?void 0:a(arguments[1]))}})},function(t,n,r){var e=r(54),i=r(2),o=r(32),u=e.has,c=e.get,f=e.key,a=function(t,n,r){if(u(t,n,r))return c(t,n,r);var e=o(n);return null!==e?a(t,e,r):void 0};e.exp({getMetadata:function(t,n){return a(t,i(n),arguments.length<3?void 0:f(arguments[2]))}})},function(t,n,r){var e=r(54),i=r(2),o=e.keys,u=e.key;e.exp({getOwnMetadataKeys:function(t){
return o(i(t),arguments.length<2?void 0:u(arguments[1]))}})},function(t,n,r){var e=r(54),i=r(2),o=e.get,u=e.key;e.exp({getOwnMetadata:function(t,n){return o(t,i(n),arguments.length<3?void 0:u(arguments[2]))}})},function(t,n,r){var e=r(54),i=r(2),o=r(32),u=e.has,c=e.key,f=function(t,n,r){if(u(t,n,r))return!0;var e=o(n);return null!==e&&f(t,e,r)};e.exp({hasMetadata:function(t,n){return f(t,i(n),arguments.length<3?void 0:c(arguments[2]))}})},function(t,n,r){var e=r(54),i=r(2),o=e.has,u=e.key;e.exp({hasOwnMetadata:function(t,n){return o(t,i(n),arguments.length<3?void 0:u(arguments[2]))}})},function(t,n,r){var e=r(54),i=r(2),o=r(26),u=e.key,c=e.set;e.exp({metadata:function(t,n){return function(r,e){c(t,n,(void 0!==e?i:o)(r),u(e))}}})},function(t,n,r){var e=r(1);e(e.P+e.R,"Set",{toJSON:r(165)("Set")})},function(t,n,r){"use strict";var e=r(1),i=r(147)(!0);e(e.P,"String",{at:function(t){return i(this,t)}})},function(t,n,r){"use strict";var e=r(1),i=r(46),o=r(16),u=r(122),c=r(120),f=RegExp.prototype,a=function(t,n){this._r=t,this._s=n};r(139)(a,"RegExp String",function(){var t=this._r.exec(this._s);return{value:t,done:null===t}}),e(e.P,"String",{matchAll:function(t){if(i(this),!u(t))throw TypeError(t+" is not a regexp!");var n=String(this),r="flags"in f?String(t.flags):c.call(t),e=new RegExp(t.source,~r.indexOf("g")?r:"g"+r);return e.lastIndex=o(t.lastIndex),new a(e,n)}})},function(t,n,r){"use strict";var e=r(1),i=r(181);e(e.P,"String",{padEnd:function(t){return i(this,t,arguments.length>1?arguments[1]:void 0,!1)}})},function(t,n,r){"use strict";var e=r(1),i=r(181);e(e.P,"String",{padStart:function(t){return i(this,t,arguments.length>1?arguments[1]:void 0,!0)}})},function(t,n,r){"use strict";r(82)("trimLeft",function(t){return function(){return t(this,1)}},"trimStart")},function(t,n,r){"use strict";r(82)("trimRight",function(t){return function(){return t(this,2)}},"trimEnd")},function(t,n,r){r(153)("asyncIterator")},function(t,n,r){r(153)("observable")},function(t,n,r){var e=r(1);e(e.S,"System",{global:r(3)})},function(t,n,r){for(var e=r(155),i=r(28),o=r(3),u=r(27),c=r(80),f=r(7),a=f("iterator"),s=f("toStringTag"),l=c.Array,h=["NodeList","DOMTokenList","MediaList","StyleSheetList","CSSRuleList"],v=0;v<5;v++){var p,d=h[v],y=o[d],g=y&&y.prototype;if(g){g[a]||u(g,a,l),g[s]||u(g,s,d),c[d]=l;for(p in e)g[p]||i(g,p,e[p],!0)}}},function(t,n,r){var e=r(1),i=r(151);e(e.G+e.B,{setImmediate:i.set,clearImmediate:i.clear})},function(t,n,r){var e=r(3),i=r(1),o=r(121),u=r(207),c=e.navigator,f=!!c&&/MSIE .\./.test(c.userAgent),a=function(t){return f?function(n,r){return t(o(u,[].slice.call(arguments,2),"function"==typeof n?n:Function(n)),r)}:t};i(i.G+i.B+i.F*f,{setTimeout:a(e.setTimeout),setInterval:a(e.setInterval)})},function(t,n,r){r(330),r(269),r(271),r(270),r(273),r(275),r(280),r(274),r(272),r(282),r(281),r(277),r(278),r(276),r(268),r(279),r(283),r(284),r(236),r(238),r(237),r(286),r(285),r(256),r(266),r(267),r(257),r(258),r(259),r(260),r(261),r(262),r(263),r(264),r(265),r(239),r(240),r(241),r(242),r(243),r(244),r(245),r(246),r(247),r(248),r(249),r(250),r(251),r(252),r(253),r(254),r(255),r(317),r(322),r(329),r(320),r(312),r(313),r(318),r(323),r(325),r(308),r(309),r(310),r(311),r(314),r(315),r(316),r(319),r(321),r(324),r(326),r(327),r(328),r(231),r(233),r(232),r(235),r(234),r(220),r(218),r(224),r(221),r(227),r(229),r(217),r(223),r(214),r(228),r(212),r(226),r(225),r(219),r(222),r(211),r(213),r(216),r(215),r(230),r(155),r(302),r(307),r(184),r(303),r(304),r(305),r(306),r(287),r(183),r(185),r(186),r(342),r(331),r(332),r(337),r(340),r(341),r(335),r(338),r(336),r(339),r(333),r(334),r(288),r(289),r(290),r(291),r(292),r(295),r(293),r(294),r(296),r(297),r(298),r(299),r(301),r(300),r(343),r(369),r(372),r(371),r(373),r(374),r(370),r(375),r(376),r(354),r(357),r(353),r(351),r(352),r(355),r(356),r(346),r(368),r(377),r(345),r(347),r(349),r(348),r(350),r(359),r(360),r(362),r(361),r(364),r(363),r(365),r(366),r(367),r(344),r(358),r(380),r(379),r(378),t.exports=r(52)},function(t,n){function r(t,n){if("string"==typeof n)return t.insertAdjacentHTML("afterend",n);var r=t.nextSibling;return r?t.parentNode.insertBefore(n,r):t.parentNode.appendChild(n)}t.exports=r},,,,,,,,,function(t,n,r){(function(n,r){!function(n){"use strict";function e(t,n,r,e){var i=n&&n.prototype instanceof o?n:o,u=Object.create(i.prototype),c=new p(e||[]);return u._invoke=s(t,r,c),u}function i(t,n,r){try{return{type:"normal",arg:t.call(n,r)}}catch(t){return{type:"throw",arg:t}}}function o(){}function u(){}function c(){}function f(t){["next","throw","return"].forEach(function(n){t[n]=function(t){return this._invoke(n,t)}})}function a(t){function n(r,e,o,u){var c=i(t[r],t,e);if("throw"!==c.type){var f=c.arg,a=f.value;return a&&"object"==typeof a&&m.call(a,"__await")?Promise.resolve(a.__await).then(function(t){n("next",t,o,u)},function(t){n("throw",t,o,u)}):Promise.resolve(a).then(function(t){f.value=t,o(f)},u)}u(c.arg)}function e(t,r){function e(){return new Promise(function(e,i){n(t,r,e,i)})}return o=o?o.then(e,e):e()}"object"==typeof r&&r.domain&&(n=r.domain.bind(n));var o;this._invoke=e}function s(t,n,r){var e=P;return function(o,u){if(e===F)throw new Error("Generator is already running");if(e===M){if("throw"===o)throw u;return y()}for(r.method=o,r.arg=u;;){var c=r.delegate;if(c){var f=l(c,r);if(f){if(f===A)continue;return f}}if("next"===r.method)r.sent=r._sent=r.arg;else if("throw"===r.method){if(e===P)throw e=M,r.arg;r.dispatchException(r.arg)}else"return"===r.method&&r.abrupt("return",r.arg);e=F;var a=i(t,n,r);if("normal"===a.type){if(e=r.done?M:j,a.arg===A)continue;return{value:a.arg,done:r.done}}"throw"===a.type&&(e=M,r.method="throw",r.arg=a.arg)}}}function l(t,n){var r=t.iterator[n.method];if(r===g){if(n.delegate=null,"throw"===n.method){if(t.iterator.return&&(n.method="return",n.arg=g,l(t,n),"throw"===n.method))return A;n.method="throw",n.arg=new TypeError("The iterator does not provide a 'throw' method")}return A}var e=i(r,t.iterator,n.arg);if("throw"===e.type)return n.method="throw",n.arg=e.arg,n.delegate=null,A;var o=e.arg;return o?o.done?(n[t.resultName]=o.value,n.next=t.nextLoc,"return"!==n.method&&(n.method="next",n.arg=g),n.delegate=null,A):o:(n.method="throw",n.arg=new TypeError("iterator result is not an object"),n.delegate=null,A)}function h(t){var n={tryLoc:t[0]};1 in t&&(n.catchLoc=t[1]),2 in t&&(n.finallyLoc=t[2],n.afterLoc=t[3]),this.tryEntries.push(n)}function v(t){var n=t.completion||{};n.type="normal",delete n.arg,t.completion=n}function p(t){this.tryEntries=[{tryLoc:"root"}],t.forEach(h,this),this.reset(!0)}function d(t){if(t){var n=t[w];if(n)return n.call(t);if("function"==typeof t.next)return t;if(!isNaN(t.length)){var r=-1,e=function n(){for(;++r<t.length;)if(m.call(t,r))return n.value=t[r],n.done=!1,n;return n.value=g,n.done=!0,n};return e.next=e}}return{next:y}}function y(){return{value:g,done:!0}}var g,b=Object.prototype,m=b.hasOwnProperty,x="function"==typeof Symbol?Symbol:{},w=x.iterator||"@@iterator",S=x.asyncIterator||"@@asyncIterator",_=x.toStringTag||"@@toStringTag",O="object"==typeof t,E=n.regeneratorRuntime;if(E)return void(O&&(t.exports=E));E=n.regeneratorRuntime=O?t.exports:{},E.wrap=e;var P="suspendedStart",j="suspendedYield",F="executing",M="completed",A={},N={};N[w]=function(){return this};var T=Object.getPrototypeOf,I=T&&T(T(d([])));I&&I!==b&&m.call(I,w)&&(N=I);var k=c.prototype=o.prototype=Object.create(N);u.prototype=k.constructor=c,c.constructor=u,c[_]=u.displayName="GeneratorFunction",E.isGeneratorFunction=function(t){var n="function"==typeof t&&t.constructor;return!!n&&(n===u||"GeneratorFunction"===(n.displayName||n.name))},E.mark=function(t){return Object.setPrototypeOf?Object.setPrototypeOf(t,c):(t.__proto__=c,_ in t||(t[_]="GeneratorFunction")),t.prototype=Object.create(k),t},E.awrap=function(t){return{__await:t}},f(a.prototype),a.prototype[S]=function(){return this},E.AsyncIterator=a,E.async=function(t,n,r,i){var o=new a(e(t,n,r,i));return E.isGeneratorFunction(n)?o:o.next().then(function(t){return t.done?t.value:o.next()})},f(k),k[_]="Generator",k.toString=function(){return"[object Generator]"},E.keys=function(t){var n=[];for(var r in t)n.push(r);return n.reverse(),function r(){for(;n.length;){var e=n.pop();if(e in t)return r.value=e,r.done=!1,r}return r.done=!0,r}},E.values=d,p.prototype={constructor:p,reset:function(t){if(this.prev=0,this.next=0,this.sent=this._sent=g,this.done=!1,this.delegate=null,this.method="next",this.arg=g,this.tryEntries.forEach(v),!t)for(var n in this)"t"===n.charAt(0)&&m.call(this,n)&&!isNaN(+n.slice(1))&&(this[n]=g)},stop:function(){this.done=!0;var t=this.tryEntries[0],n=t.completion;if("throw"===n.type)throw n.arg;return this.rval},dispatchException:function(t){function n(n,e){return o.type="throw",o.arg=t,r.next=n,e&&(r.method="next",r.arg=g),!!e}if(this.done)throw t;for(var r=this,e=this.tryEntries.length-1;e>=0;--e){var i=this.tryEntries[e],o=i.completion;if("root"===i.tryLoc)return n("end");if(i.tryLoc<=this.prev){var u=m.call(i,"catchLoc"),c=m.call(i,"finallyLoc");if(u&&c){if(this.prev<i.catchLoc)return n(i.catchLoc,!0);if(this.prev<i.finallyLoc)return n(i.finallyLoc)}else if(u){if(this.prev<i.catchLoc)return n(i.catchLoc,!0)}else{if(!c)throw new Error("try statement without catch or finally");if(this.prev<i.finallyLoc)return n(i.finallyLoc)}}}},abrupt:function(t,n){for(var r=this.tryEntries.length-1;r>=0;--r){var e=this.tryEntries[r];if(e.tryLoc<=this.prev&&m.call(e,"finallyLoc")&&this.prev<e.finallyLoc){var i=e;break}}i&&("break"===t||"continue"===t)&&i.tryLoc<=n&&n<=i.finallyLoc&&(i=null);var o=i?i.completion:{};return o.type=t,o.arg=n,i?(this.method="next",this.next=i.finallyLoc,A):this.complete(o)},complete:function(t,n){if("throw"===t.type)throw t.arg;return"break"===t.type||"continue"===t.type?this.next=t.arg:"return"===t.type?(this.rval=this.arg=t.arg,this.method="return",this.next="end"):"normal"===t.type&&n&&(this.next=n),A},finish:function(t){for(var n=this.tryEntries.length-1;n>=0;--n){var r=this.tryEntries[n];if(r.finallyLoc===t)return this.complete(r.completion,r.afterLoc),v(r),A}},catch:function(t){for(var n=this.tryEntries.length-1;n>=0;--n){var r=this.tryEntries[n];if(r.tryLoc===t){var e=r.completion;if("throw"===e.type){var i=e.arg;v(r)}return i}}throw new Error("illegal catch attempt")},delegateYield:function(t,n,r){return this.delegate={iterator:d(t),resultName:n,nextLoc:r},"next"===this.method&&(this.arg=g),A}}}("object"==typeof n?n:"object"==typeof window?window:"object"==typeof self?self:this)}).call(n,function(){return this}(),r(158))}])</script><script src="/./main.0cf68a.js"></script><script>!function(){!function(e){var t=document.createElement("script");document.getElementsByTagName("body")[0].appendChild(t),t.setAttribute("src",e)}("/slider.e37972.js")}()</script>



{% if theme.mermaid.enable %}
  <script src='https://unpkg.com/mermaid@{{ theme.mermaid.version }}/dist/mermaid.min.js'></script>
  <script>
    if (window.mermaid) {
      mermaid.initialize({{ JSON.stringify(theme.mermaid.options) }});
    }
  </script>
{% endif %}

    
<div class="tools-col" q-class="show:isShow,hide:isShow|isFalse" q-on="click:stop(e)">
  <div class="tools-nav header-menu">
    
    
      
      
      
    
      
      
      
    
      
      
      
    
    

    <ul style="width: 70%">
    
    
      
      <li style="width: 33.333333333333336%" q-on="click: openSlider(e, 'innerArchive')"><a href="javascript:void(0)" q-class="active:innerArchive">All Items</a></li>
      
        
      
      <li style="width: 33.333333333333336%" q-on="click: openSlider(e, 'friends')"><a href="javascript:void(0)" q-class="active:friends">Links</a></li>
      
        
      
      <li style="width: 33.333333333333336%" q-on="click: openSlider(e, 'aboutme')"><a href="javascript:void(0)" q-class="active:aboutme">About Me</a></li>
      
        
    </ul>
  </div>
  <div class="tools-wrap">
    
    	<section class="tools-section tools-section-all" q-show="innerArchive">
        <div class="search-wrap">
          <input class="search-ipt" q-model="search" type="text" placeholder="find something…">
          <i class="icon-search icon" q-show="search|isEmptyStr"></i>
          <i class="icon-close icon" q-show="search|isNotEmptyStr" q-on="click:clearChose(e)"></i>
        </div>
        <div class="widget tagcloud search-tag">
          <p class="search-tag-wording">tag:</p>
          <label class="search-switch">
            <input type="checkbox" q-on="click:toggleTag(e)" q-attr="checked:showTags">
          </label>
          <ul class="article-tag-list" q-show="showTags">
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">QR Code</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">周易大师</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Hexo Note</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">p62 Draft</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">PHMG</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Smallpdf</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">HPV30</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Nrf2</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">HPV</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">离线语音助手</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">FTP connection</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">Tags in Hexo</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Special symbols</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">FTP</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">English Correction online</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Image attached</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Root Explorer</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">RE</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">ES File Explorer</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Exiftool</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Legado</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">Document Clean-up</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Document Organization</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">SRA</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">integrity</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">vdb-validate</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Macro in Linux WPS</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Hantavirus evolution</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">LibreOffice</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">OpenOffice</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">Apache OpenOffice</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">zcat</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">ICTV</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">virus</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">International Committee on Taxonomy of Viruses (ICTV)</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">hexo index</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">hexo content</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Comments in Rmd</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">KinhDown</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">pan.baidu.com</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">SRA tumor cell lines</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Zhang Li</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">BaiduNetDisk</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">SRA analysis</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">HLA-G analysis</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">HEV Sequence Download</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Yin Wenjiao</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Guo Yifan</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">E-MTAB-1733</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">AnyDesk update</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Ji Tianjiao</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Size reduction of PDF</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">tBHQ</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">Tert-Butylhydroquinone</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Authentication Required</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Gene Report</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">BioGPS</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Usage of my mobile device</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">Windows Docker Machine</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">container</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Tips of Hexo GitHub Pages</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Build Github Pages With Jekyll</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">SDS-PAGE</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Protein Marker</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">txid</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">ETE/ETE3</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Minoxidil</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">p62</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">E6</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">E7</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">wget</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">rsync</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">SRA-Tools</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">miRBase</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">RNA-Seq</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Endometrial Cancer</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Wang Jiao</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">Sex Determination</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">endometrial cancer</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">HLA-G</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">ACNUC</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Microsoft Office</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">pdftk</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">PDF password</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">MUSCLE</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">MSA</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">RGB color</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">VPN</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">AppImage</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">Outline</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Watch charging</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">代码的魅力</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">Flatpak</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">Shotcut</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">HEV</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">NSFC</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">E-MTAB-173</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Essential</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Kaiju</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Phage RnD</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">Mobile phone</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">NCBI Datasets</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Hexo categories</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">COVID-19 Data Download</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Glossary</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Bacteriophage</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">ContraFect</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Intralytix</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">phagelux</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">菲吉乐科</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">QQ浏览器</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Word</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Flu Biosafety</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">EBV Detection</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Hexo Categories</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">COVID-19  Vaccine</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Nrf2-HSV</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Backup</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">HSV Exosome cn draft</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">B95-8_Nrf2 project</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">B95-8-Nrf2 project</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Offline map</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Google Patents</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">File manager</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Reactive Oxygen Species</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Herpesviridae sequence download</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">HFMD</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">STGCN Model</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">Offline Wikipedia</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">flatpak</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">Installation Log in X1</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Virus proliferation cell free system</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">CEB</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">COVID-19 Vaccine</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">PDF_Learning</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">R update</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">CRAN update</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Cairo-Dock</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Scheme of README documents</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">Bacteriophage database</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Thesis Nrf2</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">凉山项目</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">凉山</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Hexo</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">HSV</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">CRAN</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Monkey B virus</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">恢复真实容量</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Pen drive Encrypting</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">中文语音识别</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">speech recognition</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">scrcpy</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">ADB</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">山西疾控</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">审计</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Antibody Purification</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Water</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">算命</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">七千人大会</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Long COVID-19</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">COVID-19</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">Vaccine</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">疾病负担</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">BOD</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">Burden of Disease</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Collison</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Nebulizer</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">气溶胶发生器</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Figma</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Chinese conversion</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">OpenCC</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">中文简繁转换</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">pH meter</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Potassium Chloride</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">KCl</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Android File Transfer</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">openvpn</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">openvpn3</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">WINE</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">R</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Gist</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">miR</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">HSV-1</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">Exosome</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">miR prediction</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Primer organization</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">国家自然科学基金</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">NFS</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Kiwix</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Wikipedia</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">HGNC</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">file extension</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">RNA-Seq pipeline</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">HEV sequence download</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">HEVnet</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">CSV</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Excel</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Sheet</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Tree structure</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Markdown syntax</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">西部世界</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">IME in Ubuntu</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Markdown Mermaid examples</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Markdown mermaid</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">flowchart</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">MarkText</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Small RNA</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">PDF Note</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Markdown</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Text Editor</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Obsidian</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Gingko</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Manuskript</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">FocusWriter</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Typora</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">TreeSheets</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Zettlr</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Protein-Protein Docking</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">PPI</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Margarine</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">Protein Assay</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Proteomics</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">GanttProject</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Tasker</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">Shortcuts</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color1">Agent</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">氢原子</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color5">氢自由基</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">My Linux</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Qv2ray</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">HIV</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">Scheme</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color4">HD Management</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color2">README</a>
              </li>
             
              <li class="article-tag-list-item">
                <a href="javascript:void(0)" class="js-tag color3">HD Structure</a>
              </li>
            
            <div class="clearfix"></div>
          </ul>
        </div>
        <ul class="search-ul">
          <p q-show="jsonFail" style="padding: 20px; font-size: 12px;">
            缺失模块。<br/>1、请确保node版本大于6.2<br/>2、在博客根目录（注意不是yilia根目录）执行以下命令：<br/> npm i hexo-generator-json-content --save<br/><br/>
            3、在根目录_config.yml里添加配置：
<pre style="font-size: 12px;" q-show="jsonFail">
  jsonContent:
    meta: false
    pages: false
    posts:
      title: true
      date: true
      path: true
      text: false
      raw: false
      content: false
      slug: false
      updated: false
      comments: false
      link: false
      permalink: false
      excerpt: false
      categories: false
      tags: true
</pre>
          </p>
          <li class="search-li" q-repeat="items" q-show="isShow">
            <a q-attr="href:path|urlformat" class="search-title"><i class="icon-quo-left icon"></i><span q-text="title"></span></a>
            <p class="search-time">
              <i class="icon-calendar icon"></i>
              <span q-text="date|dateformat"></span>
            </p>
            <p class="search-tag">
              <i class="icon-price-tags icon"></i>
              <span q-repeat="tags" q-on="click:choseTag(e, name)" q-text="name|tagformat"></span>
            </p>
          </li>
        </ul>
    	</section>
    

    
    	<section class="tools-section tools-section-friends" q-show="friends">
  		
        <ul class="search-ul">
          
            <li class="search-li">
              <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" class="search-title"><i class="icon-quo-left icon"></i>Old entry NCBI</a>
            </li>
          
            <li class="search-li">
              <a href="https://www.nejm.org/" target="_blank" class="search-title"><i class="icon-quo-left icon"></i>The New England J of Medicine</a>
            </li>
          
            <li class="search-li">
              <a href="https://www.thelancet.com/" target="_blank" class="search-title"><i class="icon-quo-left icon"></i>Lancet</a>
            </li>
          
            <li class="search-li">
              <a href="https://www.nature.com/nature/current-issue" target="_blank" class="search-title"><i class="icon-quo-left icon"></i>Nature</a>
            </li>
          
            <li class="search-li">
              <a href="https://www.sciencemag.org/" target="_blank" class="search-title"><i class="icon-quo-left icon"></i>Science</a>
            </li>
          
            <li class="search-li">
              <a href="https://www.cell.com/" target="_blank" class="search-title"><i class="icon-quo-left icon"></i>Cell</a>
            </li>
          
            <li class="search-li">
              <a href="https://www.pnas.org/" target="_blank" class="search-title"><i class="icon-quo-left icon"></i>PNAS</a>
            </li>
          
            <li class="search-li">
              <a href="https://www.jbc.org/" target="_blank" class="search-title"><i class="icon-quo-left icon"></i>JBC</a>
            </li>
          
            <li class="search-li">
              <a href="https://rupress.org/jcb" target="_blank" class="search-title"><i class="icon-quo-left icon"></i>JCB</a>
            </li>
          
            <li class="search-li">
              <a href="https://clinicaltrials.gov/" target="_blank" class="search-title"><i class="icon-quo-left icon"></i>ClinicalTrials.gov</a>
            </li>
          
            <li class="search-li">
              <a href="http://192.168.1.10:8000/" target="_blank" class="search-title"><i class="icon-quo-left icon"></i>MarginNote 3</a>
            </li>
          
            <li class="search-li">
              <a href="https://my.pcloud.com/" target="_blank" class="search-title"><i class="icon-quo-left icon"></i>pCloud</a>
            </li>
          
        </ul>
  		
    	</section>
    

    
    	<section class="tools-section tools-section-me" q-show="aboutme">
  	  	
  	  		<div class="aboutme-wrap" id="js-aboutme">Step by step.</div>
  	  	
    	</section>
    
  </div>
  
</div>
    <!-- Root element of PhotoSwipe. Must have class pswp. -->
<div class="pswp" tabindex="-1" role="dialog" aria-hidden="true">

    <!-- Background of PhotoSwipe. 
         It's a separate element as animating opacity is faster than rgba(). -->
    <div class="pswp__bg"></div>

    <!-- Slides wrapper with overflow:hidden. -->
    <div class="pswp__scroll-wrap">

        <!-- Container that holds slides. 
            PhotoSwipe keeps only 3 of them in the DOM to save memory.
            Don't modify these 3 pswp__item elements, data is added later on. -->
        <div class="pswp__container">
            <div class="pswp__item"></div>
            <div class="pswp__item"></div>
            <div class="pswp__item"></div>
        </div>

        <!-- Default (PhotoSwipeUI_Default) interface on top of sliding area. Can be changed. -->
        <div class="pswp__ui pswp__ui--hidden">

            <div class="pswp__top-bar">

                <!--  Controls are self-explanatory. Order can be changed. -->

                <div class="pswp__counter"></div>

                <button class="pswp__button pswp__button--close" title="Close (Esc)"></button>

                <button class="pswp__button pswp__button--share" style="display:none" title="Share"></button>

                <button class="pswp__button pswp__button--fs" title="Toggle fullscreen"></button>

                <button class="pswp__button pswp__button--zoom" title="Zoom in/out"></button>

                <!-- Preloader demo http://codepen.io/dimsemenov/pen/yyBWoR -->
                <!-- element will get class pswp__preloader--active when preloader is running -->
                <div class="pswp__preloader">
                    <div class="pswp__preloader__icn">
                      <div class="pswp__preloader__cut">
                        <div class="pswp__preloader__donut"></div>
                      </div>
                    </div>
                </div>
            </div>

            <div class="pswp__share-modal pswp__share-modal--hidden pswp__single-tap">
                <div class="pswp__share-tooltip"></div> 
            </div>

            <button class="pswp__button pswp__button--arrow--left" title="Previous (arrow left)">
            </button>

            <button class="pswp__button pswp__button--arrow--right" title="Next (arrow right)">
            </button>

            <div class="pswp__caption">
                <div class="pswp__caption__center"></div>
            </div>

        </div>

    </div>

</div>
  </div>
</body>
</html>